---
title: |
  | Supplementary Materials for
  |
  | **An integrative genomic and phenomic analysis to investigate the nature of plant species in _Escallonia_ (Escalloniaceae)**
author: "Sarah J. Jacobs, Michael C. Grundler, Claudia L. Henriquez, Felipe Zapata"
fontsize: 11pt
geometry: margin=0.8in
header-includes:
  - \usepackage{float,xcolor,lineno,caption}
  - \floatplacement{figure}{H}  
output:
  bookdown::pdf_document2:
    includes:
    fig_caption: yes
    keep_tex: yes
    md_extensions: +raw_attribute
    toc: true
    toc_depth: 4
    number_sections: true
linkcolor: black
bibliography: references.bib
csl: scientific-reports.csl
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(out.extra = '', eval = TRUE)
opts <- options(knitr.kable.NA="")
```

```{r libraries, include=FALSE}
library(float)
library(tidyverse)
library(patchwork)
library(knitr)
library(reshape)
library(kableExtra)
library(ggtree)
library(treeio)
library(ape)
library(phytools)
library(RColorBrewer)
library(ggstance)
library(GoodmanKruskal)
library(gridExtra)
library(eulerr)
```

```{r read_data, echo=FALSE, message=FALSE, warning=FALSE}

escalloniaSampling = read_csv("supplementary_material_files/allClades_dataForSamplingTable.csv")

```

```{r themes, echo=FALSE}
# set up base map and theme
mapa = borders("world",
               regions = c("Brazil",
                           "Uruguay",
                           "Argentina",
                           "French Guiana",
                           "Suriname",
                           "Colombia",
                           "Venezuela",
                           "Bolivia",
                           "Ecuador",
                           "Chile",
                           "Paraguay",
                           "Peru",
                           "Guyana",
                           "Panama",
                           "Costa Rica",
                           "Nicaragua",
                           "Honduras",
                           "El Salvador",
                           "Belize",
                           "Guatemala",
                           "Mexico",
                           "Trinidad and Tobago",
                           "Caribe",
                           "Puerto Rico",
                           "Dominican Republic",
                           "Haiti",
                           "Jamaica",
                           "Cuba",
                           "Bahamas",
                           "Antiles",
                           "Dominica",
                           "Saba"),
               fill = "white",
               colour = "black",
               size = 0.25)
myBaseMap = ggplot() +
  mapa +
  theme_bw() +
  xlab("Longitude (decimals)") +
  ylab("Latitude (decimals)") +
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "grey60"),
        panel.grid.minor = element_blank())
```

```{r CladesamplingCounts, echo=FALSE}

# Clade I Totals
CladeItotalPhenome = escalloniaSampling %>%
  filter(CladeID == "I" & is.na(`no measurements`)) %>%
  nrow()
CladeIphenomeFertile = escalloniaSampling %>%
  filter(CladeID == "I" & `Floral and Leaf` == 'x') %>%
  nrow()
CladeIphenomeSterile = escalloniaSampling %>%
  filter(CladeID == "I" & `Leaf only` == 'x') %>%
  nrow()
CladeItotalGenome = escalloniaSampling %>%
  filter(CladeID == "I" & libraryNumber != 'NA') %>%
  nrow()
CladeIgenomeFertile = escalloniaSampling %>%
  filter(CladeID == "I" & Genogroup != 'NA' & Phenogroup != 'NA') %>%
  nrow()
CladeIgenomeSterile = escalloniaSampling %>%
  filter(CladeID == "I" & `Leaf only` == 'x' & libraryNumber != 'NA') %>%
  nrow()
CladeIgenomeNoPhenome = escalloniaSampling %>%
  filter(CladeID == "I" & libraryNumber != 'NA' & `no measurements` == 'x') %>%
  nrow()
CladeItotalSampling = escalloniaSampling %>%
  filter(CladeID == "I") %>%
  nrow()

#Clade II Totals
CladeIItotalPhenome = escalloniaSampling %>%
  filter(CladeID == "II" & is.na(`no measurements`)) %>%
  nrow()
CladeIIphenomeFertile = escalloniaSampling %>%
  filter(CladeID == "II" & `Floral and Leaf` == 'x') %>%
  nrow()
CladeIIphenomeSterile = escalloniaSampling %>%
  filter(CladeID == "II" & `Leaf only` == 'x') %>%
  nrow()
CladeIItotalGenome = escalloniaSampling %>%
  filter(CladeID == "II" & libraryNumber != 'NA') %>%
  nrow()
CladeIIgenomeFertile = escalloniaSampling %>%
  filter(CladeID == "II" & Genogroup != 'NA' & Phenogroup != 'NA') %>%
  nrow()
CladeIIgenomeSterile = escalloniaSampling %>%
  filter(CladeID == "II" & `Leaf only` == 'x' & libraryNumber != 'NA') %>%
  nrow()
CladeIIgenomeNoPhenome = escalloniaSampling %>%
  filter(CladeID == "II" & libraryNumber != 'NA' & `no measurements` == 'x') %>%
  nrow()
CladeIItotalSampling = escalloniaSampling %>%
  filter(CladeID == "II") %>%
  nrow()

# Clade III Totals
CladeIIItotalPhenome = escalloniaSampling %>%
  filter(CladeID == "III" & is.na(`no measurements`)) %>%
  nrow()
CladeIIIphenomeFertile = escalloniaSampling %>%
  filter(CladeID == "III" & `Floral and Leaf` == 'x') %>%
  nrow()
CladeIIIphenomeSterile = escalloniaSampling %>%
  filter(CladeID == "III" & `Leaf only` == 'x') %>%
  nrow()
CladeIIItotalGenome = escalloniaSampling %>%
  filter(CladeID == "III" & libraryNumber != 'NA') %>%
  nrow()
CladeIIIgenomeFertile = escalloniaSampling %>%
  filter(CladeID == "III" & Genogroup != 'NA' & Phenogroup != 'NA') %>%
  nrow()
CladeIIIgenomeSterile = escalloniaSampling %>%
  filter(CladeID == "III" & `Leaf only` == 'x' & libraryNumber != 'NA') %>%
  nrow()
CladeIIIgenomeNoPhenome = escalloniaSampling %>%
  filter(CladeID == "III" & libraryNumber != 'NA' & `no measurements` == 'x') %>%
  nrow()
CladeIIItotalSampling = escalloniaSampling %>%
  filter(CladeID == "III") %>%
  nrow()

# Clade IV Totals
CladeIVtotalPhenome = escalloniaSampling %>%
  filter(CladeID == "IV" & is.na(`no measurements`)) %>%
  nrow()
CladeIVphenomeFertile = escalloniaSampling %>%
  filter(CladeID == "IV" & `Floral and Leaf` == 'x') %>%
  nrow()
CladeIVphenomeSterile = escalloniaSampling %>%
  filter(CladeID == "IV" & `Leaf only` == 'x') %>%
  nrow()
CladeIVtotalGenome = escalloniaSampling %>%
  filter(CladeID == "IV" & libraryNumber != 'NA') %>%
  nrow()
CladeIVgenomeFertile = escalloniaSampling %>%
  filter(CladeID == "IV" & Genogroup != 'NA' & Phenogroup != 'NA') %>%
  nrow()
CladeIVgenomeSterile = escalloniaSampling %>%
  filter(CladeID == "IV" & `Leaf only` == 'x' & libraryNumber != 'NA') %>%
  nrow()
CladeIVgenomeNoPhenome = escalloniaSampling %>%
  filter(CladeID == "IV" & libraryNumber != 'NA' & `no measurements` == 'x') %>%
  nrow()
CladeIVtotalSampling = escalloniaSampling %>%
  filter(CladeID == "IV") %>%
  nrow()

# Clade V Totals
CladeVtotalPhenome = escalloniaSampling %>%
  filter(CladeID == "V" & is.na(`no measurements`)) %>%
  nrow()
CladeVphenomeFertile = escalloniaSampling %>%
  filter(CladeID == "V" & `Floral and Leaf` == 'x') %>%
  nrow()
CladeVphenomeSterile = escalloniaSampling %>%
  filter(CladeID == "V" & `Leaf only` == 'x') %>%
  nrow()
CladeVtotalGenome = escalloniaSampling %>%
  filter(CladeID == "V" & libraryNumber != 'NA') %>%
  nrow()
CladeVgenomeFertile = escalloniaSampling %>%
  filter(CladeID == "V" & Genogroup != 'NA' & Phenogroup != 'NA') %>%
  nrow()
CladeVgenomeSterile = escalloniaSampling %>%
  filter(CladeID == "V" & `Leaf only` == 'x' & libraryNumber != 'NA') %>%
  nrow()
CladeVgenomeNoPhenome = escalloniaSampling %>%
  filter(CladeID == "V" & libraryNumber != 'NA' & `no measurements` == 'x') %>%
  nrow()
CladeVtotalSampling = escalloniaSampling %>%
  filter(CladeID == "V") %>%
  nrow()

# Clade VI Totals
CladeVItotalPhenome = escalloniaSampling %>%
  filter(CladeID == "VI" & is.na(`no measurements`)) %>%
  nrow()
CladeVIphenomeFertile = escalloniaSampling %>%
  filter(CladeID == "VI" & `Floral and Leaf` == 'x') %>%
  nrow()
CladeVIphenomeSterile = escalloniaSampling %>%
  filter(CladeID == "VI" & `Leaf only` == 'x') %>%
  nrow()
CladeVItotalGenome = escalloniaSampling %>%
  filter(CladeID == "VI" & libraryNumber != 'NA') %>%
  nrow()
CladeVIgenomeFertile = escalloniaSampling %>%
  filter(CladeID == "VI" & Genogroup != 'NA' & Phenogroup != 'NA') %>%
  nrow()
CladeVIgenomeSterile = escalloniaSampling %>%
  filter(CladeID == "VI" & `Leaf only` == 'x' & libraryNumber != 'NA') %>%
  nrow()
CladeVIgenomeNoPhenome = escalloniaSampling %>%
  filter(CladeID == "VI" & libraryNumber != 'NA' & `no measurements` == 'x') %>%
  nrow()
CladeVItotalSampling = escalloniaSampling %>%
  filter(CladeID == "VI") %>%
  nrow()

```

```{r currentStateOfTaxonomyCladeResults, echo=FALSE, message=FALSE, warning=FALSE}
# Read results files and set the assignment possibilities for plotting (for inHull assignments only)

# Clade I
melted_CladeI_hullResults_All = as.data.frame(read_csv("supplementary_material_files/CladeI_resultsFiles/melted_CladeI_hullResults_All.csv"))
melted_CladeI_hullResults_All$col = factor(melted_CladeI_hullResults_All$col, levels = c("assignment", "no assignment"))
melted_CladeI_overlapProp_All = as.data.frame(read_csv("supplementary_material_files/CladeI_resultsFiles/melted_CladeI_overlapProp_All.csv"))

colnames(melted_CladeI_overlapProp_All) = c("uniqueID",
                                            "X1",
                                            "X2",
                                            "value")

cladeIassigned = melted_CladeI_hullResults_All %>%
  filter(melted_CladeI_hullResults_All$col=='assignment') %>%
  count()
cladeInotassigned = melted_CladeI_hullResults_All %>%
  filter(melted_CladeI_hullResults_All$col=='no assignment') %>%
  count()
cladeIpercentassigned = round(cladeIassigned/cladeInotassigned,
                              digits = 4)

#Clade II
melted_CladeII_hullResults_All = as.data.frame(read_csv("supplementary_material_files/CladeII_resultsFiles/melted_CladeII_hullResults_All.csv"))

melted_CladeII_hullResults_All$col = factor(melted_CladeII_hullResults_All$col, levels = c("assignment", "no assignment"))

melted_CladeII_overlapProp_All = as.data.frame(read_csv("supplementary_material_files/CladeII_resultsFiles/melted_CladeII_overlapProp_All.csv"))

colnames(melted_CladeII_overlapProp_All) = c("uniqueID",
                                            "X1",
                                            "X2",
                                            "value")

cladeIIassigned = melted_CladeII_hullResults_All %>%
  filter(melted_CladeII_hullResults_All$col=='assignment') %>%
  count()

cladeIInotassigned = melted_CladeII_hullResults_All %>%
  filter(melted_CladeII_hullResults_All$col=='no assignment') %>%
  count()

cladeIIpercentassigned = round(cladeIIassigned/cladeIInotassigned,
                               digits = 4)

# Clade III
melted_CladeIII_hullResults_All = as.data.frame(read_csv("supplementary_material_files/CladeIII_resultsFiles/melted_CladeIII_hullResults_All.csv"))

melted_CladeIII_hullResults_All$col = factor(melted_CladeIII_hullResults_All$col, levels = c("assignment", "no assignment"))

melted_CladeIII_overlapProp_All = as.data.frame(read_csv("supplementary_material_files/CladeIII_resultsFiles/melted_CladeIII_overlapProp_All.csv"))

colnames(melted_CladeIII_overlapProp_All) = c("uniqueID",
                                            "X1",
                                            "X2",
                                            "value")

cladeIIIassigned = melted_CladeIII_hullResults_All %>%
  filter(melted_CladeIII_hullResults_All$col=='assignment') %>%
  count()

cladeIIInotassigned = melted_CladeIII_hullResults_All %>%
  filter(melted_CladeIII_hullResults_All$col=='no assignment') %>%
  count()

cladeIIIpercentassigned = round(cladeIIIassigned/cladeIIInotassigned,
                                digits = 4)

# Clade IV
melted_CladeIV_hullResults_All = as.data.frame(read_csv("supplementary_material_files/CladeIV_resultsFiles/melted_CladeIV_hullResults_All.csv"))

melted_CladeIV_hullResults_All$col = factor(melted_CladeIV_hullResults_All$col, levels = c("assignment", "no assignment"))

melted_CladeIV_overlapProp_All = as.data.frame(read_csv("supplementary_material_files/CladeIV_resultsFiles/melted_CladeIV_overlapProp_All.csv"))

colnames(melted_CladeIV_overlapProp_All) = c("uniqueID",
                                            "X1",
                                            "X2",
                                            "value")

cladeIVassigned = melted_CladeIV_hullResults_All %>%
  filter(melted_CladeIV_hullResults_All$col=='assignment') %>%
  count()

cladeIVnotassigned = melted_CladeIV_hullResults_All %>%
  filter(melted_CladeIV_hullResults_All$col=='no assignment') %>%
  count()

cladeIVpercentassigned = round(cladeIVassigned/cladeIVnotassigned,
                               digits = 4)

# Clade V
melted_CladeV_hullResults_All = as.data.frame(read_csv("supplementary_material_files/CladeV_resultsFiles/melted_CladeV_hullResults_All.csv"))

melted_CladeV_hullResults_All$col = factor(melted_CladeV_hullResults_All$col, levels = c("assignment", "no assignment"))

melted_CladeV_overlapProp_All = as.data.frame(read_csv("supplementary_material_files/CladeV_resultsFiles/melted_CladeV_overlapProp_All.csv"))

colnames(melted_CladeV_overlapProp_All) = c("uniqueID",
                                            "X1",
                                            "X2",
                                            "value")

cladeVassigned = melted_CladeV_hullResults_All %>%
  filter(melted_CladeV_hullResults_All$col=='assignment') %>%
  count()

cladeVnotassigned = melted_CladeV_hullResults_All %>%
  filter(melted_CladeV_hullResults_All$col=='no assignment') %>%
  count()

cladeVpercentassigned = round(cladeVassigned/cladeVnotassigned,
                              digits = 4)

# Clade VI
melted_CladeVI_hullResults_All = as.data.frame(read_csv("supplementary_material_files/CladeVI_resultsFiles/melted_CladeVI_hullResults_All.csv"))

melted_CladeVI_hullResults_All$col = factor(melted_CladeVI_hullResults_All$col, levels = c("assignment", "no assignment"))

melted_CladeVI_overlapProp_All = as.data.frame(read_csv("supplementary_material_files/CladeVI_resultsFiles/melted_CladeVI_overlapProp_All.csv"))

colnames(melted_CladeVI_overlapProp_All) = c("uniqueID",
                                            "X1",
                                            "X2",
                                            "value")

cladeVIassigned = melted_CladeVI_hullResults_All %>%
  filter(melted_CladeVI_hullResults_All$col=='assignment') %>%
  count()

cladeVInotassigned = melted_CladeVI_hullResults_All %>%
  filter(melted_CladeVI_hullResults_All$col=='no assignment') %>%
  count()

cladeVIpercentassigned = round(cladeVIassigned/cladeVInotassigned,
                               digits = 4)

```

# Materials and Methods

## Sampling

**Taxon sampling** All $848$ specimens included in the study were assigned to $29$ taxonomic species according to the only genus-wide taxonomic monograph [@Sleumer:1968ts]. These specimens covered the geographic range of all taxonomic species whenever possible (i.e., specimens came from well-spaced localities across the geographic range of each species). Specimens assigned to _E. virgata_ and _E. pulverulenta_ were excluded from our study because in initial phylogenetic analyses these taxonomic species did not show stable and well supported phylogenetic relationships. Specimens assigned to _E. callcottiae, E. chlorophylla, E. cordobensis, E. gayana, E. harrisii, E. hispida, E. obtusissima, E. salicifolia_, and _E. serrata_ were not included in our study because we could not find enough herbarium specimens to reliably investigate the nature of these taxonomic species, we failed to extract ancient DNA from old herbarium specimens, or we could not locate any populations of these taxonomic species in the field.

**Phenomic data collection** We meaured leaf length, leaf width, pedicel length, ovary length, length of calyx tube, length of calyx lobes, petal length, petal width, filament length, style length to characterize the geographic pattern of phenotypic variation across _Escallonia_. We chose those traits because they have been used in previous taxonomic studies to characterize and delimit all taxonomic species in this genus [@Sleumer:1968ts; See also @sede2018taxonomic]. Furthermore, after our careful study of ca. $3,500$ herbarium collections, we confirmed these traits are variable across the entire geographic range of this genus. We used only mature leaves and flowers in all specimens to measure each phenotypic trait. All traits were measured using a digital caliper (Digimatic CD-6” CS, Mitutoyo Japan). Vegetative traits were measured on dried specimens. Floral traits were measured on flowers that were rehydrated and examined on a stereoscopic dissecting microscope (SMZ645, Nikon USA). All measurements were recorded from three different structures (i.e., three flowers) for each specimen whenever possible, and then averaged to generate character measurements for each specimen.

**Genomic data collection** Genomic DNA extraction and purification was performed following a modified version of the CTAB extraction protocol [@doyle1987rapid; @Cullings1992] that incorporates a pre-wash step [@li2007optimized] to aid in the removal of polyphenols and proteins prior to extraction. Genomic DNA quantification was performed using a Qubit fluorometer v.3.0 (Invitrogen by Thermo Fisher Scientific, Carlsbad, CA, USA) and an Agilent 2200 TapeStation (Agilent Technologies, Santa Clara, CA, USA). Prior to sequencing, size selection for fragment length of $375-525$ bp was performed on a PippenPrep (Sage Science, Beverly, MA, USA). Libraries were pooled in groups of $96$ specimens and sequenced across multiple lanes of 100PE sequencing on the Illumina HiSeq4000 Sequencing Platform at the Broad Stem Cell Research Center at the University of California, Los Angeles. To determine potential batch effects, we prepared and sequenced a subset of the same specimens across different lanes. Though we recovered different number of sequence reads across replicated runs, the loci were largely repeatable and duplicated specimens always clustered together in preliminary phylogenetic analyses. Therefore, we inferred no noticeable batch effects. We used \texttt{iPyrad} v0.7.28 [@eaton2020ipyrad] to demultiplex, filter, assemble, and call variants. We filtered reads with a Phred score less than $33$ at more than five positions and required a minimum of $10$ reads for statistical base calling. We used a clustering threshold of $0.90$ sequence similarity to cluster a maximum of $5,000$ reads into loci. We required at least four specimens to include a locus to generate an initial data matrix. We used \texttt{VCFtools} v0.1.14 [@danecek2011variant] and custom-made scripts to further filter this data matrix. We first filtered specimens with missing data from $95\%$ of loci, retained only biallelic sites, and then chose the single nucleotide polymorphism (SNP) closest to the center of each locus to minimize the effects of linkage disequilibrium for some analyses. For other analyses, we used the full sequence per locus (See below). We used the same approach to generate three matrices with different amounts of missing data by first filtering loci missing from $25\%$, $50\%$, and $75\%$ of the specimens and then applying the same filters described above. For analysis within clades, we removed the outgroup ( _Valdivia gayana_ ) and applied our assembly and filtering strategy for each clade independently (See Results).

## Phenomics

**Current state of taxonomic species** Based on the most recent taxonomic monograph [@Sleumer:1968ts], we extracted the minimum and maximum values reported for ten quantitative traits used in the descriptions of all taxonomic species. In a few selected cases, the taxonomic description provided a single value for a given trait. Because it is impossible for any quantitative continuous trait to be fixed with no variation (or measurement error) in nature, we conservatively added and subtracted $0.5$ units (in our case mm.) to the reported value to create a range. When this adjustment resulted in a minimum value $\leqslant 0$, we adjusted this value to be $>0$ according to the precision of our measurement tool ($0.1$ mm.). For example, when the reported measurement was 0.5 mm., the standard procedure would indicate the range to be $[0,1]$ mm. Therefore, we adjusted this range to be $[0.1,1]$ mm. Of the $39$ taxonomic species described in the taxonomic monograph [@Sleumer:1968ts], $11$ included subspecies. Because we only considered variation at the level of species, we subsumed subspecific variation into cumulative values at the species level. Hence, we conservatively made the range of species with subspecies as inclusive as possible.

**Model-based evidence for species** The fundamental model for the distribution of continuous quantitative traits within a species is well-grounded in quantitative genetics and evolutionary theory [@fisher1919xv]. In addition to the assumption of polygenic inheritance and random mating, it also assumes that phenotypic differences between species do not reflect ontogenetic or environmentally-induced variation [@templeton2006population]. Because we only measured mature organs (See above), we inferred that any phenotypic differences did not reflect ontogenetic variation. Additionally, though the genetic architecture of the phenotypic traits we measured in _Escallonia_ is not known, previous studies in other angiosperms have shown that these vegetative [@frary2003qtl; @frary2004comparative; @ferris2015leaf] and reproductive [@brock2010floral; @abraham2013natural; @zhang2018genetic] traits are polygenic, with limited environmental effects. Therefore, we assumed the the same genetic architecture for the traits used here. Lastly, no formal studies on the reproductive biology of _Escallonia_ are available; however, observations of populations in the field do not indicate any skew in reproductive mode (pers. obs.). Therefore, we assumed these plants display random mating. The Fisherian model, of course, has stronger explanatory power in an explicit spatial context, particularly in situations that afford the opportunity for species to come into contact [@futuyma2013evolution]. Because our continent-wide sampling includes specimens from multiple populations in contact, we could examine phenotypic variation when species co-occur.

In addition to the information criteria used to determine the best fit model, we used a likelihood ratio test (LRT) to assess whether models with nested numbers of phenogroups of equal shape and orientation as the best fit model were appropriate. To carry out this analysis, we specified the parameters of the best fit model, sequentially increased the number of phenogroups, and estimated the LRT statistics (LRTs) comparing the simple model (i.e., fewer phenogroups) _vs._ the more complex model (i.e., more phenogroups). A large value of the LRTs provides evidence against the simpler model. To assess significance of the LRT, we created a null distribution of LRTs using bootstrap simulations with $999$ replicates. For this analysis, we used \texttt{mclust} v5.4.5 [@Scrucca:2016wq].

## Genomics

**Model-based evidence for species**

_Sensitivity Tests_ In order to assess the sensitivity of our analyses to the amount of missing data, we used the three matrices ($25\%$, $50\%$, $75\%$ missing data) in each clade for three kinds of analysis. First, we used a concatenated matrix with complete sequences for all loci to run a phylogenetic analysis in \texttt{IQ-TREE} v2.0.3 with ultrafast bootstrap approximation to assess branch support [@hoang2018ufboot2; @minh2020iq]. Second, we used our matrix with one SNP per locus to run Principal Component Analysis (PCA) [@patterson2006population] and visually detect clusters. For this analysis, we used the R package \texttt{SNPRelate} v4.0 [@zheng2012high]. Third, using the same matrix we used for PCA, we ran model-based clustering to detect ancestry of specimens [@alexander2009admixture]. For this analysis, we ran \texttt{ADMIXTURE} [@alexander2009admixture] independently $10$ times specifying $1$ to $n + n/2$ number of genomic clusters, where $n$ is equal to the number of taxonomic species currently hypothesized to exist within each clade.  

_RI model (reproductive isolation model)_ \texttt{BPP} requires that specimens are assigned to demes _a priori_. To infer the number of demes and the assignment of specimens to demes, we first ran the software \texttt{STRUCTURE} v2.3.4 [@pritchard2000inference] with default priors for $10$ replicates in parallel [@besnier2013parallelstructure]. In each replicate, we fitted different models specifying $1$ to $n + n/2$ number of demes, where $n$ is equal to the number of taxonomic species currently hypothesized to exist within each clade. We recorded $1,000,000$ samples from the posterior probability after applying a burnin of $10\%$, summarized the results across replicates with \texttt{StructureHarvester} v0.6.94 [@earl2012structure], and used the $\Delta$K statistic to select the best fit model [@evanno2005detecting]. Based on the best supported model, we assigned specimens to demes using the software \texttt{CLUMPP} v1.1.2 [@jakobsson2007clumpp]. Second, we used the software \texttt{rMaverick} v1.0.5 [@verity2016estimating] which uses thermodynamic integration instead of the heuristic estimators used in \texttt{STRUCTURE} to infer the number of demes and the assignment of specimens to demes. For this analysis, we used the same data matrix and fitted the same models specifying different number of demes as with \texttt{STRUCTURE}. We used an admixture model ($\alpha=0.1$) with default priors and ran $20$ rungs. Each rung used $30,000$ sample iterations and $20\%$ samples as burn-in. To assess convergence of the results, we plotted the weighted log-likelihoods for each rung and assessed for smooth transitions between rungs. We chose the best fit model based on the highest posterior probability.

In addition to deme assignments, \texttt{BPP} requires specifying prior probabilities for the species delimitation/species tree models and prior probabilities within each of these models for the population sizes parameter ($\theta$) and the age of the root in each species tree ($\tau_0$). For the species delimitation/species tree models, we used the default prior (Prior 1), which assigns equal probabilities to the rooted species trees [@yang2014unguided]. Within each species delimitation/species tree model, we assigned the inverse-gamma prior ($\alpha=2$, $\beta=0.001$ or $\beta=0.01$) for both $\theta$ and $\tau_0$. We assigned the same priors across all clades. 

\texttt{BPP} uses a reversible-jump Markov chain Monte Carlo (rjMCMC) algorithm to move between different species delimitation models [@Yang:2010jz]. To determine proper mixing of the rjMCMC, we ran a series of replicated analyses within each clade. When we assigned specimens to two demes, we ran four replicates. When we assigned specimens to three or more demes, we ran eight replicates. Across replicates, we changed the algorithms ($0$ and $1$) implemented in \texttt{BPP} [@Yang:2010jz] and the random starting tree. We further replicated the same series of analyses but using ca. $5\%$ of the loci randomly sampled without replacement. Because analyses with fewer loci are less susceptible to mixing problems, we used results from these analyses to indicate potential issues with the larger datasets.

_Model selection_ It was computationally infeasible to use BFD\* in \texttt{SNAPP} with the full datasets given the large number of specimens and loci within each clade (See Results). Therefore, we subsampled each genogroup to two specimens maximizing the data available (genomics, phenomics, and geographic distribution) and then reduced the number of loci to a computationally tractable size. To accomplish this, we used a concordance factor analysis as implemented \texttt{IQ-TREE} v2.0.3 [@minh2020new]. For this analysis, we first inferred phylogenies for each locus using \texttt{IQ-TREE} (coupled with model selection) [@kalyaanamoorthy2017modelfinder]. We then used the phylogeny generated with the concatenated matrix per clade to calculate gene concordance factors (gCF: the fraction of decisive gene trees concordant for a branch) and the site concordance factors (sCF: the fraction of decisive alignment sites supporting a branch) [@minh2020iq]. We ranked loci according to sCF and selected sites with sCF $\geqslant 0.5$. From the resulting list of loci, we applied our filters to choose one SNP per locus as described above.

\texttt{SNAPP} requires specifying prior probabilities for the speciation rate ($\lambda$) and the expected divergence ($\Theta$). We used a $\Gamma$ distribution as hyperprior to accommodate uncertainty in $\lambda$. To set the parameters $\alpha$ and $\beta$ describing the $\Gamma$ distribution, we used \texttt{pylue} (https://github.com/joaks1/pyule) to estimate the expected height of the species trees. Based on these analyses, we used a diffused setting for $\alpha=2$ and adjusted $\beta$ such that the mean of the $\Gamma$ distribution centered around the expected height of the species trees. Similarly, we assigned a $\Gamma$ prior on $\Theta$ using $\alpha=2$ and adjusted $\beta$ such that the mean of the $\Gamma$ distribution centered around the average pairwise sequence divergence among all individuals within clades. 

_Model adequacy_ Because genogroups inferred with \texttt{mPTP} [@kapli2017multi] were consistently favored by Bayes Factor delimitation in all but one case, we performed a goodness-of-fit test to determine if the model used by \texttt{mPTP} was an adequate fit to the data. \texttt{mPTP} delimits species by using the empirical distribution of branch lengths (measured as substitutions per site) to identify transition points from a between species branching process to a within species branching process. Specifically, for each delimited species \texttt{mPTP} fits the branching events to a distinct exponential distribution. We therefore assessed the adequacy of \texttt{mPTP} by asking whether or not the branching events within each delimited species could plausibly have originated from the fitted exponential distribution. We used the Kolmogorov-Smirnov test to compare the empirical cumulative distribution function of branch lengths to the theoretical cumulative distribution function under the hypothesis that the data were generated from an exponential distribution. A large discrepancy between these two distributions (revealed by a small P-value) indicates that the observed data are unlikely to have been generated from the fitted exponential distribution and that \texttt{mPTP} is inadequately modeling the coalescent process within delimited species. Results from these tests are shown in (Table \@ref(tab:ModelAdequacyTable)).

## Data Integration

To quantify the association between phenogroups and genogroups within clades, we used the Goodman Kruskal's tau ($GK\tau$) statistic [@goddman1954measures]. This is an asymmetric association measure between two categorical variables with values ranging from zero (no association between variables) to one (perfect association between variables). The asymmetry of $GK\tau$ is particularly useful because in many instances the variability in variable $x$ that is explainable by variations in $y$ may be different from the variability in $y$ that is explainable by variations in $x$. Therefore, we estimated the association between phenogroups and genogroups ($GK\tau_{pg}$) and the association between genogroups and phenogroups ($GK\tau_{gp}$) within each clade using all specimens that had both types of data. When the value of either of these indices is equal to $1$, there is perfect association between one grouping category and the other grouping category, implying that knowledge of one group membership perfectly predicts membership of the other type of grouping. For instance, $GK\tau_{pg}=1$ means that knowledge of phenogroup membership perfectly predicts genogroup membership. Therefore, if a clade only contains 'good species', $GK\tau_{pg}$ and $GK\tau_{gp}$ are equal to $1$. By contrast, values of $GK\tau<1$ imply that knowledge of one grouping category provides lower predictive ability of the alternative grouping category. For instance, $GK\tau_{pg}<1$ means that knowledge of phenogroup membership provides lower ability to predict genogroup membership. Thus, a clade with $GK\tau_{pg}<1$ implies the presence of 'phenotypic cryptic species'. Similarly, a clade with $GK\tau_{gp}<1$ implies the presence of 'genetic cryptic species'. The $GK\tau$ statistic is calculated on specimens with both phenotypic and genetic data available, therefore it does not incorporate information from unknown specimens. To calculate $GK\tau$, we used the \texttt{R} package \texttt{GoodmanKruskal} v0.0.2 [@PearsonGKt2016].

# Results

## Current state of taxonomic species using genomics data

The lineage and species trees showed that the phylogeny of _Escallonia_ consisted of six well-supported clades (hereafter, Clades I-VI) (Figure \@ref(fig:EscalloniaSpeciesTreeTrimmedPlot)). When we included more than two specimens per taxonomic species, we recovered the same six clades, yet not all specimens determined to the same taxonomic species were each other's closest relatives (Figure \@ref(fig:EscalloniaSpeciesTreeVersion2Plot)).

## Clade I

### Sampling

A total of $`r CladeItotalSampling`$ specimens were included in this clade (Figure \@ref(fig:CladeIVennDiagram)). We measured phenotypic traits on $`r CladeItotalPhenome`$ specimens;  $`r CladeIphenomeFertile`$ were fertile specimens (thus having both leaf and floral traits) and $`r CladeIphenomeSterile`$ were sterile specimens (thus having leaf traits only). Only fertile specimens were included in downstream phenomic analyses. We collected genomic data for $`r CladeItotalGenome`$ specimens; $`r CladeIgenomeFertile`$ specimens were fertile, $`r CladeIgenomeSterile`$ were sterile, and $`r CladeIgenomeNoPhenome`$ were not available for phenotypic analyses. All $`r CladeItotalGenome`$ specimens were included in downstream genomic analyses except in our model selection analysis (See below). The $`r CladeItotalSampling`$ specimens covered the geographic range of this clade and belonged to two taxonomic species (Figure \@ref(fig:CladeIsamplingMaps), Table \@ref(tab:SamplingTable)).

### Phenomics

#### Current state of taxonomic species

The 10-cubes defining the taxonomic species that belong to this clade did not overlap in 10-dimensional phenospace (Figure \@ref(fig:currentStateOfTaxonomyCladeIPlots)). The matching-prediction analysis showed that $`r cladeIpercentassigned`$\% of the specimens fall inside any 10-cube, and no specimens fall inside their correct 10-cube (Figure \@ref(fig:currentStateOfTaxonomyCladeIPlots)).

#### Model-based species discovery

We found four principal components to be most useful for group discrimination. The naive model specifying two distinct phenogroups of equal shape and volume received the strongest support using both the BIC and ICL criteria (BIC$=54.03$, ICL$=53.86$). Support for alternative models was considerably lower, including the Taxonomy model [@Sleumer:1968ts] specifying two phenogroups ($\Delta$BIC=$8.225$) and the Taxonomy Unaware model specifying one phenogroup ($\Delta$BIC$=20.574$) (Figure \@ref(fig:cladeImorphologicalDelimitationPlots)). A LRT further corroborated that a model with two phenogroups of equal shape and volume was a better fit to the data than models with alternative phenogroup composition ($p$-value $<0.05$).

### Genomics

#### Sensitivity Tests

In total, we recovered $44,630$ loci across $`r CladeItotalGenome`$ sequenced individuals.  $22,999$ loci were present in at least four individuals. The number of loci in the three matrices with different levels of missing data is presented in Table \@ref(tab:CladeIgenomicDatasetAssemblies). Results from Principal Components Analysis, phylogenetic analysis, and genomic clustering did not differ (or differed minimally) across the three matrices, suggesting that our data were robust to the amount of missing data (Figures \@ref(fig:cladeIgenomicSensitivityAnalysesLargestplots), \@ref(fig:cladeIgenomicSensitivityAnalysesMiddleplots), \@ref(fig:cladeIgenomicSensitivityAnalysesSmallestplots)). Therefore, we performed all downstream analyses using the smallest data matrix for computational efficiency.


#### Model-based species discovery

**Genotypic clusters (GC model)** We found two dimensions to be most useful to faithfully represent the genotypic data in fewer dimensions (2 dimensions $=0.0079\%$ stress). In this reduced space, a model specifying three genotypic clusters received the highest support (BIC$=431.5805$, next best model $\Delta$BIC$=5.4468$, three clusters; Figure \@ref(fig:cladeImolecularDelimitationsNmms)).

**Transition between cladogenesis and anagenesis (CA model)** Given the phylogeny reconstructed with the concatenated alignment of full sequences per locus, a model with two genogroups was the best fit to the data (Figure \@ref(fig:cladeImolecularDelimitationsPhylogeny2)). Across all independent runs, all nodes identifying genogroups strongly identified such nodes as transitions between cladogenesis and anagenesis (Fig \@ref(fig:cladeImolecularDelimitationsPhylogeny2); See values closer to $0$ subtending red groups).

**Reproductive isolation (RI model)** A model specifying two demes received the strongest support based on the $\Delta$K statistic using \texttt{STRUCTURE}. An identical model specifying two demes using \texttt{MAVERICK} had the highest posterior probability ($pp>0.99$). Using these deme assignments, we found consistent and strong support for a model specifying two genogroups, for both the full and reduced datasets ($pp=1.0$, Figure \@ref(fig:cladeImolecularDelimitationPopGenBpp)).

**Model Selection** We filtered our matrix and retained $2,320$ loci to conduct this analysis (Table \@ref(tab:CladeIgenomicDatasetAssemblies)). Of the four models described above, three were identical. The CA model and the two RI models (using deme assignments based on \texttt{STRUCTURE} and \texttt{MAVERICK}) each identified and assigned the same specimens to the same two genogroups. We compared this model with the GC model using Bayes Factors. Each model required more than $500,000$ samples to stabilize (as estimated from ESS values for each step). In total, each analysis used $24$ steps. The GC model (three genogroups) was the top-ranked model with decisive support relative to the other model [@kass1995bayes] (Table \@ref(tab:cladeIbfdTable)).

### Data Integration

Based on the best fitting models for species discovery using phenotypic and genome-wide variation, we assigned each specimen to its corresponding phenogroup and genogroup (Figure \@ref(fig:cladeIfinalDelimitation2)). In total, we assigned specimens to two species (See main text, Figure 2). These two species matched uniquely one phenogroup to one genogroup (Figure \@ref(fig:cladeIfinalDelimitation2); See for example phenogroup 1 - genogroup 3). Therefore, these two species are recognized as 'good species'. We also discovered one genogroup with specimens for which we did not have phenotypic data available (unknown specimens) (Figure \@ref(fig:cladeIfinalDelimitation2), genogroup 2; See below). The measure of association of phenogroups to genogroups was $GK\tau_{pg}=1.0$ and the association of genogroups to phenogroups was $GK\tau_{gp}=1.0$. This indicated that genogroup membership is a perfect predictor of phenogroup membership (and _vice versa_), and is reflected in the fact that this clade only includes 'good species'.

We examined all the specimens in their phylogenetic and geographic contexts to gain insight into the plausible species identity of the unknown specimens and the nature of species in this clade. Given that specimens in both phenogroups 1 and 2 do not co-occur locally, we speculate that the unknown specimens assigned to phenogroup 1 most likely belong to genogroup 3, and that the unknown specimens assigned to phenogroup 2 most likely belong to either genogroup 1 or 2 (Figure \@ref(fig:cladeIfinalDelimitation2)). Given this plausible assignment and that genogroups 1 and 2 are sister lineages, our reasoning implies that species in phenogroup 2 and genogroups 1 and 2 are 'phenotypic cryptic species' (Figure \@ref(fig:cladeIfinalDelimitation2)). However, it is also plausible that genogroups 1 and 2 correspond to isolated populations of a single species, which is genetically structured along elevation. These uncertainties aside, our results showed that at the spatial scale of our study, the 'good species' do not co-occur in close geographic proximity with one another suggesting complete differentiation in allopatry (Figure \@ref(fig:cladeIfinalDelimitation2); See phenogroup 1 - genogroup 3 and phenogroup 2 - genogroup 3).

### Correspondence between taxonomic species and model-based species

We assigned all specimens to their corresponding taxonomic species, phenogroup, and genogroup. Only specimens assigned to one taxonomic species ( _E. micrantha_ ) showed perfect correspondence to a single phenogroup and a single genogroup. All other specimens in this clade were assigned to the taxonomic species _E. millegrana_, a single phenogroup (phenogroup 2), and multiple genogroups (See main text, Table 4; Table \@ref(tab:cladeIfinalSummary)).


## Clade II

### Sampling

A total of $`r CladeIItotalSampling`$ specimens were included in this clade (Figure \@ref(fig:CladeIIVennDiagram)). We measured phenotypic traits on $`r CladeIItotalPhenome`$ specimens;  $`r CladeIIphenomeFertile`$ were fertile specimens (thus having both leaf and floral traits) and $`r CladeIIphenomeSterile`$ were sterile specimens (thus having leaf traits only). Only fertile specimens were included in downstream phenomic analyses. We collected genomic data for $`r CladeIItotalGenome`$ specimens; $`r CladeIIgenomeFertile`$ specimens were fertile, $`r CladeIIgenomeSterile`$ were sterile, and $`r CladeIIgenomeNoPhenome`$ were not avaialable for phenotypic analyses. All $`r CladeIItotalGenome`$ specimens were included in downstream genomic analyses except in our model selection analysis (See below). The $`r CladeIItotalSampling`$ specimens covered the geographic range of this clade and belonged to two taxonomic species (Figure \@ref(fig:CladeIIsamplingMaps), Table \@ref(tab:SamplingTable)).

### Phenomics

#### Current state of taxonomic species

The 10-cubes defining the taxonomic species that belong to this clade did not overlap in 10-dimensional phenospace (Figure \@ref(fig:currentStateOfTaxonomyCladeIIPlots)). The matching-prediction analysis showed that $`r cladeIIpercentassigned`$\% of the specimens fall inside any 10-cube, with no specimens falling inside their correct 10-cube (Figure \@ref(fig:currentStateOfTaxonomyCladeIIPlots)).

#### Model-based species discovery

We found four principal components to be most useful for group discrimination. The naive model specifying three distinct phenogroups, of equal shape, volume, and orientation received the strongest support using both the BIC and ICL criteria (BIC$=71.729$, ICL$=69.411$). Support for alternative models was considerably lower, including the Taxonomy model [@Sleumer:1968ts] specifying two phenogroups ($\Delta$BIC$=53.956$) and the Taxonomy Unaware model specifying one phenogroup ($\Delta$BIC$=24.201$) (Figure \@ref(fig:cladeIImorphologicalDelimitationPlots)). A LRT further corroborated that a model with three phenogroups of equal shape, volume, and orientation was a better fit to the data than models with more phenogroups ($p$-value $<0.05$)

### Genomics

#### Sensitivity Tests

In total, we recovered $66,064$ loci across $`r CladeIItotalGenome`$ sequenced individuals. $30,440$ loci were present in at least four individuals. The number of loci in the three matrices with different levels of missing data is presented in Table \@ref(tab:CladeIIgenomicDatasetAssemblies). Results from Principal Components Analysis, phylogenetic analysis, and genomic clustering did not differ (or differed minimally) across the three matrices, suggesting that our data were robust to the amount of missing data (Figures \@ref(fig:cladeIIgenomicSensitivityAnalysesLargestplots), \@ref(fig:cladeIIgenomicSensitivityAnalysesMiddleplots), \@ref(fig:cladeIIgenomicSensitivityAnalysesSmallestplots)). Therefore, we performed all downstream analyses using the smallest data matrix for computational efficiency.

#### Model-based species discovery

**Genotypic clusters (GC model)** We found two dimensions to be most useful to faithfully represent the genotypic data in fewer dimensions (2 dimensions $=0.0082\%$ stress). In this reduced space, a model specifying three genotypic clusters received the highest support (BIC$=347.3602$, next best model $\Delta$BIC$=41.7965$, three clusters; Figure \@ref(fig:cladeIImolecularDelimitationsNmms)).

**Transition between cladogenesis and anagenesis (CA model)** Given the phylogeny reconstructed with the concatenated alignment of full sequences per locus, a model with four genogroups was the best fit to the data (Figure \@ref(fig:cladeIImolecularDelimitationsPhylogeny2)). Across all independent runs, all nodes identifying genogroups strongly identified such nodes as transitions between cladogenesis and anagenesis (Fig \@ref(fig:cladeIImolecularDelimitationsPhylogeny2); See values closer to $0$ subtending red groups).

**Reproductive isolation (RI model)** A model specifying two demes received the strongest support based on the $\Delta$K statistic using \texttt{STRUCTURE}. Using this deme assignment, we found consistent and strong support for a model specifying two genogroups, for both the full and reduced datasets ($pp=1.0$, Figure \@ref(fig:cladeIImolecularDelimitationPopGenBpp)). A model specifying three demes using \texttt{MAVERICK} had the highest posterior probability ($pp>0.99$). Using this deme assignment, we found consistent and strong support for a model specifying three genogroups, for both the full and reduced datasets ($pp=1.0$, Figure \@ref(fig:cladeIImolecularDelimitationPopGenBpp)). All replicated analyses used to identify genogroups with both deme assignments showed consistent support for two and three genogroups, respectively.

**Model Selection** We filtered our matrix and retained $3,005$ loci to conduct this analysis (Table \@ref(tab:CladeIIgenomicDatasetAssemblies)). Of the four models described above, the CA model identified four genogroups, the GC model and the RI models using deme assignments based on \texttt{MAVERICK}) both identified and assigned the same specimens to the same three genogroups, and the RI model using \texttt{STRUCTURE} for deme assignment recognized two genogroups (Table \@ref(tab:cladeIIbfdTable)). We compared all models using Bayes Factors and found that each model required more than $500,000$ samples to stabilize (as estimated from ESS values for each step), with the exception of the RI model based on \texttt{STRUCTURE} assignments which required $250,000$ samples to stabilize. In total, each analysis used $24$ steps. The CA model (four genogroups) was the top-ranked model with decisive support relative to the other models [@kass1995bayes] (Table \@ref(tab:cladeIIbfdTable)).

### Data Integration

Based on the best fitting models, we assigned specimens with overlapping phenotypic and genomic data to five species (See main text, Figure 2). None of these assignments matched uniquely one phenogroup to one genogroup, indicating there are no 'good species' in this clade (Figure \@ref(fig:cladeIIfinalDelimitation2)). The specimens assigned to phenogroup 1 were asssigned to genogroups 1 and 3, and the specimens assigned to phenogroup 3 were assigned to genogroups 1, 2, and 4. Therefore, we recognized these sets as 'phenotypic cryptic species'. Other specimens assigned to genogroup 1 were assigned to phenogroups 1 and 3, which indicates there are also 'genetic cryptic species' in this clade (Figure \@ref(fig:cladeIIfinalDelimitation2)). We did not have genomic data available for the specimens that we assigned to phenogroup 2 (unknown specimens). The measure of association of phenogroups to genogroups was $GK\tau_{pg}=0.322$, while the association of genogroups to phenogroups was $GK\tau_{gp}=0.762$. This indicated that genogroup membership is a better predictor of phenogroup membership than the phenogroup membership is of genogroup membership. In other words, this shows there are more 'phenotypic cryptic species' in this clade than 'genotypic cryptic species'.

We examined all the specimens in their phylogenetic and geographic contexts to gain insight into the plausible species identity of the unknown specimens and the nature of species in this clade. Considering the geographic distribution of the unknown specimens assigned to genogroup 2 and genogroup 3 as well as our extensive field and herbarium work, we speculate that these specimens likely belong to phenogroups 3 and 1, respectively. However, it is difficult to speculate the likely assignment of the unknown specimens in genogroup 1 given that this genogroup includes specimens from phenogroups 1 and 3, which do not show a clear pattern of geographic structure. Indeed, given the overall lack of geographic structure of phenogroups 1 and 3, it is not possible to suggest the genogroup assignment of all the unknwon specimens in these phenogroups (See also estimates of $GK\tau_{pg}$ above). These uncertainties aside, our results showed that at the spatial scale of our study all species co-occur in close geographic proximity with at least one other species (Figure \@ref(fig:cladeIIfinalDelimitation2)). Our phylogenetic and geographic results also showed that the 'phenotypic cryptic species' are allopatric and not sister to each other suggesting convergent evolution in phenotype perhaps driven by niche conservatism [@wiens2004speciation] (See for example phenogroup 3; all specimens are restricted to dry inter-andean valleys). However, extensive sampling is needed to discern the nature of species in this clade with increased rigor.

### Correspondence between taxonomic species and model-based species

We assigned all specimens to their corresponding taxonomic species, phenogroup, and genogroup. The specimens assigned to either taxonomic species did not show perfect correspondence to a single phenogroup and a single genogroup. Instead, all specimens were assigned to phenogroups and genogroups which were shared across both taxonomic species (See main text, Table 4; Table \@ref(tab:cladeIIfinalSummary)).

## Clade III

### Sampling

A total of $`r CladeIIItotalSampling`$ specimens were included in this clade (Figure \@ref(fig:CladeIIIVennDiagram)). We measured phenotypic traits on $`r CladeIIItotalPhenome`$ specimens;  $`r CladeIIIphenomeFertile`$ were fertile specimens (thus having both leaf and floral traits) and $`r CladeIIIphenomeSterile`$ were sterile specimens (thus having leaf traits only). Only fertile specimens were included in downstream phenomic analyses. We collected genomic data for $`r CladeIIItotalGenome`$ specimens; $`r CladeIIIgenomeFertile`$ specimens were fertile, $`r CladeIIIgenomeSterile`$ were sterile, and $`r CladeIIIgenomeNoPhenome`$ were not avaialable for phenotypic analyses. All $`r CladeIIItotalGenome`$ specimens were included in downstream genomic analyses except in our model selection analysis (See below). The $`r CladeIIItotalSampling`$ specimens covered the geographic range of this clade and belonged to six taxonomic species (Figure \@ref(fig:CladeIIIsamplingMaps), Table \@ref(tab:SamplingTable))

### Phenomics

#### Current state of taxonomic species

The 10-cubes defining the taxonomic species that belong to this clade did not overlap in 10-dimensional phenospace. The matching-prediction analysis showed that only $`r cladeIIIpercentassigned`$\% of the specimens fall inside any 10-cube, with one specimen falling inside its correct 10-cubes (Figure \@ref(fig:currentStateOfTaxonomyCladeIIIPlots)).

#### Model-based species discovery

We found eight principal components to be most useful for group discrimination. The naive model specifying five distinct phenogroups of equal shape and orientation received the strongest support using both the BIC and ICL criteria (BIC$=387.153$, ICL$=382.608$). Support for alternative models was considerably lower, including the Taxonomy model [@Sleumer:1968ts] specifying six phenogroups ($\Delta$BIC$=333.767$) and the Taxonomy Unaware model specifying four phenogroups ($\Delta$BIC$=216.314$) (Figure \@ref(fig:cladeIIImorphologicalDelimitationPlots)). A LRT further corroborated that a model with five phenogroups of equal shape and orientation was a better fit to the data than models with more phenogroups ($p$-value $< 0.05$)

### Genomics

#### Sensitivity Tests

In total, we recovered $91,032$ loci across $`r CladeIIItotalGenome`$ sequenced individuals. $43,597$ loci were present in at least four individuals. The number of loci in the three matrices with different levels of missing data is presented in Table \@ref(tab:CladeIIIgenomicDatasetAssemblies). Results from Principal Components Analysis, phylogenetic analysis, and genomic clustering did not differ (or differed minimally) across datasets, suggesting that our data are robust to the amount of missing data (Figures \@ref(fig:cladeIIIgenomicSensitivityAnalysesLargestplots), \@ref(fig:cladeIIIgenomicSensitivityAnalysesMiddleplots), \@ref(fig:cladeIIIgenomicSensitivityAnalysesSmallestplots)). Therefore, we performed all downstream analyses using the smallest data matrix for computational efficiency.

#### Model-based species discovery

**Genotypic clusters (GC model)** We found two dimensions to be most useful to faithfully represent the genotypic data in fewer dimensions (2 dimensions $=0.009\%$ stress). In this reduced space, a model specifying three genotypic clusters received the highest support (BIC$=1668.27$, next best model $\Delta$BIC$=5.2501$, four clusters; Figure \@ref(fig:cladeIIImolecularDelimitationsNmms)).

**Transition between cladogenesis and anagenesis (CA model)** Given the phylogeny reconstructed with the concatenated alignment of full sequences per locus, a model with seven genogroups was the best fit to the data (Figure \@ref(fig:cladeIIImolecularDelimitationsPhylogeny2)). Across all independent runs, all nodes identifying genogroups strongly identified such nodes as transitions between cladogenesis and anagenesis (Fig \@ref(fig:cladeIIImolecularDelimitationsPhylogeny2); See values closer to $0$ subtending red groups).

**Reproductive isolation (RI model)** A model specifying three demes received the strongest support based on the $\Delta$K statistic using \texttt{STRUCTURE}. Using this deme assignment, we found consistent and strong support for a model specifying three genogroups ($pp=1.0$, Figure \@ref(fig:cladeIIImolecularDelimitationPopGenBpp)), for both the full and reduced datasets (See Methods for details). A model specifying five demes using \texttt{MAVERICK} had the highest posterior probability ($pp>0.99$). Using this deme assignment, we found consistent and strong support for a model specifying five genogroups ($pp=1.0$, Figure \@ref(fig:cladeIIImolecularDelimitationPopGenBpp)), for both the full and reduced datasets. Although all our analyses used to identify genogroups with both deme assignments showed some mixing issues, every replicated analysis consistently supported three and five genogroups, respectively.

**Model Selection** We filtered our matrix and retained $1,993$ loci to conduct this analysis (Table \@ref(tab:CladeIIIgenomicDatasetAssemblies)). We compared the four models described above using Bayes Factors. The GC model and both RI models using population assignments based on \texttt{STRUCTURE} and \texttt{MAVERICK} each identified and assigned the same individuals to the same three genogroups. The GC model and the RI model using \texttt{STRUCTURE} required $250,000$ samples to stabilize (as estimated from ESS values for each step), while the CA model and the RI model using population assignments based on \texttt{MAVERICK} required more than $500,000$ samples. In total, each analysis used $24$ steps. The CA model (seven genogroups) was the top-ranked model with decisive support relative to the two other models [@kass1995bayes] (Table \@ref(tab:cladeIIIbfdTable)).

### Data Integration

Based on the best fitting models, we assigned each specimen to its corresponding phenogroup and genogroup (Figure \@ref(fig:cladeIIIfinalDelimitation2)). In total, we assigned specimens to eight species (See main text, Figure 2). Two of these species matched uniquely one phenogroup to one genogroup (Figure \@ref(fig:cladeIIIfinalDelimitation2)); See for example phenogroup 4 - genogroup 3). Therefore, these two species are recognized as 'good species'. Five species belonged to two phenogroups across five genogroups (Figure \@ref(fig:cladeIIIfinalDelimitation2)); See for example phenogroup 2 - genogroups 1, 2). These five species are thus recognized as 'phenotypic cryptic species'. Two species belonged to one genogroup across two phenogroups (Figure \@ref(fig:cladeIIIfinalDelimitation2)); See for example phenogroup 1, 3 - genogroup 5). These two species are recongnized as 'genetic cryptic species'. The association of phenogroups to genogroups was $GK\tau_{pg}=0.731$, while the association of genogroups to phenogroups was $GK\tau_{gp}=0.913$. This indicates that genogroup membership is slightly better predictor of phenogroup membership than phenogroup membership is of genogroup membership (i.e., there are more 'phenotypic cryptic species' in this clade than 'genetic cryptic species').

We examined all the specimens in their phylogenetic and geographic contexts to gain insight into the plausible species identity of the unknown specimens and the nature of species in this clade. Considering the geographic distribution of the unknown specimens assigned to phenogroup 4, in concert with our extensive field and herbarium work, we speculate that these specimens most likely belong to genogroup 3. Similarly, we speculate that the unknown specimens assigned to genogroup 3 most likely belong to phenogroup 4. Applying the same reasoning, we speculate that the unknown specimens in phenogroups 1 and 5 most likely belong to genogroups 5 and 4, respectively. Likewise, we suggest that the unknown specimens in genogroups 4 and 5 most likely belong to phenogroups 5 and 1, respectively. For the unknown specimens in genogroups 1, 2, and 6, we speculate that these specimens most likely belong to phenogroups 2, and 3. However, we cannot speculate to which genogroups the unknown specimens in phenogroups 2 and 3 belong (See also $GK\tau_{pg}$ above). These uncertainties aside, together our results showed that at the spatial scale of our study all species co-occur in close geographic proximity with at least one other species (Figure \@ref(fig:cladeIIIfinalDelimitation2)). Our phylogenetic and geographic results also showed that while the 'good species' are allopatric, they co-occur with other species which are not their closest relatives. By contrast, the 'phenotypic cryptic species' and 'genetic cryptic species' largely co-occur with their closest relatives.

### Correspondence between taxonomic species and model-based species

We assigned all specimens to their corresponding taxonomic species, phenogroup, and genogroup. Only specimens assigned to one taxonomic species ( _E. schreiteri_ ) showed perfect correspondence to a single phenogroup and a single genogroup. All other specimens are assigned to phenogroups and genogroups shared across multiple taxonomic species (See main text, Table 4; Table \@ref(tab:cladeIIIfinalSummary)).

## Clade IV

### Sampling

A total of $`r CladeIVtotalSampling`$ specimens were included in this clade (Figure \@ref(fig:CladeIVVennDiagram)). We measured phenotypic traits on $`r CladeIVtotalPhenome`$ specimens;  $`r CladeIVphenomeFertile`$ were fertile specimens (thus having both leaf and floral traits) and $`r CladeIVphenomeSterile`$ were sterile specimens (thus having leaf traits only). Only fertile specimens were included in downstream phenomic analyses. We collected genomic data for $`r CladeIVtotalGenome`$ specimens; $`r CladeIVgenomeFertile`$ specimens were fertile, $`r CladeIVgenomeSterile`$ were sterile, and $`r CladeIVgenomeNoPhenome`$ were not avaialable for phenotypic analyses. All $`r CladeIVtotalGenome`$ specimens were included in downstream genomic analyses except in our model selection analysis (See below). The $`r CladeIVtotalSampling`$ specimens covered the geographic range of this clade and belonged to two taxonomic species (Figure \@ref(fig:CladeIVsamplingMaps), Table \@ref(tab:SamplingTable))

### Phenomics

#### Current state of taxonomic species

The 10-cubes defining the taxonomic species that belong to this clade did not overlap in 10-dimensional phenospace. The matching-prediction analysis showed that $`r cladeIVpercentassigned`$\% of the specimens fall inside any 10-cube, with no specimens falling inside their correct 10-cube (Figure \@ref(fig:currentStateOfTaxonomyCladeIVPlots)).

#### Model-based species discovery

We found five principal components to be most useful for group discrimination. The Taxonomy [@Sleumer:1968ts] and Taxonomy Unaware models, both specifying the same two phenogroups, received the strongest support using the BIC criterion (BIC$=115.0039$). We could not use the ICL criterion because the best fit model was not the naive model and \texttt{mclust} does not implement the ICL criterion for models with _a priori_ classification. Support for the naive model specifying three distinct phenogroups of equal volume, elliptical, and identical orientation was only slightly lower ($\Delta$BIC$=0.8951$). Because the difference in BIC scores between the best fit and competing models is not significant [@kass1995bayes], we present results for both models (Figure \@ref(fig:cladeIVmorphologicalDelimitationPlots)). As expected, the LRT is consistent with the BIC model selection framework and showed that a model with three distinct phenogroups was a better fit to the data than models with more (or less) phenogroups ($p$=value $<0.05$).

### Genomics

#### Sensitivity Tests

In total, we recovered $79,865$ loci across $`r CladeIVtotalGenome`$ sequenced individuals. $31,840$ loci were present in at least four individuals. The number of loci in the three matrices with different levels of missing data is presented in Table \@ref(tab:CladeIVgenomicDatasetAssemblies). Results from Principal Components Analysis, phylogenetic analysis, and genomic clustering did not differ (or differed minimally) across datasets, suggesting that our data are robust to the amount of missing data (Figures \@ref(fig:cladeIVgenomicSensitivityAnalysesLargestplots), \@ref(fig:cladeIVgenomicSensitivityAnalysesMiddleplots), \@ref(fig:cladeIVgenomicSensitivityAnalysesSmallestplots)). Therefore, we performed all downstream analyses using the smallest data matrix for computational efficiency.

#### Model-based species discovery

**Genotypic clusters (GC model)** We found two dimensions to be most useful to faithfully represent the genotypic data in fewer dimensions (2 dimensions $= 1.2967\%$ stress). In this reduced space, a model specifying three genotypic clusters received the highest support (BIC$=973.8316$, next best model $\Delta$BIC$=3.6340$, three clusters; Figure \@ref(fig:cladeIVmolecularDelimitationsNmms)).

**Transition between cladogenesis and anagenesis (CA model)** Given the phylogeny reconstructed with the concatenated alignment of full sequences per locus, a model with six genogroups was the best fit to the data (Figure \@ref(fig:cladeIVmolecularDelimitationsPhylogeny2)). Across all independent runs, all nodes identifying genogroups strongly identified such nodes as transitions between cladogenesis and anagenesis (Fig \@ref(fig:cladeIVmolecularDelimitationsPhylogeny2); See values closer to $0$ subtending red groups).

**Reproductive isolation (RI model)** A model specifying two demes received the strongest support based on the $\Delta$K statistic using \texttt{STRUCTURE}. A model specifying three demes using \texttt{MAVERICK} had the highest posterior probability ($pp>0.99$); however, one of these demes never received a majority posterior assignment for any specimen. Because we used a majority posterior assignment to assign specimens to demes, we thus used a model with two demes, which was identical to the model specified using \texttt{STRUCTURE}. Therefore, the RI model used the same deme assignment (two demes) with both \texttt{STRUCTURE} and \texttt{MAVERICK}. Using this deme assignment, we found consistent and strong support for a model specifying two genogroups ($pp=1.0$, Figure \@ref(fig:cladeIVmolecularDelimitationPopGenBpp)), for both the full and reduced datasets (See Methods for details).

**Model Selection** We filtered our matrix and retained $2,245$ loci to conduct this analysis (Table \@ref(tab:CladeIVgenomicDatasetAssemblies)). We compared the three models specified above using Bayes Factors. Each model required approximately $250,000$ samples to stabilize (as estimated from ESS values for each step) with the exception of the CA model which required more than $500,000$ samples to stabilize. In total, each analysis used $24$ steps. The CA model (six genogroups) was the top-ranked model with decisive support relative to the other models [@kass1995bayes] (Table \@ref(tab:cladeIVbfdTable)).

### Data Integration

Based on the best fitting models, we assigned each specimen to its corresponding phenogroup and genogroup (Figure \@ref(fig:cladeIVfinalDelimitation2)). In total, we assigned specimens to six species (See main text, Figure 2). One species matched uniquely a single phenogroup and a single genogroup (Figure \@ref(fig:cladeIVfinalDelimitation2); phenogroup 1 - genogroup 3). This species is thus recognized as a 'good species'. The remaining specimens were assigned to a single phenogroup and five genogroups (Figure \@ref(fig:cladeIVfinalDelimitation2); phenogroup 2 - genogroups 1, 2, 4, 5, and 6). Therefore, these species are collectively recongnized as 'phenotypic cryptic species'. The association of phenogroups to genogroups was $GK\tau_{pg}= 0.214$, while the association of genogroups to phenogroups was $GK\tau_{gp}= 1.0$. This indicates that genogroup membership is a perfect predictor of phenogroup membership (i.e., there are no 'genetic cryptic species') but phenogroup membership is a poor predictor of genogroup membership (i.e., there are many 'phenotypic cryptic species' in this clade).

We examined all the specimens in their phylogenetic and geographic contexts to gain insight into the plausible species identity of the unknown specimens and the nature of species in this clade. Based on the geographic distribution of the unknown specimens assigned to genogroups 1, 2, 4, and 6, combined with our extensive field and herbarium work, we infer that all these specimens belong to phenogroup 2 (See also $GK\tau_{gp}$ above). Likewise, we suggest that the unknown specimens assigned to phenogroup 1 most likely belong to genogroup 3. However, it is not possible to infer the plausible genogroup assignment for all the unknown specimens in phenogroup 2 (See also $GK\tau_{pg}$ above). This is particularly challenging south of latitude $-10^\circ$ where genogroups 2, 4, 5, and 6 co-occurr and do not display any clear pattern of geographic structure. These uncertainties aside, our results showed that at the spatial scale of our study all species largely co-occur in close geographic proximity with other species. The closely related species in genogroups 1 and 3 are restricted to latitudes north of $-8^\circ$, whereas their closely related species in genogroup 2 co-occurs with the group of related species in genogroups 4, 5, and 6 south of $-10^\circ$. Notably, the closely related 'phenotypic cryptic species' in genogroups 4, 5, and 6 all co-occur locally, which indicates that these species are isolated in sympatry despite sharing phenotypic similarities (Figure \@ref(fig:cladeIVfinalDelimitation2)).

### Correspondence between taxonomic species and model-based species

We assigned all specimens to their corresponding taxonomic species, phenogroup, and genogroup. Only specimens assigned to one taxonomic species ( _E. polifolia_ ) showed perfect correspondence to a single phenogroup and a single genogroup. The other specimens, all of which were assigned to the taxonomic species  _E. myrtilloides_, shared a single phenogroup across multiple genogroups (See main text, Table 4; Table \@ref(tab:cladeIVfinalSummary)).

## Clade V

### Sampling

A total of $`r CladeVtotalSampling`$ specimens were included in this clade (Figure \@ref(fig:CladeVVennDiagram)). We measured phenotypic traits on $`r CladeVtotalPhenome`$ specimens; $`r CladeVphenomeFertile`$ were fertile specimens (thus having both leaf and floral traits) and $`r CladeVphenomeSterile`$ were sterile specimens (thus having leaf traits only). Only fertile specimens were included in downstream phenomic analyses. We collected genomic data for $`r CladeVtotalGenome`$ specimens; $`r CladeVgenomeFertile`$ specimens were fertile, $`r CladeVgenomeSterile`$ were sterile, and $`r CladeVgenomeNoPhenome`$ were not available for phenotypic analyses. All $`r CladeVtotalGenome`$ specimens were included in downstream genomic analyses except in our model selection analysis (See below). The $`r CladeVtotalSampling`$ specimens covered the geographic range of this clade and belonged to seven taxonomic species (Figure \@ref(fig:CladeVsamplingMaps), Table \@ref(tab:SamplingTable))

### Phenomics

#### Current state of taxonomic species

The 10-cubes based defining the taxonomic species that belong to this clade did not overlap in 10-dimensional phenospace. The matching-prediction analysis showed that $`r cladeVpercentassigned`$\% of the specimens fall inside any 10-cube, with no specimens falling inside their correct 10-cube (Figure \@ref(fig:currentStateOfTaxonomyCladeVPlots)).

#### Model-based species discovery

We found five principal components to be most useful for group discrimination. The naive model specifying eight distinct ellipsoidal phenogroups of equal volume, shape, and orientation received the strongest support using both the BIC and ICL criteria (BIC$=516.723$, ICL$=552.033$). Support for alternative models was considerably lower, including the Taxonomy model [@Sleumer:1968ts] specifying seven phenogroups ($\Delta$BIC$=274.73$) and the Taxonomy Unaware model specifying four phenogroups ($\Delta$BIC$=131.315$) (Figure \@ref(fig:cladeVmorphologicalDelimitationPlots)). A LRT further corroborated that a model with eight phenogroups of equal shape and orientation was a better fit to the data than models with more phenogroups ($p$-value $<0.05$)

### Genomics

#### Sensitivity Tests

In total, we recovered $133,181$ loci across $`r CladeVtotalGenome`$ sequenced individuals. $50,898$ loci were present in at least four individuals. The number of loci in the three matrices with different levels of missing data is presented in Table \@ref(tab:CladeVgenomicDatasetAssemblies)). Results from Principal Components Analysis, phylogenetic analysis, and genomic clustering did not differ (or differed minimally) across datasets, suggesting that our data are robust to the amount of missing data (Figures \@ref(fig:cladeVgenomicSensitivityAnalysesLargestplots), \@ref(fig:cladeVgenomicSensitivityAnalysesMiddleplots), \@ref(fig:cladeVgenomicSensitivityAnalysesSmallestplots)). Therefore, we performed all downstream analyses using the smallest data matrix for computational efficiency.

#### Model-based species discovery

**Genotypic clusters (GC model)** We found two dimensions to be most useful to faithfully represent the genotypic data in fewer dimensions (2 dimensions = $15.6\%$ stress). In this reduced space, a model specifying six genotypic clusters received the highest support (BIC$=583.3511$, next best model $\Delta$BIC$=1.7043$, six clusters; Figure \@ref(fig:cladeVmolecularDelimitationsNmms)).

**Transition between cladogenesis and anagenesis (CA model)** Given the phylogeny reconstructed with the concatenated alignment of full sequences per locus, a model with ten genogroups was the best fit to the data (Figure \@ref(fig:cladeVmolecularDelimitationsPhylogeny2)). Across all independent runs, all nodes identifying genogroups strongly identified such nodes as transitions between cladogenesis and anagenesis (Fig \@ref(fig:cladeVmolecularDelimitationsPhylogeny2); See values closer to $0$ subtending red groups).

**Reproductive isolation (RI model)** A model specifying two demes received the strongest support based on the $\Delta$K statistic using \texttt{STRUCTURE} (Figure \@ref(fig:cladeVmolecularDelimitationPopGenBpp)). Using this deme assignment, we found consistent support for a model specifying two genogroups, for both the full and reduced datasets despite some mixing issues ($pp=1.0$, Figure \@ref(fig:cladeVmolecularDelimitationPopGenBpp)). A model specifying four demes using \texttt{MAVERICK} had the highest posterior probability ($pp>0.99$); however, one of these demes never received a majority posterior assignment for any specimen. Because we used a majority posterior assignment to assign specimens to demes, we used a model with three demes. Using this deme assignment, we found consistent support for a model specifying three genogroups ($pp=1.0$, Figure \@ref(fig:cladeVmolecularDelimitationPopGenBpp)), for both the full and reduced datasets.

**Model Selection** We filtered our matrix and retained $742$ loci to conduct this analysis (Table \@ref(tab:CladeVgenomicDatasetAssemblies)). We compared the four models specified above using Bayes Factors. The RI models required $250,000$ samples to stabilize (as estimated from ESS values for each step), while the GC and CA models required $500,000$. In total, each analysis used $24$ steps. The CA model (ten genogroups) was the top-ranked model with decisive support relative to the other models [@kass1995bayes] (Table \@ref(tab:cladeVbfdTable)).

### Data Integration

Based on the best fitting models, we assigned each specimen to its corresponding phenogroup and genogroup (Figure \@ref(fig:cladeVfinalDelimitation2)). In total, we assigned specimens to 20 species (See main text, Figure 2). No species matched uniquely one phenogroup to one genogroup, showing there are no 'good species' in this clade (Figure \@ref(fig:cladeVfinalDelimitation2)). Specimens assigned to six of the eight phenogroups were assigned to multiple genogroups (Figure \@ref(fig:cladeVfinalDelimitation2); See for example phenogroup 2 and genogroups 4, 6, 9, and 10). This indicates there are multiple 'phenotypic cryptic species' in this clade. Similarly, specimens assigned to six of the ten genogroups were assigned to different phenogroups (Figure \@ref(fig:cladeVfinalDelimitation2); See for example genogroup 9 and phenogroups 2, 3, 4, and 7). This in turn indicates that several of the 'phenotypic cryptic species' in this clade are also 'genetic cryptic species'. The measure of association between phenogroups and genogroups reflects this result. The association between phenogroups and genogroups was $GK\tau_{pg}=0.488$, while the association between genogroups and phenogroups was $GK\tau_{gp}=0.467$. This shows that genogroup membership is a poor predictor of phenogroup membership and that phenogroup membership is also poor predictor of genogroup membership. This is a consequence of the large number of 'phenotypic cryptic' and 'genetic cryptic' species in this clade.

We examined all the specimens in their phylogenetic and geographic contexts to gain insight into the plausible species identity of the unknown specimens and the nature of species in this clade. Because all phenogroups and genogroups showed extensive overlap and limited geographic strucuture, it is difficult to infer the likely assignment of the unknown specimens based on geographic information alone (Figure \@ref(fig:cladeVfinalDelimitation2)). In addition, the predictability of each grouping category to infer the other category is extremely low (See estimates of $GK\tau$ above). Nonetheless, based on our extensive field and herbarium work, we speculate that the unknown specimens in genogroup 1 most likely belong to phenogroups 4 and 8, and the unknown specimens in genogroup 2 most likely belong to phenogroup 3. We also speculate that the unknown specimens in genogroup 5 likely belong to phenogroup 6. There were no other clear cases of plausible phenogroup assignment based on genogroup information. Inferring genogroup based on phenogroup membership was particularly difficult, with very few exceptions. For instance, we could only speculate that the unknown specimens in phenogroup 1 likely belong to genogroup 3 (Figure \@ref(fig:cladeVfinalDelimitation2)). Our phylogenetic and geographic results also showed that three genogroups are widespread across latitude and elevation (genogroups 1, 9, and 10) while the remaining genogroups are seemingly less widespread (Figure \@ref(fig:cladeVfinalDelimitation2)). Notably, each widespread genogroup is part of different 'genetic cryptic species' (i.e., one genogroup across multiple phenogroups). Because the specimens assigned to the different phenogroups within each of the widespread genogroups are closely related and co-occur locally in sympatry, our results are consistent with the intriguing possibility that these 'genetic cryptic species' correspond to phenotypically distinct species interconnected by gene exchange--so-called syngameons [@lotsy1925species; @cronk2018role] (Figure \@ref(fig:cladeVfinalDelimitation2); See for example genogroup 9). Further sampling and proper population genetic models are required to test this hypothesis with increased rigor. Our results also showed that genogroups 5, 6, and 8 are a group of closely related 'phenotypic cryptic species' which co-occur in broad sympatry. Specimens assigned to phenogroups 2 and 3 and to genogroup 7 are nested within or sister to the genogroups 5, 6, and 8. Because phenogroups 2 and 3 are common in the widespread genogroup 10, we speculate the specimens belonging to these phenogroups which are nested or sister to the group of 'phenotypic cryptic species' likely represent cases of local hybridization between different genogroups (Figure \@ref(fig:cladeVfinalDelimitation2)). Clearly, extensive sampling is necessary in the region of sympatry to discern the nature of these plant species.

### Correspondence between taxonomic species and model-based species

We assigned all specimens to their corresponding taxonomic species, phenogroup, and genogroup. There were no cases of perfect correspondence between a taxonomic species and a single phenogroup and genogroup. Instead, all specimens were assigned to phenogroups and genogroups that were shared across multiple taxonomic species (See main text, Table 4; Table \@ref(tab:cladeVfinalSummary)).

## Clade VI

### Sampling

A total of $`r CladeVItotalSampling`$ specimens were included in this clade (Figure \@ref(fig:CladeVIVennDiagram)). We measured phenotypic traits on $`r CladeVItotalPhenome`$ specimens;  $`r CladeVIphenomeFertile`$ were fertile specimens (thus having both leaf and floral traits) and $`r CladeVIphenomeSterile`$ were sterile specimens (thus having leaf traits only). Only fertile specimens were included in downstream phenomic analyses. We collected genomic data for $`r CladeVItotalGenome`$ specimens; $`r CladeVIgenomeFertile`$ specimens were fertile, $`r CladeVIgenomeSterile`$ were sterile, and $`r CladeVIgenomeNoPhenome`$ were not available for phenotypic analyses. All $`r CladeVItotalGenome`$ specimens were included in downstream genomic analyses except in our model selection analysis (See below). The $`r CladeVItotalSampling`$ specimens covered the geographic range of this clade and belonged to ten taxonomic species (Figure \@ref(fig:CladeVIsamplingMaps), Table \@ref(tab:SamplingTable))

### Phenomics

#### Current state of taxonomic species

The 10-cubes defining the taxonomic species belonging to this clade did not overlap in 10-dimensional phenospace. The matching-prediction analysis showed that only $`r cladeVIpercentassigned`$\% of the specimens fall inside any 10-cube, with no specimens falling inside their correct 10-cube (Figure \@ref(fig:currentStateOfTaxonomyCladeVIPlots)).

#### Model-based species discovery

We found nine principal components to be most useful for group discrimination. The naive model specifying eight phenogroups of equal shape and orientation received the strongest support using both the BIC and ICL criteria (BIC$=231.247$, ICL$=224.2705$). Support for alternative models was considerably lower, including the Taxonomy model [@Sleumer:1968ts] specifying ten phenogroups ($\Delta$BIC$=749.011$) and the Taxonomy Unaware model specifying ten phenogroups ($\Delta$BIC$=30.944$) (Figure \@ref(fig:cladeVImorphologicalDelimitationPlots)). A LRT further corroborated that a model with eight phenogroups of equal shape and orientation was a better fit to the data than models with more phenogroups ($p$-value $<0.05$)

### Genomics

#### Sensitivity Tests

In total, we recovered $133,123$ loci across $`r CladeVItotalGenome`$ sequenced individuals. $49,105$ loci were present in at least four individuals. The number of loci in the three matrices with different levels of missing data is presented in Table \@ref(tab:CladeVIgenomicDatasetAssemblies). Results from Principal Components Analysis, phylogenetic analysis, and genomic clustering did not differ (or differed minimally) across datasets, suggesting that our data are robust to the amount of missing data (Figures \@ref(fig:cladeVIgenomicSensitivityAnalysesLargestplots), \@ref(fig:cladeVIgenomicSensitivityAnalysesMiddleplots), \@ref(fig:cladeVIgenomicSensitivityAnalysesSmallestplots)). Therefore, we performed all downstream analyses using the smallest data matrix for computational efficiency.

#### Model-based species discovery

**Genotypic clusters (GC model)** We found two dimensions to be most useful to faithfully represent the genotypic data in fewer dimensions (2 dimensions $= 9.34\%$ stress). In this reduced space, a model specifying seven genotypic clusters received the highest support (BIC$=1061.975$, next best model $\Delta$BIC$=1.6393$, seven clusters; Figure \@ref(fig:cladeVImolecularDelimitationsNmms)).

**Transition between cladogenesis and anagenesis (CA model)** Given the phylogeny reconstructed with the concatenated alignment of full sequences per locus, a model with eleven genogroups was the best fit to the data (Figure \@ref(fig:cladeVImolecularDelimitationsPhylogeny2)). Across all independent runs, all nodes identifying genogroups strongly identified such nodes as transitions between cladogenesis and anagenesis (Fig \@ref(fig:cladeVImolecularDelimitationsPhylogeny2); See values closer to $0$ subtending red groups).

**Reproductive isolation (RI model)** A model specifying four demes received the strongest support based on the $\Delta$K statistic using \texttt{STRUCTURE}. A model specifying five demes using \texttt{MAVERICK} had the highest posterior probability ($pp>0.99$); however, one of these demes never received a majority posterior assignment for any specimen. Because we used a majority posterior assignment to assign specimens to demes, we thus used a model with four demes, which was identical to the model specified using \texttt{STRUCTURE}. Therefore, the RI model used the same deme assignment (four demes) with both \texttt{STRUCTURE} and \texttt{MAVERICK}. Using this deme assignment, we found consistent and strong support for a model specifying four genogroups ($pp=1.0$, Figure \@ref(fig:cladeVImolecularDelimitationPopGenBpp)), for both the full and reduced datasets (See Methods for details).

**Model Selection** We filtered our matrix and retained $915$ loci to conduct this analysis (Table \@ref(tab:CladeVIgenomicDatasetAssemblies)). We compared the three models specified above using Bayes Factors. Each model required approximately $250,000$ samples to stabilize (as estimated from ESS values for each step) with the exception of the CA model which required more than $500,000$ samples to stabilize. In total, each analysis used $24$ steps. The CA model (eleven genogroups) was the top-ranked model with decisive support relative to the other models [@kass1995bayes] (Table \@ref(tab:cladeVIbfdTable)).

### Data Integration

Based on the best fitting models, we assigned each specimen to its corresponding phenogroup and genogroup (Figure \@ref(fig:cladeVIfinalDelimitation2)). In total, we assigned specimens to 10 species (See main text, Figure 2). Three species matched uniquely a single phenogroup to a single genogroup (Figure \@ref(fig:cladeVIfinalDelimitation2); See for example phenogroup 2 - genogroup 5). Therefore, these species are recognized as 'good species'. Six species belonged to two phenogroups across five genogroups (Figure \@ref(fig:cladeVIfinalDelimitation2); See for example phenogroup 1 - genogroups 2, 4, 10, 11). These species are collectively recongnized as 'phenotypic cryptic species'. Four species belonged to two genogroups across two phenogroups (Figure \@ref(fig:cladeVIfinalDelimitation2); See for example genogroup 4 - phenogroups 1, 4). These species are collectively recognized as 'genotypic cryptic species'. We also discovered two phenogroups for which we had no genomic data (Figure \@ref(fig:cladeVIfinalDelimitation2); See phenogroups 3 and 6), and three genogroups for which we had no phenotypic data (Figure \@ref(fig:cladeVIfinalDelimitation2); See genogroups 6, 8, and 9). The association of phenogroups to genogroups was $GK\tau_{pg}=0.733$, while the association of genogroups to phenogroups was $GK\tau_{gp}=0.845$. This indicates that genogroup membership is a slightly better predictor of phenogroup membership than phenogroup membership is of genogroup membership. In other words, this shows there are more 'phenotypic cryptic species' in this clade than 'genetic cryptic species'.

We examined all the specimens in their phylogenetic and geographic contexts to gain insight into the plausible species identity of the unknown specimens and the nature of species in this clade. In contrast to other clades, phenogroups and genogroups in clade VI show an overall strong pattern of geographic structure (Figure \@ref(fig:cladeVIfinalDelimitation2)). Considering the geographic distribution of the unknown specimens assigned to genogroup 4, in combination with our extensive field and herbarium work, we speculate that these specimens most likely belong to phenogroups 1 or 4. In addition, given the narrow longitudinal range of genogroup 4, we further speculate that the unknown specimens assigned to phenogroups 1 or 4 that occur around longitude $-65^\circ$ likely belong to genogroup 4. These assignments would indicate that the closely related phenogroups 1 and 4 within genogroup 4 segregate largely along elevation (Figure \@ref(fig:cladeVIfinalDelimitation2)). Similarly, genogroups 1, 2, and 3 have a narrow longitudinal range with phenogroup 7 largely restricted to lower elevations (Figure \@ref(fig:cladeVIfinalDelimitation2)). Therefore, we suggest that the unknown specimens in genogroup 1 most likely belong to phenogroup 7, and the unknown specimens in phenogroup 7 likely belong to genogroup 1. By contrast, owing to our limited sampling of phenotypic data for specimens east of longitude $-60^\circ$ as well as the broad sympatry of all species in this geographic region, it is difficult to infer the possible phenogroup membership of the unknown specimens in genogroups 5, 6, 7, 8, and 9. Given our extensive field and herbarium work, we can only speculate that the unknown specimens in genogroups 10 and 11 most likely belong to phenogroup 1. These uncertainties aside, together our results showed that at the spatial scale of our study, groups of species co-occur in close geographic proximity with at least one other closely related species (Figure \@ref(fig:cladeVIfinalDelimitation2)). Our phylogenetic and geographic results also showed that while the 'good species' are allopatric with one another, they co-occur in broad sympatry with other species which are not their closest relatives. The species in one group of 'phenotypic cryptic species' (phenogroup 1) are not all closely related and are allopatric, suggesting convergent evolution in phenotypes, perhaps driven by niche conservatism [@wiens2004speciation]. Both groups of 'genetic cryptic species' (genogroups 2 and 4) co-occur in broad sympatry, yet their respective phenogroups seeminlgy segregate along elevation (Figure \@ref(fig:cladeVIfinalDelimitation2); See genogroup 4 - phenogroups 1, 4). This pattern is consistent with the hypothesis of ecological speciation along elevation gradients, but extensive sampling and modeling is needed to test this hypothesis.

### Correspondence between taxonomic species and model-based species

We assigned all specimens to their corresponding taxonomic species, phenogroup, and genogroup. Only specimens assigned to two taxonomic species ( _E. bifida_ and _E. farinacea_; Figure \@ref(fig:cladeVIfinalDelimitation2); See phenogroup 2 - genogroup 5, and phenogroup 5 - genogroup 7, respectively) showed perfect correspondence to a single phenogroup and a single genogroup. All other specimens were assigned to phenogroups and genogroups shared across multiple taxonomic species (See main text, Table 4; Table \@ref(tab:cladeVIfinalSummary)).

\pagebreak

# Tables

\let\oldthefigure=\thefigure
\let\oldthetable=\thetable

\setcounter{figure}{0}
\makeatletter
\renewcommand{\thefigure}{S\@arabic\c@figure}
\renewcommand{\thetable}{S\@arabic\c@table}
\makeatother

(ref:ms-nature-of-species-github) (See https://github.com/zapata-lab/ms_nature_of_species for further details)


### Table S1: Taxon Sampling
```{r SamplingTable, echo=FALSE, dpi=200}
escalloniaSampling_msTable = escalloniaSampling %>%
  mutate( Phenomics = ifelse(`Floral and Leaf` == 'x',
                             'yes',
                             'no' )) %>%
  mutate( Genomics = ifelse(is.na(libraryNumber),
                            'no',
                            'yes' )) %>%
  mutate( sampledForBFD = ifelse(sampledForBFD == 1,
                                 'yes',
                                 ifelse(sampledForBFD == 0,
                                        'no',
                                        NA))) %>%
  dplyr::rename(`Taxonomic species` = Taxon,
                `Taxonomy unaware group` = MorphogroupID,
                `Sampled for BFD` = sampledForBFD) %>%
  replace_na(list(Phenogroup = 'unk')) %>%
  mutate(Genogroup = gsub("[group]",
                          "",
                          Genogroup)) %>%
  mutate(Genogroup = gsub("[species]",
                          "",
                          Genogroup)) %>%
  mutate(Phenogroup = ifelse(is.na(Phenogroup),
                             'unk',
                             Phenogroup)) %>%
  mutate(Genogroup = ifelse(is.na(Genogroup),
                            'unk',
                            Genogroup))


escalloniaSampling_msTable = escalloniaSampling_msTable %>%
  select(SpecimenID,
         `Taxonomic species`,
         `Taxonomy unaware group`,
         CladeID,
         Latitude,
         Longitude,
         Elevation,
         Phenomics,
         Genomics,
         `Sampled for BFD`,
         Phenogroup,
         Genogroup)

knitr::kable(escalloniaSampling_msTable, caption = 'Specimens used in this study. Vouchers are deposited in different herbaria (ref:ms-nature-of-species-github)',
             align = c('c','c','c','c','c','c','c','c','c','c','c','c'),
             booktabs = T,
             longtable = T,
             digits = 2,
             # col.names=c("Specimen ID", "Herbarium Accession","Taxonomic ID", "Morphogroup ID", "Clade ID",
             #                "Latitidue", "Longitude","Elevation","Fertile","Sterile","Library Number","Sampled for BFD",
             #                "Phenogroup","Genogroup")
             ) %>%
  kable_styling(font_size = 6, full_width = T)

```
### Table S2: Model Adequacy
```{r ModelAdequacyTable, echo=FALSE, dpi=200}

escModelAdequacy = read.csv("supplementary_material_files/modelAdequacy.csv", header=TRUE)

escModelAdequacy %>%
  kable(format = 'latex',
        booktabs = T,
        linesep = "",
        align = c('c','c','c','c'),
        caption = "Kolmogorov's D Statistic and associated p-values calculated for each genogroup identified within each clade.") %>%
  kable_styling(row_label_position = 'r',
                latex_options = "HOLD_position")

```
## Clade I
#### Table S2: Genomic dataset details
```{r, CladeIgenomicDatasetAssemblies, echo=FALSE, message=FALSE, warning=FALSE}

assemblyInfo = read.csv("supplementary_material_files/CladeI_resultsFiles/CladeI_molecularAssemblies.csv", header = TRUE, row.names=1)

assemblyInfo %>%
  kable(format = 'latex',
        booktabs = T,
        align = c('l','c','c','c'),
        caption = "Counts of specimens, loci, and sites associated with each data matrix.") %>%
  kable_styling(row_label_position = 'r',
                latex_options = "HOLD_position") %>%
  pack_rows("iPyrad Assembly",
            1,
            3) %>%
  pack_rows("Filtering with VCFTOOLS",
            4,
            8)

```
Return to [Clade I Genomics](#sensitivity-tests)
\pagebreak

#### Table S3: Genogroup delimitation
```{r, cladeIbfdTable, echo=FALSE, message=FALSE, warning=FALSE}

bfdInfo = read.csv("supplementary_material_files/CladeI_resultsFiles/CladeI_bfdResults.csv",
                   header = TRUE)

bfdInfo %>%
  kable(format ='latex',
        booktabs = T,
        escape = F,
        align = 'c',
        caption = "Genomic modeling for genogroup delimitation and model selection using Bayes factors (BF)") %>%
        #col.names = c("Clade","Number of loci","Model","Number of putative species","MLE","Rank","Bayes Factor")) %>%
  kable_styling(latex_options = c("scale_down",
                                  "HOLD_position")) %>%
  footnote(alphabet = c("specimens assigned to demes using MAVERICK",
                        "specimens assigned to demes using STRUCTURE"),
           threeparttable = T)

```
Return to [Clade I Model-based species discovery](#model-based-species-discovery-1)
\pagebreak

#### Table S4: Correspondence between taxonomic species, phenogroups, and genogroups
```{r cladeIfinalSummary, warning=FALSE, message=FALSE, echo=FALSE}
CladeI_finalSummary = read_csv("supplementary_material_files/CladeI_resultsFiles/CladeI_finalSummary.csv")

CladeI_finalSummary %>%
  mutate(
    `taxonomic species`= ...1,
    p1 = cell_spec(Pheno1,
               "latex",
               background = factor(as.factor(Pheno1),
                                   c("0","1"),
                                   c("#FFFFFF","#BBBBBB")),
               color = factor(as.factor(Pheno1),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
    p2 = cell_spec(Pheno2,
                 "latex",
                 background = factor(as.factor(Pheno2),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno2),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
    g1 = cell_spec(Geno1,
                 "latex",
                 background = factor(as.factor(Geno1),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno1),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
    g2 = cell_spec(Geno2,
                 "latex",
                 background = factor(as.factor(Geno2),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno2),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
    g3 = cell_spec(Geno3,
                 "latex",
                 background = factor(as.factor(Geno3),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno3),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB")))) %>%
  select(`taxonomic species`,p1,p2,g1,g2,g3) %>%
  kable(format = "latex",
        escape =F,
        booktabs =T,
        linesep = "",
        caption = "Correspondence between taxonomic species and best-fit phenogroups and genogroups. Shaded cells show specimens assigned to a particular combination of taxonomic species, best fit phenogroup, and best fit genogroup.") %>%
  row_spec(1,
           hline_after = T) %>%
  column_spec(1,
              border_right = T,
              width = "15mm") %>%
  column_spec(2,
              border_right = T,
              width = "5mm") %>%
  column_spec(3,
              border_right = T,
              width = "5mm") %>%
  column_spec(4,
              border_right = T,
              border_left = T,
              width = "5mm") %>%
  column_spec(5,
              border_right = T,
              width = "5mm") %>%
  column_spec(6,
              border_right = T,
              width = "5mm") %>%
  add_header_above(c("",
                     "Phenogroups" = 2,
                     "Genogroups" = 3)) %>%
  kable_styling(font_size = 8,
                latex_options = "HOLD_position")
```
Return to [Clade I Correspondence between taxonomic species and model-based species](#correspondence-between-taxonomic-species-and-model-based-species)
\pagebreak

## Clade II
#### Table S5: Genomic dataset details
```{r, CladeIIgenomicDatasetAssemblies, echo=FALSE, message=FALSE, warning=FALSE}

assemblyInfo = read.csv("supplementary_material_files/CladeII_resultsFiles/CladeII_molecularAssemblies.csv",
                        header = TRUE,
                        row.names = 1)

assemblyInfo %>%
  kable(format = 'latex',
        booktabs = T,
        align = c('l','c','c','c'),
        caption = "Counts of specimens, loci, and sites associated with each data matrix.") %>%
  kable_styling(row_label_position = 'r',
                latex_options = "HOLD_position") %>%
  pack_rows("iPyrad Assembly",
            1,
            3) %>%
  pack_rows("Filtering with VCFTOOLS",
            4,
            8)

```
Return to [Clade II Genomics](#sensitivity-tests-1)
\pagebreak

#### Table S6: Genogroup delimitation
```{r, cladeIIbfdTable, echo=FALSE, message=FALSE, warning=FALSE}

bfdInfo = read.csv("supplementary_material_files/CladeII_resultsFiles/CladeII_bfdResults.csv",
                   header = TRUE)

options(knitr.kable.NA = '')

bfdInfo %>%
  kable(format ='latex',
        booktabs = T,
        escape = F,
        align = 'c',
        caption = "Genomic modeling for genogroup delimitation and model selection using Bayes factors (BF)") %>%
        #col.names = c("Clade","Number of loci","Model","Number of putative species","MLE","Rank","Bayes Factor")) %>%
  kable_styling(latex_options = c("scale_down",
                                  "HOLD_position")) %>%
  footnote(alphabet = c("specimens assigned to demes using MAVERICK",
                        "specimens assigned to demes using STRUCTURE"),
           threeparttable = T)

```
Return to [Clade II Model-based species discovery](#model-based-species-discovery-3)
\pagebreak

#### Table S7: Correspondence between taxonomic species, phenogroups, and genogroups
```{r cladeIIfinalSummary, warning=FALSE, message=FALSE, echo=FALSE}
CladeII_finalSummary = read_csv("supplementary_material_files/CladeII_resultsFiles/CladeII_finalSummary.csv")

CladeII_finalSummary %>%
  mutate(`taxonomic species`= ...1,
         p1 = cell_spec(Pheno1,
                        "latex",
                        background = factor(as.factor(Pheno1),
                                            c("0","1"),
                                            c("#FFFFFF","#BBBBBB")),
                        color = factor(as.factor(Pheno1),
                                       c("0","1"),
                                       c("#FFFFFF","#BBBBBB"))),
         p2 = cell_spec(Pheno2,
                        "latex",
                        background = factor(as.factor(Pheno2),
                                          c("0","1"),
                                          c("#FFFFFF","#BBBBBB")),
                        color = factor(as.factor(Pheno2),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB"))),
         p3 = cell_spec(Pheno2,
                        "latex",
                        background = factor(as.factor(Pheno2),
                                            c("0","1"),
                                            c("#FFFFFF","#BBBBBB")),
                        color = factor(as.factor(Pheno2),
                                       c("0","1"),
                                       c("#FFFFFF","#BBBBBB"))),
         g1 = cell_spec(Geno1,
                        "latex",
                        background = factor(as.factor(Geno1),
                                            c("0","1"),
                                            c("#FFFFFF","#BBBBBB")),
                        color = factor(as.factor(Geno1),
                                       c("0","1"),
                                       c("#FFFFFF","#BBBBBB"))),
         g2 = cell_spec(Geno2,
                        "latex",
                        background = factor(as.factor(Geno2),
                                            c("0","1"),
                                            c("#FFFFFF","#BBBBBB")),
                        color = factor(as.factor(Geno2),
                                       c("0","1"),
                                       c("#FFFFFF","#BBBBBB"))),
         g3 = cell_spec(Geno3,
                        "latex",
                        background = factor(as.factor(Geno3),
                                            c("0","1"),
                                            c("#FFFFFF","#BBBBBB")),
                        color = factor(as.factor(Geno3),
                                       c("0","1"),
                                       c("#FFFFFF","#BBBBBB"))),
         g4 = cell_spec(Geno4,
                        "latex",
                        background = factor(as.factor(Geno4),
                                            c("0","1"),
                                            c("#FFFFFF","#BBBBBB")),
                        color = factor(as.factor(Geno4),
                                       c("0","1"),
                                       c("#FFFFFF","#BBBBBB")))) %>%
  select(`taxonomic species`,p1,p2,p3,g1,g2,g3,g4) %>%
  kable(format = "latex",
        escape = F,
        booktabs = T,
        linesep = "",
        caption = "Correspondence between taxonomic species and best-fit phenogroups and genogroups. Shaded cells show specimens assigned to a particular combination of taxonomic species, best fit phenogroup, and best fit genogroup.") %>%
  row_spec(1,
           hline_after = T) %>%
  column_spec(1:8,
              border_right = T) %>%
  column_spec(5,
              border_right = T,
              border_left = T) %>%
  add_header_above(c("","Phenogroups"= 3,"Genogroups"= 4)) %>%
  kable_styling(font_size = 8,
                latex_options = "HOLD_position")
```
Return to [Clade II Correspondence between taxonomic species and model-based species](#correspondence-between-taxonomic-species-and-model-based-species-1)
\pagebreak

## Clade III
#### Table S8: Genomic dataset details
```{r, CladeIIIgenomicDatasetAssemblies, echo=FALSE, message=FALSE, warning=FALSE}

assemblyInfo = read.csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_molecularAssemblies.csv",
                        header = TRUE,
                        row.names = 1)

assemblyInfo %>%
  kable(format = 'latex',
        booktabs = T,
        align = c('l','c','c','c'),
        caption = "Counts of specimens, loci, and sites associated with each data matrix.") %>%
  kable_styling(row_label_position = 'r',
                latex_options = "HOLD_position") %>%
  pack_rows("iPyrad Assembly",
            1,
            3) %>%
  pack_rows("Filtering with VCFTOOLS",
            4,
            8)

```
Return to [Clade III Genomics](#sensitivity-tests-2)
\pagebreak

#### Table S9: Genogroup delimitation
```{r, cladeIIIbfdTable, echo=FALSE, message=FALSE, warning=FALSE}

bfdInfo = read.csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_bfdResults.csv",
                   header = TRUE)

options(knitr.kable.NA = '')

bfdInfo %>%
  kable(format = 'latex',
        booktabs = T,
        escape = F,
        align = 'c',
        caption ="Genomic modeling for genogroup delimitation and model selection using Bayes factors (BF)") %>%
        #col.names = c("Clade","Number of loci","Model","Number of putative species","MLE","Rank","Bayes Factor")) %>%
  kable_styling(latex_options = c("scale_down",
                                  "HOLD_position")) %>%
  footnote(alphabet = c("specimens assigned to demes using MAVERICK",
                        "specimens assigned to demes using STRUCTURE"),
           threeparttable = T)

```
Return to [Clade III Model-based species discovery](#model-based-species-discovery-5)
\pagebreak

#### Table S10: Correspondence between taxonomic species, phenogroups, and genogroups
```{r cladeIIIfinalSummary, warning=FALSE, message=FALSE, echo=FALSE}
CladeIII_finalSummary = read_csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_finalSummary.csv")
CladeIII_finalSummary %>%
  mutate(
  `taxonomic species`= ...1,
  p1 = cell_spec(Pheno1,
                 "latex",
                 background = factor(as.factor(Pheno1),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno1),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p2 = cell_spec(Pheno2,
                 "latex",
                 background = factor(as.factor(Pheno2),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno2),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p3 = cell_spec(Pheno3,
                 "latex",
                 background = factor(as.factor(Pheno3),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno3),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p4 = cell_spec(Pheno4,
                 "latex",
                 background = factor(as.factor(Pheno4),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno4),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p5 = cell_spec(Pheno5,
                 "latex",
                 background = factor(as.factor(Pheno5),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno5),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g1 = cell_spec(Geno1,
                 "latex",
                 background = factor(as.factor(Geno1),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno1),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g2 = cell_spec(Geno2,
                 "latex",
                 background = factor(as.factor(Geno2),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno2),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g3 = cell_spec(Geno3,
                 "latex",
                 background = factor(as.factor(Geno3),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno3),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g4 = cell_spec(Geno4,
                 "latex",
                 background = factor(as.factor(Geno4),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno4),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g5 = cell_spec(Geno5,
                 "latex",
                 background = factor(as.factor(Geno5),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno5),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g6 = cell_spec(Geno6,
                 "latex",
                 background = factor(as.factor(Geno6),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno6),
                                c("0","1"), 
                                c("#FFFFFF","#BBBBBB"))),
  g7 = cell_spec(Geno7,
                 "latex",
                 background = factor(as.factor(Geno7),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno7),
                                c("0","1"), 
                                c("#FFFFFF","#BBBBBB")))) %>%
  select(`taxonomic species`,p1,p2,p3,p4,p5,g1,g2,g3,g4,g5,g6,g7) %>%
  kable(format = "latex",
        escape = F,
        booktabs = T,
        linesep = "",
        caption = "Correspondence between taxonomic species and best-fit phenogroups and genogroups. Shaded cells show specimens assigned to a particular combination of taxonomic species, best fit phenogroup, and best fit genogroup.") %>%
  row_spec(1:5,
           hline_after = T) %>%
  column_spec(1:12,
              border_right = T) %>%
  column_spec(7,
              border_right = T,
              border_left = T) %>%
  add_header_above(c("","Phenogroups"=5,"Genogroups"=7)) %>%
  kable_styling(font_size = 8,
                latex_options = "HOLD_position")
```
Return to [Clade III Correspondence between taxonomic species and model-based species](#correspondence-between-taxonomic-species-and-model-based-species-2)
\pagebreak

## Clade IV
#### Table S11: Genomic dataset details
```{r, CladeIVgenomicDatasetAssemblies, echo=FALSE, message=FALSE, warning=FALSE}

assemblyInfo = read.csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_molecularAssemblies.csv",
                        header = TRUE,
                        row.names = 1)

assemblyInfo %>%
  kable(format = 'latex',
        booktabs = T,
        align = c('l','c','c','c'),
        caption = "Counts of specimens, loci, and sites associated with each data matrix.") %>%
  kable_styling(row_label_position = 'r',
                latex_options = "HOLD_position") %>%
  pack_rows("iPyrad Assembly",
            1,
            3) %>%
  pack_rows("Filtering with VCFTOOLS",
            4,
            8)

```
Return to [Clade IV Genomics](#sensitivity-tests-3)
\pagebreak

#### Table S12: Genogroup delimitation
```{r, cladeIVbfdTable, echo=FALSE, message=FALSE, warning=FALSE}

bfdInfo = read.csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_bfdResults.csv",
                   header = TRUE)

options(knitr.kable.NA = '')

bfdInfo %>%
  kable(format = 'latex',
        booktabs = T,
        escape = F,
        align = 'c',
        caption = "Genomic modeling for genogroup delimitation and model selection using Bayes factors (BF)") %>%
        #col.names = c("Clade","Number of loci","Model","Number of putative species","MLE","Rank","Bayes Factor")) %>%
  kable_styling(latex_options = c("scale_down",
                                  "HOLD_position")) %>%
  footnote(alphabet = c("specimens assigned to demes using MAVERICK",
                        "specimens assigned to demes using STRUCTURE"),
           threeparttable = T)

```
Return to [Clade IV Model-based species discovery](#model-based-species-discovery-7)
\pagebreak

#### Table S13: Correspondence between taxonomic species, phenogroups, and genogroups
```{r cladeIVfinalSummary, warning=FALSE, message=FALSE, echo=FALSE}
CladeIV_finalSummary = read_csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_finalSummary.csv")
CladeIV_finalSummary %>%
  mutate(
  `taxonomic species`= ...1,
  p1 = cell_spec(Pheno1,
                 "latex",
                 background = factor(as.factor(Pheno1),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno1),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p2 = cell_spec(Pheno2,
                 "latex",
                 background = factor(as.factor(Pheno2),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno2),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g1 = cell_spec(Geno1,
                 "latex",
                 background = factor(as.factor(Geno1),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno1),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g2 = cell_spec(Geno2,
                 "latex",
                 background = factor(as.factor(Geno2),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno2),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g3 = cell_spec(Geno3,
                 "latex",
                 background = factor(as.factor(Geno3),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno3),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g4 = cell_spec(Geno4,
                 "latex",
                 background = factor(as.factor(Geno4),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color=factor(as.factor(Geno2),
                              c("0","1"),
                              c("#FFFFFF","#BBBBBB"))),
  g5 = cell_spec(Geno5,
                 "latex",
                 background = factor(as.factor(Geno5),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno2),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g6 = cell_spec(Geno6,
                 "latex",
                 background = factor(as.factor(Geno6),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno2),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB")))) %>%
  select(`taxonomic species`,p1,p2,g1,g2,g3,g4,g5,g6) %>%
  kable(format = "latex",
        escape = F,
        booktabs = T,
        align = "lcccccccc",
        linesep = "",
        caption ="Correspondence between taxonomic species and best-fit phenogroups and genogroups. Shaded cells show specimens assigned to a particular combination of taxonomic species, best fit phenogroup, and best fit genogroup.") %>%
  row_spec(1,
           hline_after = T) %>%
  column_spec(1,
              border_right = T,
              width = "15mm") %>%
  column_spec(2,
              border_right = T,
              width = "5mm") %>%
  column_spec(3,
              border_right = T,
              width ="5mm") %>%
  column_spec(4,
              border_right = T,
              border_left = T,
              width ="5mm") %>%
  column_spec(5,
              border_right = T,
              width ="5mm") %>%
  column_spec(6,
              border_right = T,
              width ="5mm") %>%
  column_spec(7,
              border_right = T,
              width ="5mm") %>%
  column_spec(8,
              border_right = T,
              width ="5mm") %>%
  column_spec(9,
              border_right = T,
              width ="5mm") %>%
  add_header_above(c("","Phenogroups"=2,"Genogroups"=6)) %>%
  kable_styling(font_size = 8,
                latex_options = "HOLD_position")
```
Return to [Clade IV Correspondence between taxonomic species and model-based species](#correspondence-between-taxonomic-species-and-model-based-species-3)
\pagebreak

## Clade V
#### Table S14: Genomic dataset details
```{r, CladeVgenomicDatasetAssemblies, echo=FALSE, message=FALSE, warning=FALSE}

assemblyInfo = read.csv("supplementary_material_files/CladeV_resultsFiles/CladeV_molecularAssemblies.csv",
                        header = TRUE,
                        row.names=1)

assemblyInfo %>%
  kable(format = 'latex',
        booktabs = T,
        align = c('l','c','c','c'),
        caption = "Counts of specimens, loci, and sites associated with each data matrix.") %>%
  kable_styling(row_label_position = 'r',
                latex_options = "HOLD_position") %>%
  pack_rows("iPyrad Assembly",
            1,
            3) %>%
  pack_rows("Filtering with VCFTOOLS",
            4,
            8)

```
Return to [Clade V Genomics](#sensitivity-tests-4)
\pagebreak

#### Table S15: Genogroup delimitation
```{r, cladeVbfdTable, echo=FALSE, message=FALSE, warning=FALSE}

bfdInfo = read.csv("supplementary_material_files/CladeV_resultsFiles/CladeV_bfdResults.csv",
                   header = TRUE)

options(knitr.kable.NA = '')

bfdInfo %>%
  kable(format = 'latex',
        booktabs = T,
        escape = F,
        align = 'c',
        caption = "Genomic modeling for genogroup delimitation and model selection using Bayes factors (BF)") %>%
  kable_styling(latex_options = c("scale_down",
                                  "HOLD_position")) %>%
  footnote(alphabet = c("specimens assigned to demes using MAVERICK",
                        "specimens assigned to demes using STRUCTURE"),
           threeparttable = T)

```
Return to [Clade V Model-based species discovery](#model-based-species-discovery-9)
\pagebreak

#### Table S16: Correspondence between taxonomic species, phenogroups, and genogroups
```{r cladeVfinalSummary, warning=FALSE, message=FALSE, echo=FALSE}
CladeV_finalSummary = read_csv("supplementary_material_files/CladeV_resultsFiles/CladeV_finalSummary.csv")
CladeV_finalSummary %>%
  mutate(
  `taxonomic species`= ...1,
  p1 = cell_spec(Pheno1,
                 "latex",
                 background = factor(as.factor(Pheno1),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno1),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p2 = cell_spec(Pheno2,
                 "latex",
                 background = factor(as.factor(Pheno2),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno2),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p3 = cell_spec(Pheno3,
                 "latex",
                 background = factor(as.factor(Pheno3),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno3),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p4 = cell_spec(Pheno4,
                 "latex",
                 background = factor(as.factor(Pheno4),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno4),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p5 = cell_spec(Pheno5,
                 "latex",
                 background = factor(as.factor(Pheno5),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno5),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p6 = cell_spec(Pheno6,
                 "latex",
                 background = factor(as.factor(Pheno6),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno6),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p7 = cell_spec(Pheno7,
                 "latex",
                 background = factor(as.factor(Pheno7),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno7),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p8 = cell_spec(Pheno8,
                 "latex",
                 background = factor(as.factor(Pheno8),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno8),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
   g1 = cell_spec(Geno1,
                  "latex",
                  background = factor(as.factor(Geno1),
                                      c("0","1"),
                                      c("#FFFFFF","#BBBBBB")),
                  color = factor(as.factor(Geno1),
                                 c("0","1"),
                                 c("#FFFFFF","#BBBBBB"))),
  g2 = cell_spec(Geno2,
                 "latex",
                 background = factor(as.factor(Geno2),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno2),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g3 = cell_spec(Geno3,
                 "latex",
                 background = factor(as.factor(Geno3),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno3),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g4 = cell_spec(Geno4,
                 "latex",
                 background = factor(as.factor(Geno4),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno4),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g5 = cell_spec(Geno5,
                 "latex",
                 background = factor(as.factor(Geno5),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno5),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g6 = cell_spec(Geno6,
                 "latex",
                 background = factor(as.factor(Geno6),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno6),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g7 = cell_spec(Geno7,
                 "latex",
                 background = factor(as.factor(Geno7),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno7),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g8 = cell_spec(Geno8,
                 "latex",
                 background = factor(as.factor(Geno8),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color  = factor(as.factor(Geno8),
                                 c("0","1"),
                                 c("#FFFFFF","#BBBBBB"))),
  g9 = cell_spec(Geno9,
                 "latex",
                 background = factor(as.factor(Geno9),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno9),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g10 = cell_spec(Geno10,
                  "latex",
                  background = factor(as.factor(Geno10),
                                      c("0","1"),
                                      c("#FFFFFF","#BBBBBB")),
                  color = factor(as.factor(Geno10),
                                 c("0","1"),
                                 c("#FFFFFF","#BBBBBB")))) %>%
  select(`taxonomic species`,p1,p2,p3,p4,p5,p6,p7,p8,g1,g2,g3,g4,g5,g6,g7,g8,g9,g10) %>%
  kable(format = "latex",
        escape = F,
        booktabs = T,
        linesep = "",
        caption = "Correspondence between taxonomic species and best-fit phenogroups and genogroups. Shaded cells show specimens assigned to a particular combination of taxonomic species, best fit phenogroup, and best fit genogroup.") %>%
  row_spec(1:6,
           hline_after = T) %>%
  column_spec(1:19,
              border_right = T) %>%
  column_spec(10,
              border_right = T,
              border_left = T) %>%
  add_header_above(c("","Phenogroups"=8,"Genogroups"=10)) %>%
  kable_styling(font_size = 8,
                latex_options = "HOLD_position")
```
Return to [Clade V Correspondence between taxonomic species and model-based species](#correspondence-between-taxonomic-species-and-model-based-species-4)
\pagebreak

## Clade VI
#### Table S17: Genomic dataset details
```{r, CladeVIgenomicDatasetAssemblies, echo=FALSE, message=FALSE, warning=FALSE}

assemblyInfo = read.csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_molecularAssemblies.csv",
                        header = TRUE,
                        row.names = 1)

assemblyInfo %>%
  kable(format = 'latex',
        booktabs = T,
        align = c('l','c','c','c'),
        caption = "Counts of specimens, loci, and sites associated with each data matrix.") %>%
  kable_styling(row_label_position = 'r',
                latex_options = "HOLD_position") %>%
  pack_rows("iPyrad Assembly",
            1,
            3) %>%
  pack_rows("Filtering with VCFTOOLS",
            4,
            8)

```
Return to [Clade VI Genomics](#sensitivity-tests-5)
\pagebreak

#### Table S18: Genogroup delimitation
```{r, cladeVIbfdTable, echo=FALSE, message=FALSE, warning=FALSE}

bfdInfo = read.csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_bfdResults.csv",
                   header = TRUE)

options(knitr.kable.NA = '')

bfdInfo %>%
  kable(format = 'latex',
        booktabs = T,
        escape = F,
        align = 'c',
        caption = "Genomic modeling for genogroup delimitation and model selection using Bayes factors (BF)") %>%
        #col.names = c("Clade","Number of loci","Model","Number of putative species","MLE","Rank","Bayes Factor")) %>%
  kable_styling(latex_options = c("scale_down",
                                  "HOLD_position")) %>%
  footnote(alphabet = c("specimens assigned to demes using MAVERICK",
                        "specimens assigned to demes using STRUCTURE"),
           threeparttable = T)

```
Return to [Clade VI Model-based species discovery](#model-based-species-discovery-11)
\pagebreak

#### Table S19: Correspondence between taxonomic species, phenogroups, and genogroups
```{r cladeVIfinalSummary, warning=FALSE, message=FALSE, echo=FALSE}
CladeVI_finalSummary = read_csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_finalSummary.csv")

CladeVI_finalSummary %>%
  mutate(
  `taxonomic species` = ...1,
  p1 = cell_spec(Pheno1,
                 "latex",
                 background = factor(as.factor(Pheno1),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno1),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p2 = cell_spec(Pheno2,
                 "latex",
                 background = factor(as.factor(Pheno2),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno2),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p3 = cell_spec(Pheno3,
                 "latex",
                 background = factor(as.factor(Pheno3),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno3),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p4 = cell_spec(Pheno4,
                 "latex",
                 background = factor(as.factor(Pheno4),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno4),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p5 = cell_spec(Pheno5,
                 "latex",
                 background = factor(as.factor(Pheno5),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno5),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p6 = cell_spec(Pheno6,
                 "latex",
                 background = factor(as.factor(Pheno6),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno6),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p7 = cell_spec(Pheno7,
                 "latex",
                 background = factor(as.factor(Pheno7),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno7),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  p8 = cell_spec(Pheno8,
                 "latex",
                 background = factor(as.factor(Pheno8),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Pheno8),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g1 = cell_spec(Geno1,
                 "latex",
                 background = factor(as.factor(Geno1),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno1),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g2 = cell_spec(Geno2,
                 "latex",
                 background = factor(as.factor(Geno2),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno2),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g3 = cell_spec(Geno3,
                 "latex",
                 background = factor(as.factor(Geno3),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno3),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g4 = cell_spec(Geno4,
                 "latex",
                 background = factor(as.factor(Geno4),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno4),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g5 = cell_spec(Geno5,
                 "latex",
                 background = factor(as.factor(Geno5),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno5),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g6 = cell_spec(Geno6,
                 "latex",
                 background = factor(as.factor(Geno6),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno6),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g7 = cell_spec(Geno7,
                 "latex",
                 background = factor(as.factor(Geno7),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno7),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g8 = cell_spec(Geno8,
                 "latex",
                 background = factor(as.factor(Geno8),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno8),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g9 = cell_spec(Geno9,
                 "latex",
                 background = factor(as.factor(Geno9),
                                     c("0","1"),
                                     c("#FFFFFF","#BBBBBB")),
                 color = factor(as.factor(Geno9),
                                c("0","1"),
                                c("#FFFFFF","#BBBBBB"))),
  g10 = cell_spec(Geno10,
                  "latex",
                  background = factor(as.factor(Geno10),
                                      c("0","1"),
                                      c("#FFFFFF","#BBBBBB")),
                  color = factor(as.factor(Geno10),
                                 c("0","1"),
                                 c("#FFFFFF","#BBBBBB"))),
  g11 = cell_spec(Geno11,
                  "latex",
                  background = factor(as.factor(Geno11),
                                      c("0","1"),
                                      c("#FFFFFF","#BBBBBB")),
                  color = factor(as.factor(Geno11),
                                 c("0","1"),
                                 c("#FFFFFF","#BBBBBB")))) %>%
  select(`taxonomic species`,p1,p2,p3,p4,p5,p6,p7,p8,g1,g2,g3,g4,g5,g6,g7,g8,g9,g10,g11) %>%
  kable("latex", 
        escape = F, 
        booktabs = T,
        linesep = "",
        caption = "Correspondence between taxonomic species and best-fit phenogroups and genogroups. Shaded cells show specimens assigned to a particular combination of taxonomic species, best fit phenogroup, and best fit genogroup.") %>%
  row_spec(1:9,
           hline_after = T) %>%
  column_spec(1:20,
              border_right = T) %>%
  column_spec(10,
              border_right = T,
              border_left = T) %>%
  add_header_above(c("","Phenogroups"= 8,"Genogroups"= 11)) %>%
  kable_styling(font_size = 8,
                latex_options = "HOLD_position")
```
Return to [Clade VI Correspondence between taxonomic species and model-based species](#correspondence-between-taxonomic-species-and-model-based-species-5)
\pagebreak

# Figures
## Species Trees
```{r EscalloniaSpeciesTreeTrimmedmakePlots, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}

allEscData = escalloniaSampling
mySppTreeColors = c(brewer.pal(6,'Dark2'),
                    'black')
names(mySppTreeColors) = c("I","II","III","V","VI","IV","outgroup")

# read in phylogeny and root
trimmed = read.tree("supplementary_material_files/SpeciesTrees/miss75.trimmed.concat.contree")
trimmed_rooted = root.phylo(trimmed,
                            outgroup = "FZ99_Valdivia_gayana_Plate1")
#plot.phylo(trimmed_rooted, cex=0.5, align.tip.label = TRUE, show.node.label = TRUE)

# read in species tree and root
trimmed_svd = read.nexus("supplementary_material_files/SpeciesTrees/original.trimmed.miss75.svdquartet.boots.tre")
trimmed_svd_rooted = root.phylo(trimmed_svd,
                                outgroup = "Valdivia_gayana")
#plot.phylo(trimmed_svd_rooted, cex=0.5, align.tip.label = TRUE, show.node.label = TRUE, direction = 'leftwards')

# build sampling list and match clade identity
ggtree(trimmed_rooted)
myTipNames = as.data.frame(get_taxa_name())
colnames(myTipNames) = "originalName"
myTipNames = myTipNames %>%
  separate(originalName,
           into=c('SpecimenID',
                  'toss1',
                  'Taxon',
                  'libraryNumber'),
           remove = FALSE) %>%
  unite(col = 'adjustedName',
        c(toss1,Taxon),
        sep = "_",
        remove = FALSE)

myTipNames$CladeID = allEscData$CladeID[match(myTipNames$Taxon,
                                              allEscData$Taxon)]
myTipNames$CladeID[myTipNames$originalName=="FZ99_Valdivia_gayana_Plate1"] = 'outgroup'

ggtree(trimmed_svd_rooted)
myTipNames2 = as.data.frame(get_taxa_name())
colnames(myTipNames2) = "originalName"
myTipNames2 = myTipNames2 %>%
  separate(originalName,
           into=c('toss1',
                  'Taxon'),
           remove = FALSE) %>%
  unite(col = 'adjustedName',
        c(toss1,Taxon),
        sep = "_",
        remove = FALSE)

myTipNames2$CladeID = allEscData$CladeID[match(myTipNames2$Taxon, allEscData$Taxon)]
myTipNames2$CladeID[myTipNames2$originalName=="Valdivia_gayana"] = 'outgroup'

# plot lineage tree facing species tree; color tips by clade membership
p1 = ggtree(trimmed_rooted) %<+% myTipNames
lineageTreeD = p1$data
subset_labels_lineageTree = lineageTreeD %>%
  filter(node==107 | node==104 | node==64 | node==100 | node==81 | node==68)

p2 = p1 + geom_text(aes(label = adjustedName,
                        color = CladeID),
                    size = 2,
                    hjust = -.15,
                    show.legend = FALSE) +
  ggplot2::xlim(0,0.06) +
  scale_color_manual(values = mySppTreeColors) +
  geom_nodelab(data = subset_labels_lineageTree,
               nudge_x = -0.003,
               nudge_y = 0.75,
               size=3)

p3 = ggtree(trimmed_svd_rooted, branch.length = 'none') %<+% myTipNames2
svdTreeD = p3$data

subset_labels_svdTree = svdTreeD %>%
  filter(node==54 | node==53 | node==47 | node==55 | node==36 | node==34)

p4 = p3 + geom_text(aes(label = adjustedName,
                        color = CladeID),
                    size = 2,
                    hjust = 1.25,
                    show.legend = FALSE) +
  ggplot2::scale_x_reverse(limits = c(25,0)) +
  scale_color_manual(values = mySppTreeColors) +
  geom_nodelab(data=subset_labels_svdTree,
               nudge_x = 0.9,
               nudge_y = 0.5,
               size = 3)

```


### Fig S1: Phylogenetic trees (two specimens per taxonomic species)
```{r EscalloniaSpeciesTreeTrimmedPlot, echo=FALSE, warning=FALSE, message=FALSE, dpi=300, fig.height=6, fig.cap='Phylogenetic trees of \\textit{Escallonia}. Left: Maximum likelihood tree using two specimens per taxonomic species and 364 concatenated loci. Right: Quartet-based species tree. In both trees, colors indicate clades I to VI from bottom to top. Bootstrap support values for focal clades is shown above corresponding bipartition.'}
p2 + p4
```
Return to [Current state of taxonomic species using genomics data](#current-state-of-taxonomic-species-using-genomics-data)
\pagebreak

```{r EscalloniaSpeciesTreeVersion2makePlots, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}

# read in phylogeny and root
version2 = read.tree("supplementary_material_files/SpeciesTrees/miss75.version2.concat.contree")
version2_rooted = root.phylo(version2, outgroup="FZ99_Valdivia_gayana_Plate1")
#plot.phylo(version2_rooted, cex=0.5, align.tip.label = TRUE, show.node.label = TRUE)

# read in species tree and root
version2_svd = read.nexus("supplementary_material_files/SpeciesTrees/trimmedPlus.miss75.svdquartet.boots.tre")
version2_svd_rooted = root.phylo(version2_svd,
                                 outgroup = "Valdivia_gayana")
#plot.phylo(version2_svd_rooted, cex=0.5, align.tip.label = TRUE, show.node.label = TRUE, direction = 'leftwards')

# build sampling list and match clade identity
ggtree(version2_rooted)
myTipNames3 = as.data.frame(get_taxa_name())
colnames(myTipNames3) = "originalName"
myTipNames3 = myTipNames3 %>%
  separate(originalName,
           into=c('SpecimenID',
                  'toss1',
                  'Taxon',
                  'libraryNumber'),
           remove = FALSE) %>%
  unite(col = 'adjustedName',
        c(toss1,Taxon),
        sep = "_",
        remove = FALSE)
myTipNames3$CladeID = allEscData$CladeID[match(myTipNames3$Taxon, allEscData$Taxon)]
myTipNames3$CladeID[myTipNames3$originalName=="FZ99_Valdivia_gayana_Plate1"] = 'outgroup'

ggtree(version2_svd_rooted)
myTipNames4 = as.data.frame(get_taxa_name())
colnames(myTipNames4) = "originalName"
myTipNames4 = myTipNames4 %>%
  separate(originalName,
           into = c('toss1',
                  'Taxon'),
           remove = FALSE) %>%
  unite(col ='adjustedName',
        c(toss1,Taxon),
        sep = "_",
        remove = FALSE)

myTipNames4$CladeID = allEscData$CladeID[match(myTipNames4$Taxon, allEscData$Taxon)]
myTipNames4$CladeID[myTipNames4$originalName=="Valdivia_gayana"] = 'outgroup'

# plot lineage tree facing species tree; color tips by clade membership
p5 = ggtree(version2_rooted) %<+% myTipNames3
lineageTreeV2D = p5$data

subset_labels_lineageV2Tree = lineageTreeV2D %>%
  filter(node==208 | node==201 | node==124 | node==193 | node==155 | node==128)

p6 = p5 + geom_text(aes(label = adjustedName,
                        color = CladeID),
                    size = 2,
                    hjust = -.15,
                    show.legend = FALSE) +
  ggplot2::xlim(0,0.045) +
  scale_color_manual(values = mySppTreeColors) +
  geom_nodelab(data = subset_labels_lineageV2Tree,
               nudge_x = -0.0015,
               nudge_y = 0.9,
               size = 2.5)

p7 = ggtree(version2_svd_rooted,
            branch.length = 'none') %<+% myTipNames4
svdTreeV2D = p7$data
subset_labels_svdV2Tree = svdTreeV2D %>%
  filter(node==52 | node==53 | node==46 | node==54 | node==35 | node==33)

p8 = p7 + geom_text(aes(label = adjustedName,
                        color = CladeID),
                    size = 2,
                    hjust = 1.25,
                    show.legend = FALSE) +
  ggplot2::scale_x_reverse(limits = c(15,0)) +
  scale_color_manual(values=mySppTreeColors) +
  geom_nodelab(data=subset_labels_svdV2Tree,
               nudge_x = 0.6,
               nudge_y = 0.5,
               size = 2.5)

```

### Fig S2: Phylogenetic trees (four specimens per taxonomic species)
```{r EscalloniaSpeciesTreeVersion2Plot, echo=FALSE, warning=FALSE, message=FALSE, dpi=300, fig.height=6, fig.cap='Phylogenetic trees of \\textit{Escallonia}. Maximum likelihood tree using four specimens per taxonomic species and 181 concatenated loci (left). Quartet-based species tree (right). In both trees, colors indicate clades I to VI from bottom to top. Bootstrap support values for focal clades is presented (all other bipartitions were also well-supported)'}
p6 + p8
```
Return to [Current state of taxonomic species using genomics data](#current-state-of-taxonomic-species-using-genomics-data)
\pagebreak

## Clade I
### Fig S3: Taxon sampling
```{r CladeIVennDiagram, echo=FALSE, warning=FALSE, message=FALSE, dpi=300, fig.width=4, fig.align='center', fig.cap="Specimens sampled according to three types of data. Specimens outside the Flowers category represent sterile specimens."}

CladeI_Venn = escalloniaSampling %>%
  filter(CladeID == "I") %>%
  mutate(Flowers = if_else(is.na(`Floral and Leaf`), 0, 1))

CladeI_Venn = CladeI_Venn %>%
  mutate(Leaves = if_else(is.na(`Leaf only`) & is.na(`Floral and Leaf`), 0,1))

CladeI_Venn = CladeI_Venn %>%
  mutate(Genes = if_else(is.na(libraryNumber), 0,1))

CladeI_Venn = CladeI_Venn %>%
  select(Flowers,
         Leaves,
         Genes)

VennForCladeI = venn(CladeI_Venn)

plot(VennForCladeI,
     fills = list(fill = c('#c2a5cf',
                           '#d9f0d3',
                           '#f7f7f7')),
     quantities = list(TRUE,
                       cex = 1.5),
     legend = list(labels,
                   cex = 1))

```
Return to [Clade I Sampling](#sampling-1)
\pagebreak

### Fig S4: Geographic distribution
```{r CladeIsamplingMaps, echo=FALSE, warning=FALSE, message=FALSE, dpi = 300, fig.align='center', fig.cap="Geographic distribution of specimens sampled for this clade according to taxonomic species. Filled symbols indicate specimens used in phenotypic analyses and empty symbols specimens used in genomic analyses."}
 #
myColors = "#cccccc"
myShapes = c(21,22,23,24,25,10)
# plot -------------------------------------------------------------
#
micranthaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'micrantha') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 3,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon=='micrantha') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 2,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21), cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21), cex = 4))) +
  coord_fixed(xlim = c(-100, -55),  
              ylim = c(-30, 20)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face='italic'),
        text = element_text(size = 6)) +
  ggtitle(label='Escallonia micrantha')


millegranaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'millegrana') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 3,
              alpha = 0.8,
              width = 0.5,
              height=0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon=='millegrana') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 2,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21), cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21), cex = 4))) +
  coord_fixed(xlim = c(-100, -55),  
              ylim = c(-30, 20)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face='italic'),
        text = element_text(size = 6)) +
  ggtitle(label='Escallonia millegrana')


micranthaMap + millegranaMap + plot_layout(ncol = 2, guides = 'collect')

```
Return to [Clade I Sampling](#sampling-1)
\pagebreak

```{r currentStateOfTaxonomyCladeIMakePlots, echo=FALSE}

pal = c('#636363')
names(pal) = c('assignment')

# plot
CladeI_propOverlap_All = ggplot(melted_CladeI_overlapProp_All,
                                aes(x = X1,
                                    y = X2)) +
  geom_tile(aes(fill = value)) +
  geom_text(aes(label = round(value,
                              digits = 3)),
            color = 'darkgrey') +
  scale_fill_gradient(low = 'white',
                      high = "black") +
  theme(axis.title = element_blank(),
        axis.text = element_text(size = 6),
        axis.text.x = element_text(angle = -45,
                                   hjust = 0.1)) +
  theme(legend.text = element_text(size = 4),
        legend.margin = margin(2,2,6,2),
        legend.title = element_text(size = 6),
        legend.background = element_rect(color="#cccccc")) +
  guides(fill = guide_colourbar(barwidth = 0.5,
                                barheight = 2))


CladeI_inHullAssignments_All = ggplot(melted_CladeI_hullResults_All,
                                      aes( x = uniqueID,
                                           y = variable,
                                           fill = col)) +
  geom_tile(aes(width = 0.99,
                height = 0.99),
            color = 'black') +
  scale_fill_manual(values = c("assignment" = '#636363',
                               "no assignment" = "white"),
                    name = NULL,
                    labels = c('yes','no'),
                    drop = FALSE) +
  scale_y_discrete(labels = c("In E. micrantha",
                            "In E. millegrana")) +
  scale_x_discrete(limits = factor(1:33)) + # limits equal to number of samples
  theme_bw() +
  theme(panel.ontop = TRUE,
        panel.background = element_rect(fill='transparent'),
        panel.border = element_blank(),
        panel.grid.major.y = element_blank(),
        panel.grid.major.x = element_blank(),
        axis.title.y = element_blank(),
        axis.title.x = element_blank(),
        axis.ticks.x = element_blank(),
        axis.text.x = element_blank(),
        legend.position = 'top',
        text = element_text(size=6),
        plot.margin = margin(1,20,50,10),
        legend.key.height = unit(0.25,'cm'),
        legend.key.width = unit(0.25,'cm')) +
  coord_cartesian(ylim = c(1.1,2),
                  clip = 'off') +
  annotate(geom = 'text',
           x = (11/2),
           y = 0.1,
           label = 'As E. micrantha',
           size = 2,
           angle = 320,
           hjust=0) +
  annotate(geom='text',
           x = (12 + (33-11)/2),
           y = 0.1,
           label = 'As E. millegrana',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom='segment',
           x = 1,
           xend = 11,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom='segment',
           x = 12,
           xend = 33,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75)

```

### Fig S5: Current state of taxonomic species with phenotypic data
```{r currentStateOfTaxonomyCladeIPlots, fig.width=7, fig.height = 5, dpi=200, echo=FALSE, fig.cap="Assessment of current state of taxonomic species with phenotypic data. Top panel: Pairwise overlap among 10-cubes describing geometrically each taxonomic species in 10-dimensional phenospace. Bottom panel: Matching-prediction analysis with each cell along the x-axis representing specimens sorted according to taxonomic species and the 10-cubes corresponding to each taxonomic species along the y-axis. If a specimen matches the prediction of the monograph (i.e., it is inside a 10-cube), the corresponding cell is shaded. If the specimen does not match the prediction, the cell is empty."}

(((CladeI_propOverlap_All + plot_spacer())/ CladeI_inHullAssignments_All) + plot_layout(tag_level='new'))

```
Return to [Clade I Current state of taxonomic species](#current-state-of-taxonomic-species)
\pagebreak

### Fig S6: Phenogroup delimitation: Gaussian finite mixture modeling
```{r cladeImorphologicalDelimitationPlots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=6.5, dpi=300, fig.cap='Gaussian finite mixture modeling (GFMM) for phenogroup delimitation and model selection using the Bayesian information criterion (BIC). Top panel: empirical support (ordinate) for Gaussian mixture models (GMM) assuming distinct number of phenogroups (abscissa). Each GMM specifies different number of phenogroups (shapes). Empirical support was measured as difference in BIC relative to the best model ($\\Delta$BIC = $0$). Bottom panel: Visualization of the phenogroups (shapes) identified by the best fit GMM; left panel shows phenogroups in the space defined by two axes obtained by linear discriminat analysis (to maximize separation and visualization), middle panel shows phenogroups in the space defined by discriminant axis 1 and latitude, and right panel shows phenogroups in the space defined by discriminant axis 1 and elevation.'}

CladeI_modelsRanked = read.csv("supplementary_material_files/CladeI_resultsFiles/CladeI_modelsRanked.csv")

CladeI_morphDR = read.csv("supplementary_material_files/CladeI_resultsFiles/CladeI_morphDR.csv")

max.BIC = 54.03099 #recorded during analysis

#plot
modelsRanked = ggplot(CladeI_modelsRanked,
                      aes(x = numMorphGroups,
                          y = deltaBIC_naive)) +  
  geom_point(aes(x = numMorphGroups,
                 y = deltaBIC_naive,
                 shape = "A_naive_models"),
             cex = 3,
             col = "black",
             bg = 'white') +
  geom_point(aes(x = numMorphGroups[order(deltaBIC_naive)[1]],
                 y = deltaBIC_naive[order(deltaBIC_naive)[1]],
                 shape = "B_best_naive_model"),
             cex = 3.5,
             col = 'black',
             bg = 'black') +
  # geom_point(aes(x=numMorphGroups[order(deltaBIC_naive)[2]],
  #                y=deltaBIC_naive[order(deltaBIC_naive)[2]],
  #                shape ="C_second_naive_model"), cex=3.5, col='black',bg='grey') +
  geom_point(aes(x = numMorphGroups[taxonBIC != 'NA'],
                 y = max.BIC-taxonBIC[taxonBIC != 'NA'],
                 shape = "D_currentTax"),
             cex = 2.5,
             col = 'black',
             bg = 'red') +
  geom_point(aes(x = numMorphGroups[morphoGroupsBIC != 'NA'],
                 y = max.BIC-morphoGroupsBIC[morphoGroupsBIC != 'NA'],
                 shape = "E_morphoGroups"),
             cex = 2.5,
             col = 'black',
             bg = 'red') +
  scale_shape_manual(values = c(21,21,25,24),
                     labels = c("Best fit model",
                              "Naive models",
                              "Taxonomy model",
                              "Taxonomy unaware model"),
                     name = NULL) +
  guides(shape = guide_legend(override.aes = list(shape = c(21,21,25,24),
                                                  bg = c('black','white','red','red'),
                                                  color = c('black','black','black','black'),
                                                  size = 3))) +
  scale_x_discrete(limits=c(1,2,3)) +
  theme_bw() +
  theme(panel.grid = element_blank()) +
  xlab('Number of phenogroups') +
  ylab(expression(paste("Empirical support (", Delta, "BIC)", sep="")))

# plot ordination of best naive model
myShapes = c(21,22)
Q2 = ggplot(CladeI_morphDR) +
  geom_boxplot(aes(x = MclustDR_Dir1,
                   y = factor(1)),
               width = 0.001,
               lwd = 0) +
  geom_jitter(aes(x = MclustDR_Dir1,
                  y = factor(1),
                  shape = as.factor(BestModel_Classification)),
              color = 'black',
              bg = '#cccccc',
              size = 2,
              stroke = 0.25,
              width = 0,
              height = 0.03) +
  scale_shape_manual(values = myShapes,
                    name = 'Classification',
                    labels = c('group1', 'group2')) +
  xlab("Dimension 1") +
  ylab("Dimension 2") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10),
        axis.ticks.y = element_blank()) +
  guides(shape = guide_legend(override.aes = list(size = 2)))


Q3 = ggplot(CladeI_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = Latitude,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group1', 'group2')) +
  xlab("Dimension 1") +
  ylab("Latitude") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color ='black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

Q4 = ggplot(CladeI_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = Elevation,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group1', 'group2')) +
  xlab("Dimension 1") +
  ylab("Elevation") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

(modelsRanked + plot_spacer() + plot_layout(widths = c(3,1))) /
  (Q2 + Q3 + Q4 + plot_layout(guides = 'collect'))

```
Return to [Clade I Phenomics: model-based species discovery](#model-based-species-discovery)
\pagebreak

```{r, CladeIgenomicSensitivityAnalyses_largest, echo=FALSE, warning = FALSE, message=FALSE, include=FALSE}

# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeI_resultsFiles/CladeI_largestMiss25_concat.contree")
load("supplementary_material_files/CladeI_resultsFiles/CladeI_largestMiss25_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeI_resultsFiles/CladeI_largestMiss25_ADMIX.2.Q")
testData = read_delim("supplementary_material_files/CladeI_resultsFiles/CladeI_largestMiss25_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeI_resultsFiles/CladeI_popFile_withLocality_ADMIXordered.csv", header=TRUE)

sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
   unite(col = 'SampleName',
         c(SpecimenID,
           toss1,
           taxon),
         sep="_",
         remove = FALSE) %>%
   select(MolecularSampleName,
          SampleName,
          SpecimenID,
          libraryNumber,
          taxon,
          country,
          Elevation,
          Latitude,
          Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)
plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)
myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:11)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeIgenomicSensitivityAnalyses_largest2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}
#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(brewer.pal(2, 'Greys'))

plot1 = ggtree(myTree_midRoot)

plot2 = plot1 +
  geom_tiplab(geom = 'text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot3 = facet_plot(plot2,
                   data = admixMelt,
                    panel = 'Admixture proportions',
                    geom = geom_barh,
                    mapping = aes(x = as.numeric(value),
                                  y = as.character(label),
                                  fill = variable),
                    color = 'grey90',
                    stat = 'identity') +
  xlim_tree(0.02) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")

plot4 = facet_widths(plot3,
                     c(6,3))

#########################
#####      PCA      #####
#########################
pch_v = c(21,22)
fastPCA_largest = ggplot(tab,
                         aes(x = EV1,
                             y = EV2,
                             shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.micrantha","E.millegrana"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size = unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_largest = as_tibble(testData %>%
                            dplyr::group_by(clusterNum) %>%
                            summarise_at("error", list(mean, sd)))

colnames(step1_largest) = c("clusterNum", "mean", "stdev")

step1_largest$me = sapply(step1_largest$stdev,
                          function(x) qt(.95,9)*x/sqrt(10))

testCI_largest = mutate(as_tibble(step1_largest),
                        lower = mean-me,
                        upper = mean+me)

ADMIX_CI_largest = ggplot(testCI_largest,
                          aes(x = clusterNum,
                              y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:3)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

#plot4 + (ADMIX_CI_largest / fastPCA_largest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation( title = "Largest data matrix: ~75% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))


```

```{r, CladeIgenomicSensitivityAnalyses_middle, echo=FALSE, warning = FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeI_resultsFiles/CladeI_middleMiss50_concat.contree")
load("supplementary_material_files/CladeI_resultsFiles/CladeI_middleMiss50_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeI_resultsFiles/CladeI_middleMiss50_ADMIX.2.Q")
testData = read_delim("supplementary_material_files/CladeI_resultsFiles/CladeI_middleMiss50_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeI_resultsFiles/CladeI_popFile_withLocality_ADMIXordered.csv", header=TRUE)
sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep = "_",
        remove = FALSE) %>%
  select(MolecularSampleName,
         SampleName,
         SpecimenID,
         libraryNumber,
         taxon,
         country,
         Elevation,
         Latitude,
         Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:11)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeIgenomicSensitivityAnalyses_middle2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}
#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(brewer.pal(2, 'Greys'))  

plot5 = ggtree(myTree_midRoot)

plot6 = plot5 +
  geom_tiplab(geom ='text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot7 = facet_plot(plot6,
                   data = admixMelt,
                   panel = 'Admixture proportions',
                   geom = geom_barh,
                   mapping = aes(x = as.numeric(value),
                                 y = as.character(label),
                                 fill = variable),
                   color = 'grey90',
                   stat = 'identity') +
  xlim_tree(0.02) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")

plot8 = facet_widths(plot7,
                     c(6,3))

#########################
#####      PCA      #####
#########################
pch_v = c(21,22)
fastPCA_middle = ggplot(tab,
                        aes(x = EV1,
                            y = EV2,
                            shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.micrantha", "E.millegrana"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size=unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_middle = as_tibble(testData %>%
                           dplyr::group_by(clusterNum) %>%
                           summarise_at("error",
                                        list(mean, sd)))

colnames(step1_middle) = c("clusterNum", "mean", "stdev")

step1_middle$me = sapply(step1_middle$stdev,
                         function(x) qt(.95,9)*x/sqrt(10))

testCI_middle = mutate(as_tibble(step1_middle),
                       lower = mean-me,
                       upper = mean+me)

ADMIX_CI_middle = ggplot(testCI_middle,
                         aes(x = clusterNum,
                             y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:3)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

#plot8 + (ADMIX_CI_middle / fastPCA_middle) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(title = "Middle-sized data matrix: ~50% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))


```

```{r, CladeIgenomicSensitivityAnalyses_smallest, echo=FALSE, warning = FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeI_resultsFiles/CladeI_smallestMiss75_concat.contree")
load("supplementary_material_files/CladeI_resultsFiles/CladeI_smallestMiss75_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeI_resultsFiles/CladeI_smallestMiss75_ADMIX.2.Q")
testData = read_delim("supplementary_material_files/CladeI_resultsFiles/CladeI_smallestMiss75_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeI_resultsFiles/CladeI_popFile_withLocality_ADMIXordered.csv", header=TRUE)

sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                  'toss1',
                  'taxon',
                  'libraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep = "_",
        remove = FALSE) %>%
   select(MolecularSampleName,
          SampleName,
          SpecimenID,
          libraryNumber,
          taxon,
          country,
          Elevation,
          Latitude,
          Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:11)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeIgenomicSensitivityAnalyses_smallest2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}

#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(brewer.pal(2, 'Greys'))  

plot9 = ggtree(myTree_midRoot)

plot10 = plot9 +
  geom_tiplab(geom = 'text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot11 = facet_plot(plot10,
                    data = admixMelt,
                    panel = 'Admixture proportions',
                    geom = geom_barh,
                    mapping = aes(x = as.numeric(value),
                                  y = as.character(label),
                                  fill = variable),
                    color = 'grey90',
                    stat = 'identity') +
  xlim_tree(0.02) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")
plot12 = facet_widths(plot11,
                      c(6,3))

#########################
#####      PCA      #####
#########################
pch_v = c(21,22)
fastPCA_smallest = ggplot(tab,
                          aes(x = EV1,
                              y = EV2,
                              shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.micrantha","E.millegrana"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size = unit(0.25, 'cm')) +
  xlab('Eigenvector 1') + 
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_smallest = as_tibble(testData %>%
                             dplyr::group_by(clusterNum) %>%
                             summarise_at("error",
                                          list(mean, sd)))

colnames(step1_smallest) = c("clusterNum",
                             "mean",
                             "stdev")
step1_smallest$me = sapply(step1_smallest$stdev,
                           function(x) qt(.95,9)*x/sqrt(10))

testCI_smallest = mutate(as_tibble(step1_smallest),
                         lower = mean-me,
                         upper = mean+me)

ADMIX_CI_smallest = ggplot(testCI_smallest,
                           aes(x = clusterNum,
                               y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:3)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

#plot12 + (ADMIX_CI_smallest / fastPCA_smallest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(title = "Smallest data matrix: ~25% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))

```

### Fig S7: Sensitivity tests with $75\%$ missing data
```{r, cladeIgenomicSensitivityAnalysesLargestplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($75\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot4 +
  (ADMIX_CI_largest / fastPCA_largest) +
  plot_layout(widths = c(4,2),
              heights = c(50,0.5)) +
  plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade I Genomics: sensitivity tests](#sensitivity-tests)
\pagebreak

### Fig S8: Sensitivity tests with $50\%$ missing data
```{r, cladeIgenomicSensitivityAnalysesMiddleplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($50\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot8 +
  (ADMIX_CI_middle / fastPCA_middle) +
  plot_layout(widths = c(4,2),
              heights = c(50,0.5)) +
  plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade I Genomics: sensitivity tests](#sensitivity-tests)
\pagebreak

### Fig S9: Sensitivity tests with $25\%$ missing data
```{r, cladeIgenomicSensitivityAnalysesSmallestplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($25\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot12 +
  (ADMIX_CI_smallest / fastPCA_smallest) +
  plot_layout(widths = c(4,2),
              heights = c(50,0.5)) +
  plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade I Genomics: sensitivity tests](#sensitivity-tests)
\pagebreak

### Fig S10: Genogroup delimitation: Genotypic cluster model
```{r cladeImolecularDelimitationsNmms, echo=FALSE, message=FALSE, fig.width=7, fig.height=2.5, dpi=300, fig.cap='Gaussian finite mixture modeling (GFMM) for genogroup delimitation. Visualization of the genogroups (shades) identified by the best fit Gaussian mixture model (GMM). A) genogroups in the space defined by two axes obtained by non-metric multidimensional scaling (NMDS); B) genogroups in the space defined by NMDS axis 1 and latitude; C) genogroups in the space defined by NMDS axis 1 and elevation.'}

# read in results of NMM delimitation
CladeI_nmms_temp = read.csv("supplementary_material_files/CladeI_resultsFiles/CladeI_smallestMiss75_NMMs.csv")
# select classification and dimensions 1 and 2 from best model
CladeI_nmms = CladeI_nmms_temp %>%
  select(SpecimenID, 
         Latitude, 
         Longitude, 
         Elevation, 
         class.dim2.n0, 
         NMDS1.dim2.n0, 
         NMDS2.dim2.n0)

colnames(CladeI_nmms)[5:7] = c('Classification','Dim1','Dim2')
# melt in prep for plotting
CladeI_nmms_melted = reshape::melt(CladeI_nmms, by='SpecimenID')

myColors = c('#f0f0f0','#bdbdbd','#636363')
# plot groups in first two dimensions
p1 = ggplot(CladeI_nmms, aes(x = Dim1, 
                             y = Dim2)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch=21,
              height = 0.05,
              width = 0.05,
              color = 'black',
              size = 3) +
  scale_fill_manual(values = myColors, name='Classification',
                     labels = c('group 1','group 2', 'group 3')) +
  xlab('Dimension 1') +
  ylab('Dimension 2') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))
# plot groups in dimension 1 against latitude
p2 = ggplot(CladeI_nmms,
             aes(x = Dim1,
                 y = Latitude)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              size = 3,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1','group 2', 'group 3')) +
  xlab('Dimension 1') +
  ylab('Latitude') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))
# plot groups in dimension1 against elevation
p3 = ggplot(CladeI_nmms,
             aes(x = Dim1,
                 y = Elevation)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              size = 3,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1','group 2', 'group 3')) +
  xlab('Dimension 1') +
  ylab('Elevation') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))

p1 + p2 + p3 + plot_layout(guides = 'collect') + plot_annotation(tag_levels = 'A') & theme(plot.tag = element_text(size = 8))

```
Return to [Clade I Genomics: model-based species discovery](#model-based-species-discovery-1)
\pagebreak

```{r cladeImolecularDelimitationsPhylogeny, echo=FALSE, include=FALSE}

myTree_midRoot = paintSubTree(myTree_midRoot,
                              node = c(16),
                              state = '2',
                              stem = FALSE)

myTree_midRoot = paintSubTree(myTree_midRoot,
                              node = c(25),
                              state = '2',
                              stem = FALSE)
```

### Fig S11: Genogroup delimitation. Cladogenesis to anagenesis model
```{r cladeImolecularDelimitationsPhylogeny2, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, out.height='.6\\textwidth', dpi=300, fig.show='asis', fig.cap='Phylogenetic modeling for genogroup delimitation. Midpoint-rooted phylogenetic tree showing genogroups in red. Values correspond to nodes at the transition point between cladogenesis (between species) to anagenesis (within species). Values closer to 0 indicate that the node was identified as a transition to anagenesis summarized over $500$ delimitations.'}
# at this point, the 'myTree_midRoot' is now of class 'simmap', so the plotting is a bit different
# because we want our trees without tip labels, i've set the fsize to be very small (essentially label-less)
# after plotting the tree with the 'painted' clades, add the custom node labels
colors = setNames(c('black','red'),
                  c('1','2'))

plotSimmap(myTree_midRoot,
           setEnv = TRUE,
           fsize = 0.001,
           lwd = 1,
           colors = colors)
nodelabels(text = '0.08',
           node = 16,
           frame = 'none',
           adj = c(1.5,-0.50),
           cex = 1)
nodelabels(text = '0.41',
           node = 25,
           frame = 'none',
           adj = c(1.5,-0.50),
           cex = 1)
# internal nodes

```
Return to [Clade I Genomics: model-based species discovery](#model-based-species-discovery-1)
\pagebreak

### Fig S12: Genogroup delimitation: Reproductive isolation model
```{r cladeImolecularDelimitationPopGenBpp, echo=FALSE, fig.width=6.5, dpi=300, fig.cap='Population genetic modeling for genogroup delimitation. Top panel: assignment of specimens to demes according to \\texttt{STRUCTURE} and posterior probability of species delimitation modeling according to \\texttt{BPP} using these demes. Bottom panel: assignment of specimens to demes according to \\texttt{MAVERICK} and posterior probability of species delimitation modeling according to \\texttt{BPP} using these demes. Specimens are sorted from north (left) to south (right) according to locality of collection.'}

mySTR = read.csv("supplementary_material_files/CladeI_resultsFiles/CladeI_smallestMiss75_STRUCTURE.csv", header=TRUE)
myRMav = read.csv("supplementary_material_files/CladeI_resultsFiles/CladeI_smallestMiss75_RMav.csv", header=TRUE)


# attach assignments
# IMPORTANT: make sure to confirm number of columns you need to attach (corresponds with number of groups to be plotted)
# number of STRUCTURE groups = 2
# number of RMaverick groups = 2
mergedGroupInfo = cbind(sampleInfo, mySTR[match(sampleInfo$MolecularSampleName, mySTR[,'MolecularSampleName']),2:3])
mergedGroupInfo = cbind(mergedGroupInfo, myRMav[match(sampleInfo$MolecularSampleName, myRMav[,'MolecularSampleName']),2:3])

# determine majority group assignment for each individual and attach these to the group information
# STRUCTURE groups are columns 11:12
# RMaverick groups are columns 13:15
mergedGroupInfo$STRassignment = as.factor(colnames(mergedGroupInfo[,10:11]) [apply(mergedGroupInfo[,10:11],1,which.max)])
mergedGroupInfo$RMavassignment = as.factor(colnames(mergedGroupInfo[,12:13]) [apply(mergedGroupInfo[,12:13],1,which.max)])

myGreys = c("#f7f7f7","#cccccc")
myBlues = c("#eff3ff","#bdd7e7")
STRplot = reshape::melt(mergedGroupInfo,
                        id.vars = c("SpecimenID","Latitude"),
                        measure.vars = c("group1","group2")) %>%
  ggplot(aes(x = reorder(as.factor(SpecimenID),
                       -Latitude),
             y = value,
             fill = as.factor(variable))) +
  geom_bar(stat = 'identity',
           show.legend = FALSE,
           color = 'black') +
  scale_fill_manual(values = myGreys[c(1,2)]) +
  ylab("Probability") +
  theme_bw() +
  #theme(axis.text.x = element_text(size=8, angle=90, vjust = 0.5, hjust=1)) +
  theme(axis.text.x = element_blank(),
        axis.title.x = element_blank()) +
  theme(text = element_text(size = 10))

RMavplot = reshape::melt(mergedGroupInfo,
                         id.vars=c("SpecimenID","Latitude"),
                         measure.vars=c("deme1","deme2")) %>%
  ggplot(aes(x = reorder(as.factor(SpecimenID),
                         -Latitude),
             y = value,
             fill = as.factor(variable))) +
  geom_bar(stat = 'identity',
           show.legend = FALSE,
           color = 'black') +
  scale_fill_manual(values = myBlues[c(1,2)]) +
  ylab("Probability") +
  xlab("Specimens sorted by Latitudinal position") +
  theme_bw() +
  theme(axis.text.x = element_text(size = 4,
                                   angle = 90,
                                   vjust = 0.5,
                                   hjust = 1)) +
  theme(text = element_text(size = 10))
  #theme(axis.text.x = element_blank(), axis.title.x = element_blank())

(STRplot + grid::textGrob("BPP post. probability\n for 2 species = 1.0") + plot_layout(widths = c(3,1))) / (RMavplot + grid::textGrob("BPP post. probability\n for 2 species = 1.0") + plot_layout(widths = c(3,1)))
```
Return to [Clade I Genomics: model-based species discovery](#model-based-species-discovery-1)
\pagebreak

```{r cladeIfinalDelimitation, echo=FALSE, include=FALSE}
# need to use the output of the morphometrics analysis, since it only includes samples that have BOTH flowers and leaves
morphDelim = read.csv("supplementary_material_files/CladeI_resultsFiles/CladeI_morphDR.csv")

molecularDelim = read.csv("supplementary_material_files/CladeI_resultsFiles/CladeI_molecularDelimitation.csv")
# if it doesn't have a SpecimenID column, you'll need to make one
molecularDelim = molecularDelim %>%
  separate(col = SampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'LibraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep = "_",
        remove = FALSE) %>%
  unite(col = 'MolecularSampleName',
        c(SpecimenID,
          toss1,
          taxon,
          LibraryNumber),
        sep = "_",
        remove = FALSE)

# join datasets, coalesce duplicate columns, select needed data
typesOfSpp_temp <- full_join(morphDelim,
                             molecularDelim,
                             by = "SpecimenID") %>%
  mutate(
    Latitude = coalesce(Latitude.x, Latitude.y),
    Longitude = coalesce(Longitude.x, Longitude.y),
    Elevation = coalesce(Elevation.x, Elevation.y)
  ) %>%
  select(MolecularSampleName,
         SpecimenID,
         Elevation,
         Latitude,
         Longitude,
         BestModel_Classification, #name of the morphoDelim column
         NMMgroups) # name of the molecularDelim column
# confirmation
#colnames(typesOfSpp_temp)
#view(typesOfSpp_temp) # confirm number of rows matches that of sampling table


# create plotting df
typesOfSpp = typesOfSpp_temp
colnames(typesOfSpp) = c("MolecularSampleName",
                         "SpecimenID",
                         "Elevation",
                         "Latitude",
                         "Longitude",
                         "PhenoGroups",
                         "GenoGroups")


# plot tree and match dataframe to tip names - ensures order of plotting is correct
myTree = read.tree("supplementary_material_files/CladeI_resultsFiles/CladeI_smallestMiss75_concat.contree")
myTree_midRoot = midpoint.root(myTree)
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()
typesOfSpp = typesOfSpp[order(match(typesOfSpp$MolecularSampleName, myTipNames)),]

```

### Fig S13: Data integration
```{r cladeIfinalDelimitation2, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, fig.height=5, dpi=300, fig.cap="Integration of phenotypic and genomic data with spatial information and evolutionary history. All specimens are assigned to their corresponding best fit phenogroup (shapes) and genogroup (colors). Specimens without phenotypic or genomic data (unknown specimens) are shown as asterisks and empty shapes, accordingly. Specimens are shown as tips of the maximum likelihood tree (left) used in the CA model analysis and mapped along latitude and elevation (right). Specimens assigned to a single phenogroup and a single genogroup delineate species that we determined as 'good species'. Specimens assigned to a single phenogroup across multiple genogroups delineate species that we determined as 'phenotypic cryptic species'. Specimens assigned to a single genogroup across multiple phenogroups delineate species that we determined as 'genetic cryptic species'."}
########################
# Phylogenetic context # ----------------------------------------------------------------------------------------
########################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups

# get number of colors and shapes that you'll need for plotting
#unique(typesOfSpp$PhenoGroups)
#unique(typesOfSpp$GenoGroups)

# set values for colors and shapes
myShapes = c(21,22,8) #the 8 is for unknown
names(myShapes) = c('1','2','NA')
myCols = c(brewer.pal(n = 3,'Dark2'),"#ffffff")
names(myCols) = c('group1','group2','group3','NA')



f = ggtree(myTree_midRoot) %<+% typesOfSpp + xlim_tree(0.025)
#f
f2 = f +
  geom_tippoint(aes(x = x,
                    fill = as.factor(GenoGroups),
                    shape = as.factor(PhenoGroups)),
                size = 3) +
  scale_shape_manual(values = myShapes) +
  scale_fill_manual(values = myCols) +
  geom_tiplab(aes(label = SpecimenID),
              hjust = -0.3,
              size = 2) +
  geom_point2(aes(subset = is.na(PhenoGroups) & isTip,
                  color = as.factor(GenoGroups)),
              pch = 8,
              size = 2,
              stroke = 0.5,
              alpha = 0.5) +
  scale_color_manual(values = myCols) +
  theme(legend.position = "none")
#f2


###################
# Spatial context # ----------------------------------------------------------------------------------------
###################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups


##### plot individuals with GenoGroup assignment, but no PhenoGroup assignment
g = ggplot(typesOfSpp,
           aes(x = Latitude,
               y = Elevation)) +
  geom_jitter(data = typesOfSpp %>%
                filter(is.na(PhenoGroups)),
              aes(x = Latitude,
                  y = Elevation,
                  color = as.factor(GenoGroups)),
              size = 2,
              alpha = 0.5,
              width = 1.5,
              height = 1.5,
              pch = 8,
              stroke = 1) +
  scale_color_manual(values = myCols) +
  scale_x_reverse() +
  theme_bw() +
  theme(legend.position = 'none') +
  theme(panel.grid.major = element_line(size = 0.55,
                                        color = '#f0f0f0'))
#g

##### plot individuals with PhenoGroup assignment, but no GenoGroup assignment
g2 = g + geom_jitter(data = typesOfSpp %>%
                       filter(is.na(GenoGroups)),
                     aes(x = Latitude,
                         y = Elevation,
                         shape = as.factor(PhenoGroups)),
                     fill = 'white',
                     size = 2,
                     width = 1.5,
                     height = 1.5) +
  scale_shape_manual(values = myShapes)
#g2

###### plots fully overlapping data (has both Geno- and Pheno- group assignments)
g3 = g2 + geom_jitter(data = typesOfSpp %>%
                        filter(GenoGroups != 'NA' & PhenoGroups != 'NA'),
                      aes(x = Latitude,
                          y = Elevation,
                          fill = as.factor(GenoGroups),
                          shape = as.factor(PhenoGroups)),
                      size = 3,
                      width = 1.5,
                      height = 1.5) +
  scale_fill_manual(values = myCols)
#g3

###### create the plot and save to an object
plotForLegend = ggplot(typesOfSpp,
                       aes(x = Latitude,
                           y = Elevation,
                           fill = as.factor(GenoGroups),
                           shape = as.factor(PhenoGroups))) +
  geom_point() +
  scale_fill_manual(values = myCols,
                    name = 'Genogroups',
                    labels = c('1','2','3','unknown')) +
  scale_shape_manual(values = myShapes,
                     name = 'Phenogroups',
                     labels = c('1','2','unknown')) +
  guides(fill = guide_legend(override.aes = list(fill = c(myCols[1:3],
                                                        'white'),
                                                 size = 3,
                                                 shape = 22,
                                                 color = 'black'))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21,22,8),
                                                  size = 3))) +
  theme(legend.key = element_rect(fill = NA),
        text = element_text(size = 8))

legend = cowplot::get_legend(plotForLegend)

#g3 + legend


##########
# GK TAU # ----------------------------------------------------------------------------------------
##########
fullOverlap = typesOfSpp %>%
  filter(PhenoGroups != 'NA' & GenoGroups != 'NA')

gktau_morph2molecules = GKtau(fullOverlap$PhenoGroups,
                              fullOverlap$GenoGroups)
gktau_molecules2morph = GKtau(fullOverlap$GenoGroups,
                              fullOverlap$PhenoGroups)


####################################
# PLOT both together # --------------------------------------------
####################################
# one row
f2 + g3 + legend + plot_layout(widths = c(4,3,1))


```
Return to [Clade I Data integration](#data-integration-1)
\pagebreak

## Clade II
### Fig S14: Taxon sampling
```{r CladeIIVennDiagram, echo=FALSE, dpi=300, fig.width=4, fig.align='center', fig.cap="Specimens sampled according to three types of data. Specimens outside the Flowers category represent sterile specimens."}

CladeII_Venn = escalloniaSampling %>%
  filter(CladeID == "II") %>%
  mutate(Flowers = if_else(is.na(`Floral and Leaf`), 0, 1))

CladeII_Venn = CladeII_Venn %>%
  mutate(Leaves = if_else(is.na(`Leaf only`) & is.na(`Floral and Leaf`), 0,1))

CladeII_Venn = CladeII_Venn %>%
  mutate(Genes = if_else(is.na(libraryNumber), 0,1))

CladeII_Venn = CladeII_Venn %>%
  select(Flowers,Leaves, Genes)

VennForCladeII = venn(CladeII_Venn)

plot(VennForCladeII,
     fills = list(fill = c('#c2a5cf','#d9f0d3','#f7f7f7')),
     quantities = list(TRUE,
                       cex = 1.5),
     legend=list(labels,
                 cex = 1))

```
Return to [Clade II Sampling](#sampling-2)
\pagebreak

### Fig S15: Geographic distribution
```{r CladeIIsamplingMaps, echo=FALSE, dpi = 300, fig.align='center', fig.cap="Geographic distribution of specimens sampled for this clade according to taxonomic species. Filled symbols indicate specimens used in phenotypic analyses and empty symbols specimens used in genomic analyses."}

myColors = "#cccccc"
myShapes = c(21,22,23,24,25,10)
herreraeMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'herrerae') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 3,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon=='herrerae') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 2,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-100, -55),  
              ylim = c(-30, 20)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia herrerae')

pendulaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'pendula') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 3,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon=='pendula') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 2,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values=myShapes[1],
                     name='Phenotype',
                     labels = NULL) +
  scale_fill_manual(values='white',
                    name='Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-100, -55),
              ylim = c(-30, 20)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia pendula')

herreraeMap + pendulaMap + plot_layout(ncol = 2, guides='collect')

```
Return to [Clade II Sampling](#sampling-2)
\pagebreak

```{r currentStateOfTaxonomyCladeIIMakePlots, echo=FALSE}

pal = c('#636363')
names(pal) = c('assignment')

# plot
CladeII_propOverlap_All = ggplot(melted_CladeII_overlapProp_All,
                                aes(x = X1,
                                    y = X2)) +
  geom_tile(aes(fill = value)) +
  geom_text(aes(label = round(value,digits = 3)),
            color = 'darkgrey') +
  scale_fill_gradient(low = 'white',
                      high = "black") +
  theme(axis.title = element_blank(),
        axis.text = element_text(size = 6),
        axis.text.x = element_text(angle = -45,
                                   hjust = 0.1)) +
  theme(legend.text = element_text(size = 4),
        legend.margin = margin(2,2,6,2),
        legend.title = element_text(size = 6),
        legend.background = element_rect(color = "#cccccc")) +
  guides(fill = guide_colourbar(barwidth = 0.5, barheight = 2))


CladeII_inHullAssignments_All = ggplot(melted_CladeII_hullResults_All,
                                      aes(x = uniqueID,
                                          y = variable,
                                          fill = col)) +
  geom_tile(aes(width = 0.99,
                height = 0.99),
            color = 'black') +
  scale_fill_manual(values = c("assignment" = '#636363',
                               "no assignment" = "white"),
                    name = NULL,
                    labels = c('yes','no'),
                    drop = FALSE) +
  scale_y_discrete(labels = c("In E. herrerae",
                            "In E. pendula")) +
  scale_x_discrete(limits = factor(1:33)) + # limits equal to number of samples
  theme_bw() +
  theme(panel.ontop = TRUE,
        panel.background = element_rect(fill = 'transparent'),
        panel.border = element_blank(),
        panel.grid.major.y = element_blank(),
        panel.grid.major.x = element_blank(),
        axis.title.y = element_blank(),
        axis.title.x = element_blank(),
        axis.ticks.x = element_blank(),
        axis.text.x = element_blank(),
        legend.position = 'top',
        text = element_text(size = 6),
        plot.margin = margin(1,20,50,10),
        legend.key.height = unit(0.25,'cm'),
        legend.key.width = unit(0.25,'cm')) +
  coord_cartesian(ylim = c(1.1,2),
                  clip = 'off') +
  annotate(geom ='text',
           x = (5/2),
           y = 0.1,
           label = 'As E. herrerae',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (6 + (33-6)/2),
           y = 0.1,
           label = 'As E. pendula',
           size= 2,
           angle= 320,
           hjust = 0) +
  annotate(geom='segment',
           x = 1,
           xend = 5,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 6,
           xend = 33,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75)

```

### Fig S16: Current state of taxonomic species with phenotypic data
```{r currentStateOfTaxonomyCladeIIPlots, fig.width=7, fig.height = 5, dpi=200, echo=FALSE, fig.cap="Assessment of current state of taxonomic species with phenotypic data. Top panel: Pairwise overlap among 10-cubes describing geometrically each taxonomic species. Bottom panel: Matching-prediction analysis with each cell along the x-axis representing specimens sorted according to taxonomic species and the 10-cubes corresponding to each taxonomic species along the y-axis. If a specimen matches the prediction of the monograph (i.e., it is inside a 10-cube), the corresponding cell is shaded. If the specimen does not match the prediction, the cell is empty."}

(((CladeII_propOverlap_All + plot_spacer())/ CladeII_inHullAssignments_All) + plot_layout(tag_level='new'))

```
Return to [Clade II Current state of taxonomic species](#current-state-of-taxonomic-species-1)
\pagebreak

### Fig S17: Phenogroup delimitation: Gaussian finite mixture modeling
```{r cladeIImorphologicalDelimitationPlots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=6.5, dpi=300, fig.cap='Gaussian finite mixture modeling (GFMM) for phenogroup delimitation and model selection using the Bayesian information criterion (BIC). Top panel: empirical support (ordinate) for Gaussian mixture models (GMM) assuming distinct number of phenogroups (abscissa). Each GMM specifies different number of phenogroups (shapes). Empirical support was measured as difference in BIC relative to the best model ($\\Delta$BIC = $0$). Bottom panel: Visualization of the phenogroups (shapes) identified by the best fit GMM; left panel shows phenogroups in the space defined by two axes obtained by linear discriminat analysis (to maximize separation and visualization), middle panel shows phenogroups in the space defined by discriminant axis 1 and latitude, and right panel shows phenogroups in the space defined by discriminant axis 1 and elevation.'}

CladeII_modelsRanked = read.csv("supplementary_material_files/CladeII_resultsFiles/CladeII_modelsRanked.csv")
CladeII_morphDR = read.csv("supplementary_material_files/CladeII_resultsFiles/CladeII_morphDR.csv")

max.BIC = 71.72976 # recorded during analysis

#plot
modelsRanked = ggplot(CladeII_modelsRanked,
                      aes(x = numMorphGroups,
                          y = deltaBIC_naive)) +
  geom_point(aes(x = numMorphGroups,
                 y = deltaBIC_naive,
                 shape = "A_naive_models"),
             cex = 3,
             col = "black",
             bg = 'white') +
  geom_point(aes(x = numMorphGroups[order(deltaBIC_naive)[1]],
                 y = deltaBIC_naive[order(deltaBIC_naive)[1]],
                 shape = "B_best_naive_model"),
             cex = 3.5,
             col = 'black',
             bg = 'black') +
  # geom_point(aes(x=numMorphGroups[order(deltaBIC_naive)[2]],
  #                y=deltaBIC_naive[order(deltaBIC_naive)[2]],
  #                shape ="C_second_naive_model"), cex=3.5, col='black',bg='grey') +
  geom_point(aes(x = numMorphGroups[taxonBIC != 'NA'],
                 y = max.BIC-taxonBIC[taxonBIC != 'NA'],
                 shape = "D_currentTax"),
             cex = 2.5,
             col = 'black',
             bg = 'red') +
  geom_point(aes(x = numMorphGroups[morphoGroupsBIC != 'NA'],
                 y = max.BIC-morphoGroupsBIC[morphoGroupsBIC != 'NA'],
                 shape = "E_morphoGroups"),
             cex = 2.5,
             col = 'black',
             bg = 'red') +
  scale_shape_manual(values = c(21,21,25,24),
                     labels =c("Best fit model",
                              "Naive models",
                              "Taxonomy model",
                              "Taxonomy unaware model"),
                     name = NULL) +
  guides(shape = guide_legend(override.aes = list(shape = c(21,21,25,24),
                                                  bg = c('black','white','red','red'),
                                                  color = c('black','black','black','black'),
                                                  size = 3))) +
  scale_x_discrete(limits = c(1,2,3)) +
  theme_bw() +
  theme(panel.grid = element_blank()) +
  xlab('Number of phenogroups') +
  ylab(expression(paste("Empirical support (", Delta, "BIC)", sep="")))



# plot ordination of best naive model
myShapes = c(21,22,23)
Q2 = ggplot(CladeII_morphDR) +
  geom_jitter(aes(x = MclustDR_Dir1,
                  y = MclustDR_Dir2,
                  shape = as.factor(BestModel_Classification)),
              color = 'black',
              bg = '#cccccc',
              size = 2,
              stroke = 0.25,
              width = 0,
              height = 0.03) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group1', 'group2', 'group3')) +
  xlab("Dimension 1") +
  ylab("Dimension 2") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10),
        axis.ticks.y = element_blank()) +
  guides(shape = guide_legend(override.aes = list(size=2)))

Q3 = ggplot(CladeII_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = Latitude,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group1', 'group2','group3')) +
  xlab("Dimension 1") +
  ylab("Latitude") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

Q4 = ggplot(CladeII_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = Elevation,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group1', 'group2','group3')) +
  xlab("Dimension 1") +
  ylab("Elevation") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

(modelsRanked + plot_spacer() + plot_layout(widths = c(3,1))) /
  (Q2 + Q3 + Q4 + plot_layout(guides = 'collect'))

```
Return to [Clade II Phenomics: model-based species discovery](#model-based-species-discovery-2)
\pagebreak

```{r, CladeIIgenomicSensitivityAnalyses_largest, echo=FALSE, warning = FALSE, message=FALSE, include=FALSE}

# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeII_resultsFiles/CladeII_largestMiss25_concat.contree")
load("supplementary_material_files/CladeII_resultsFiles/CladeII_largestMiss25_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeII_resultsFiles/CladeII_largestMiss25_ADMIX.3.Q")
testData = read_delim("supplementary_material_files/CladeII_resultsFiles/CladeII_largestMiss25_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeII_resultsFiles/CladeII_popFile_withLocality_ADMIXordered.csv", header=TRUE)
sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                  'toss1',
                  'taxon',
                  'libraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep = "_",
        remove = FALSE) %>%
  select(MolecularSampleName,
         SampleName,
         SpecimenID,
         libraryNumber,
         taxon,
         country,
         Elevation,
         Latitude,
         Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:12)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeIIgenomicSensitivityAnalyses_largest2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}
#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(brewer.pal(3, 'Greys'))  
plot1 = ggtree(myTree_midRoot)
plot2 = plot1 +
  geom_tiplab(geom = 'text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)
#plot2
plot3 = facet_plot(plot2,
                   data = admixMelt,
                   panel = 'Admixture proportions',
                   geom = geom_barh,
                   mapping = aes(x = as.numeric(value),
                                 y = as.character(label),
                                 fill = variable),
                   color = 'grey90',
                   stat = 'identity') +
  xlim_tree(0.02) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")
#plot3
plot4 = facet_widths(plot3, c(6,3))
#plot4

#########################
#####      PCA      #####
#########################
pch_v = c(21,22)
fastPCA_largest = ggplot(tab,
                         aes(x = EV1,
                             y = EV2,
                             shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.herrerae",
                                "E.pendula"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size = unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_largest = as_tibble(testData %>%
                            dplyr::group_by(clusterNum) %>%
                            summarise_at("error", list(mean, sd)))

colnames(step1_largest) = c("clusterNum", "mean", "stdev")

step1_largest$me = sapply(step1_largest$stdev,
                          function(x) qt(.95,9)*x/sqrt(10))

testCI_largest = mutate(as_tibble(step1_largest),
                        lower = mean-me,
                        upper = mean+me)

ADMIX_CI_largest = ggplot(testCI_largest,
                          aes(x = clusterNum,
                              y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name="Number of Groups (K)",
                   limits = c(1:3)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

#plot4 + (ADMIX_CI_largest / fastPCA_largest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation( title = "Largest data matrix: ~75% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))


```

```{r, CladeIIgenomicSensitivityAnalyses_middle, echo=FALSE, warning = FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeII_resultsFiles/CladeII_middleMiss50_concat.contree")
load("supplementary_material_files/CladeII_resultsFiles/CladeII_middleMiss50_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeII_resultsFiles/CladeII_middleMiss50_ADMIX.3.Q")
testData = read_delim("supplementary_material_files/CladeII_resultsFiles/CladeII_middleMiss50_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeII_resultsFiles/CladeII_popFile_withLocality_ADMIXordered.csv", header=TRUE)
sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep = "_",
        remove = FALSE) %>%
  select(MolecularSampleName,
         SampleName,
         SpecimenID,
         libraryNumber,
         taxon,
         country,
         Elevation,
         Latitude,
         Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)
myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:12)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeIIgenomicSensitivityAnalyses_middle2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}
#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(brewer.pal(3, 'Greys'))  

plot5 = ggtree(myTree_midRoot)

plot6 = plot5 +
  geom_tiplab(geom = 'text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot7 = facet_plot(plot6,
                   data = admixMelt,
                   panel = 'Admixture proportions',
                   geom = geom_barh,
                   mapping = aes(x = as.numeric(value),
                                 y = as.character(label),
                                 fill = variable),
                   color = 'grey90',
                   stat = 'identity') +
  xlim_tree(0.02) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")

plot8 = facet_widths(plot7, c(6,3))

#########################
#####      PCA      #####
#########################
pch_v = c(21,22)
fastPCA_middle = ggplot(tab,
                        aes(x = EV1,
                            y = EV2,
                            shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.herrerae","E.pendula"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size = unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_middle = as_tibble(testData %>%
                           dplyr::group_by(clusterNum) %>%
                           summarise_at("error",
                                        list(mean, sd)))

colnames(step1_middle) = c("clusterNum", "mean", "stdev")

step1_middle$me = sapply(step1_middle$stdev,
                         function(x) qt(.95,9)*x/sqrt(10))

testCI_middle = mutate(as_tibble(step1_middle),
                       lower = mean-me,
                       upper = mean+me)

ADMIX_CI_middle = ggplot(testCI_middle,
                         aes(x = clusterNum,
                             y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:3)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

#plot8 + (ADMIX_CI_middle / fastPCA_middle) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(title = "Middle-sized data matrix: ~50% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))


```

```{r, CladeIIgenomicSensitivityAnalyses_smallest, echo=FALSE, warning = FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeII_resultsFiles/CladeII_smallestMiss75_concat.contree")
load("supplementary_material_files/CladeII_resultsFiles/CladeII_smallestMiss75_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeII_resultsFiles/CladeII_smallestMiss75_ADMIX.3.Q")
testData = read_delim("supplementary_material_files/CladeII_resultsFiles/CladeII_smallestMiss75_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeII_resultsFiles/CladeII_popFile_withLocality_ADMIXordered.csv", header=TRUE)
sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep = "_",
        remove = FALSE) %>%
  select(MolecularSampleName,
         SampleName,
         SpecimenID,
         libraryNumber,
         taxon,
         country,
         Elevation,
         Latitude,
         Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex=1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:12)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeIIgenomicSensitivityAnalyses_smallest2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}

#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(brewer.pal(3, 'Greys'))  

plot9 = ggtree(myTree_midRoot)

plot10 = plot9 +
  geom_tiplab(geom = 'text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot11 = facet_plot(plot10,
                    data = admixMelt,
                    panel = 'Admixture proportions',
                    geom = geom_barh,
                    mapping = aes(x = as.numeric(value),
                                 y = as.character(label),
                                 fill = variable),
                    color = 'grey90',
                    stat = 'identity') +
  xlim_tree(0.02) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")

plot12 = facet_widths(plot11, c(6,3))

#########################
#####      PCA      #####
#########################
pch_v = c(21,22)
fastPCA_smallest = ggplot(tab,
                          aes(x = EV1,
                              y = EV2,
                              shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.herrerae","E.pendula"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size = unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_smallest = as_tibble(testData %>%
                             dplyr::group_by(clusterNum) %>%
                             summarise_at("error",
                                          list(mean, sd)))

colnames(step1_smallest) = c("clusterNum", "mean", "stdev")

step1_smallest$me = sapply(step1_smallest$stdev,
                           function(x) qt(.95,9)*x/sqrt(10))

testCI_smallest = mutate(as_tibble(step1_smallest),
                         lower = mean-me,
                         upper = mean+me)

ADMIX_CI_smallest = ggplot(testCI_smallest,
                           aes(x = clusterNum,
                               y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:3)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))


#plot12 + (ADMIX_CI_smallest / fastPCA_smallest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(title = "Smallest data matrix: ~25% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))

```

### Fig S18: Sensitivity tests with $75\%$ missing data
```{r, cladeIIgenomicSensitivityAnalysesLargestplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($75\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot4 + (ADMIX_CI_largest / fastPCA_largest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade II Genomics: sensitivity tests](#sensitivity-tests-1)
\pagebreak

### Fig S19: Sensitivity tests with $50\%$ missing data
```{r, cladeIIgenomicSensitivityAnalysesMiddleplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($50\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot8 + (ADMIX_CI_middle / fastPCA_middle) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade II Genomics: sensitivity tests](#sensitivity-tests-1)
\pagebreak

### Fig S20: Sensitivity tests with $25\%$ missing data
```{r, cladeIIgenomicSensitivityAnalysesSmallestplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($25\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot12 + (ADMIX_CI_smallest / fastPCA_smallest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade II Genomics: sensitivity tests](#sensitivity-tests-1)
\pagebreak

### Fig S21: Genogroup delimitation: Genotypic cluster model
```{r cladeIImolecularDelimitationsNmms, echo=FALSE, message=FALSE, fig.width=7, fig.height=2.5, dpi=300, fig.cap='Gaussian finite mixture modeling (GFMM) for genogroup delimitation. Visualization of the genogroups (shades) identified by the best fit Gaussian mixture model (GMM). A) genogroups in the space defined by two axes obtained by non-metric multidimensional scaling (NMDS); B) genogroups in the space defined by NMDS axis 1 and latitude; C) genogroups in the space defined by NMDS axis 1 and elevation.'}

# read in results of NMM delimitation
CladeII_nmms_temp = read.csv("supplementary_material_files/CladeII_resultsFiles/CladeII_smallestMiss75_NMMs.csv")
# select classification and dimensions 1 and 2 from best model
CladeII_nmms = CladeII_nmms_temp %>% select(SpecimenID, Latitude, Longitude, Elevation, class.dim2.n0, NMDS1.dim2.n0, NMDS2.dim2.n0)
colnames(CladeII_nmms)[5:7] = c('Classification','Dim1','Dim2')
# melt in prep for plotting
CladeII_nmms_melted = reshape::melt(CladeII_nmms, by='SpecimenID')

myColors = c('#f0f0f0','#bdbdbd','#636363')
# plot groups in first two dimensions
p1 = ggplot(CladeII_nmms,
            aes(x = Dim1,
                y = Dim2)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              color = 'black',
              size = 3) +
  scale_fill_manual(values = myColors, name='Classification',
                    labels = c('group 1','group 2', 'group 3')) +
  xlab('Dimension 1') +
  ylab('Dimension 2') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))
# plot groups in dimension 1 against latitude
p2 = ggplot(CladeII_nmms,
            aes(x = Dim1,
                y = Latitude)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              size = 3,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1','group 2', 'group 3')) +
  xlab('Dimension 1') +
  ylab('Latitude') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))
# plot groups in dimension1 against elevation
p3 = ggplot(CladeII_nmms,
             aes(x = Dim1,
                 y = Elevation)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              size = 3,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1','group 2', 'group 3')) +
  xlab('Dimension 1') +
  ylab('Elevation') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))

p1 + p2 + p3 + plot_layout(guides = 'collect') + plot_annotation(tag_levels = 'A') & theme(plot.tag = element_text(size = 8))

```
Return to [Clade II Genomics: model-based species discovery](#model-based-species-discovery-3)
\pagebreak

```{r cladeIImolecularDelimitationsPhylogeny, echo=FALSE, include=FALSE}
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(26), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(24), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(22), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(17), state='2', stem=FALSE)
```

### Fig S22: Genogroup delimitation. Cladogenesis to anagenesis model
```{r cladeIImolecularDelimitationsPhylogeny2, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, out.height='.6\\textwidth', dpi=300, fig.show='asis', fig.cap='Phylogenetic modeling for genogroup delimitation. Midpoint-rooted phylogenetic tree showing genogroups in red. Values correspond to nodes at the transition point between cladogenesis (between species) to anagenesis (within species). Values closer to 0 indicate that the node was identified as a transition to anagenesis summarized over $500$ delimitations.'}
# at this point, the 'myTree_midRoot' is now of class 'simmap', so the plotting is a bit different
# because we want our trees without tip labels, i've set the fsize to be very small (essentially label-less)
# after plotting the tree with the 'painted' clades, add the custom node labels

colors = setNames(c('black','red'),c('1','2'))
plotSimmap(myTree_midRoot, setEnv = TRUE, fsize=0.001, lwd=1, colors=colors)
nodelabels(text='0.032', node=26, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.013', node=24, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.008', node=22, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.001', node=17, frame = 'none', adj=c(1.5,-0.50), cex=1)
# internal nodes
nodelabels(text='0.54', node=23, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.98', node=21, frame = 'none', adj=c(1.5,-0.50), cex=1)

```
Return to [Clade II Genomics: model-based species discovery](#model-based-species-discovery-3)
\pagebreak

### Fig S23: Genogroup delimitation: Reproductive isolation model
```{r cladeIImolecularDelimitationPopGenBpp, echo=FALSE, fig.width=6.5, dpi=300, fig.cap='Population genetic modeling for genogroup delimitation. Top panel: assignment of specimens to demes according to \\texttt{STRUCTURE} and posterior probability of species delimitation modeling according to \\texttt{BPP} using these demes. Bottom panel: assignment of specimens to demes according to \\texttt{MAVERICK} and posterior probability of species delimitation modeling according to \\texttt{BPP} using these demes. Specimens are sorted from north (left) to south (right) according to locality of collection.'}

mySTR = read.csv("supplementary_material_files/CladeII_resultsFiles/CladeII_smallestMiss75_STRUCTURE.csv", header=TRUE)
myRMav = read.csv("supplementary_material_files/CladeII_resultsFiles/CladeII_smallestMiss75_RMav.csv", header=TRUE)


# attach assignments
# IMPORTANT: make sure to confirm number of columns you need to attach (corresponds with number of groups to be plotted)
# number of STRUCTURE groups = 2
# number of RMaverick groups = 2
mergedGroupInfo = cbind(sampleInfo, mySTR[match(sampleInfo$MolecularSampleName, mySTR[,'MolecularSampleName']),2:3])
mergedGroupInfo = cbind(mergedGroupInfo, myRMav[match(sampleInfo$MolecularSampleName, myRMav[,'MolecularSampleName']),2:4])

# determine majority group assignment for each individual and attach these to the group information
# STRUCTURE groups are columns 11:12
# RMaverick groups are columns 13:15
mergedGroupInfo$STRassignment = as.factor(colnames(mergedGroupInfo[,10:11]) [apply(mergedGroupInfo[,10:11],1,which.max)])
mergedGroupInfo$RMavassignment = as.factor(colnames(mergedGroupInfo[,12:14]) [apply(mergedGroupInfo[,12:14],1,which.max)])


myGreys = c("#f7f7f7","#cccccc")
myBlues = c("#deebf7","#9ecae1","#3182bd")

STRplot = reshape::melt(mergedGroupInfo,
                        id.vars = c("SpecimenID","Latitude"),
                        measure.vars = c("group1","group2")) %>%
  ggplot(aes(x = reorder(as.factor(SpecimenID), -Latitude),
             y = value,
             fill = as.factor(variable))) +
  geom_bar(stat = 'identity',
           show.legend = FALSE,
           color = 'black') +
  scale_fill_manual(values=myGreys[c(1,2)]) +
  ylab("Probability") +
  theme_bw() +
  #theme(axis.text.x = element_text(size=8, angle=90, vjust = 0.5, hjust=1)) +
  theme(axis.text.x = element_blank(),
        axis.title.x = element_blank()) +
  theme(text = element_text(size = 10))

RMavplot = reshape::melt(mergedGroupInfo,
                         id.vars = c("SpecimenID","Latitude"),
                         measure.vars = c("deme1","deme2","deme3")) %>%
  ggplot(aes(x = reorder(as.factor(SpecimenID), -Latitude),
             y = value,
             fill=as.factor(variable))) +
  geom_bar(stat = 'identity',
           show.legend = FALSE,
           color = 'black') +
  scale_fill_manual(values = myBlues[c(1,2,3)]) +
  ylab("Probability") +
  xlab("Specimens sorted by Latitudinal position") +
  theme_bw() +
  theme(axis.text.x = element_text(size = 4,
                                   angle = 90,
                                   vjust = 0.5,
                                   hjust = 1)) +
  theme(text = element_text(size = 10))
  #theme(axis.text.x = element_blank(), axis.title.x = element_blank())

(STRplot + grid::textGrob("BPP post. probability\n for 2 species = 1.0") + plot_layout(widths = c(3,1))) / (RMavplot + grid::textGrob("BPP post. probability\n for 3 species = 1.0") + plot_layout(widths = c(3,1)))
```
Return to [Clade II Genomics: model-based species discovery](#model-based-species-discovery-3)
\pagebreak

```{r cladeIIfinalDelimitation, echo=FALSE, include=FALSE}
# need to use the output of the morphometrics analysis, since it only includes samples that have BOTH flowers and leaves
morphDelim = read.csv("supplementary_material_files/CladeII_resultsFiles/CladeII_morphDR.csv")

molecularDelim = read.csv("supplementary_material_files/CladeII_resultsFiles/CladeII_molecularDelimitation.csv")
# if it doesn't have a SpecimenID column, you'll need to make one
molecularDelim = molecularDelim %>%
  separate(col = SampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'LibraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep = "_",
        remove = FALSE) %>%
  unite(col ='MolecularSampleName',
        c(SpecimenID,
          toss1,
          taxon,
          LibraryNumber),
        sep = "_",
        remove = FALSE)

# join datasets, coalesce duplicate columns, select needed data
typesOfSpp_temp = full_join(morphDelim, molecularDelim, by = "SpecimenID") %>%
  mutate(
    Latitude = coalesce(Latitude.x, Latitude.y),
    Longitude = coalesce(Longitude.x, Longitude.y),
    Elevation = coalesce(Elevation.x, Elevation.y)
  ) %>%
  select(MolecularSampleName,
         SpecimenID,
         Elevation,
         Latitude,
         Longitude,
         BestModel_Classification, #name of the morphoDelim column
         mPTPgroups) # name of the molecularDelim column
# confirmation
#colnames(typesOfSpp_temp)
#view(typesOfSpp_temp) # confirm number of rows matches that of sampling table


# create plotting df
typesOfSpp = typesOfSpp_temp
colnames(typesOfSpp) = c("MolecularSampleName",
                         "SpecimenID",
                         "Elevation",
                         "Latitude",
                         "Longitude",
                         "PhenoGroups",
                         "GenoGroups")


# plot tree and match dataframe to tip names - ensures order of plotting is correct
myTree = read.tree("supplementary_material_files/CladeII_resultsFiles/CladeII_smallestMiss75_concat.contree")
myTree_midRoot = midpoint.root(myTree)
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()
typesOfSpp = typesOfSpp[order(match(typesOfSpp$MolecularSampleName, myTipNames)),]

```

### Fig S24: Data integration
```{r cladeIIfinalDelimitation2, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, fig.height=5, dpi=300, fig.cap="Integration of phenotypic and genomic data with spatial information and evolutionary history. All specimens are assigned to their corresponding best fit phenogroup (shapes) and genogroup (colors). Specimens without phenotypic or genomic data (unknown specimens) are shown as asterisks and empty shapes, accordingly. Specimens are shown as tips of the maximum likelihood tree (left) used in the CA model analysis and mapped along latitude and elevation (right). Specimens assigned to a single phenogroup and a single genogroup delineate species that we determined as 'good species'. Specimens assigned to a single phenogroup across multiple genogroups delineate species that we determined as 'phenotypic cryptic species'. Specimens assigned to a single genogroup across multiple phenogroups delineate species that we determined as 'genetic cryptic species'."}
########################
# Phylogenetic context # ----------------------------------------------------------------------------------------
########################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups

# get number of colors and shapes that you'll need for plotting
#unique(typesOfSpp$PhenoGroups)
#unique(typesOfSpp$GenoGroups)

# set values for colors and shapes
myShapes = c(21,22,23,8) #the 8 is for unknown
names(myShapes) = c('1','2','3','NA')
myCols = c(brewer.pal(n=4,'Dark2'),"#ffffff")
names(myCols) = c('species1','species2','species3','species4','NA')


f = ggtree(myTree_midRoot) %<+% typesOfSpp + xlim_tree(0.025)
#f
f2 = f + geom_tippoint(aes(x = x,
                           fill = as.factor(GenoGroups),
                           shape = as.factor(PhenoGroups)),
                       size = 3) +
  scale_shape_manual(values = myShapes) +
  scale_fill_manual(values = myCols) +
  geom_tiplab(aes(label = SpecimenID),
              hjust = -0.3,
              size = 2) +
  geom_point2(aes(subset = is.na(PhenoGroups) & isTip,
                  color = as.factor(GenoGroups)),
              pch = 8,
              size = 2,
              stroke = 0.5,
              alpha = 0.5) +
  scale_color_manual(values = myCols) +
  theme(legend.position = "none")
#f2

# subset by major clades; clade numbers come from mPTP figure above
f3 = viewClade(f2, 21)
f4 = viewClade(f2, 17)


###################
# Spatial context # ----------------------------------------------------------------------------------------
###################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups


##### plot individuals with GenoGroup assignment, but no PhenoGroup assignment
g = ggplot(typesOfSpp,
           aes(x = Latitude,
               y = Elevation)) +
  geom_jitter(data = typesOfSpp %>%
                filter(is.na(PhenoGroups)),
              aes(x = Latitude,
                  y = Elevation,
                  color = as.factor(GenoGroups)),
              size = 2,
              alpha = 0.5,
              width = 1.5,
              height = 1.5,
              pch = 8,
              stroke = 1) +
  scale_color_manual(values = myCols) +
  scale_x_reverse() +
  theme_bw() +
  theme(legend.position = 'none') +
  theme(panel.grid.major = element_line(size = 0.55, color = '#f0f0f0'))
#g

##### plot individuals with PhenoGroup assignment, but no GenoGroup assignment
g2 = g + geom_jitter(data = typesOfSpp %>%
                       filter(is.na(GenoGroups)),
                     aes(x = Latitude,
                         y = Elevation,
                         shape = as.factor(PhenoGroups)),
                     fill = 'white',
                     size = 2,
                     width = 1.5,
                     height = 1.5) +
  scale_shape_manual(values = myShapes)
#g2

###### plots fully overlapping data (has both Geno- and Pheno- group assignments)
g3 = g2 + geom_jitter(data = typesOfSpp %>%
                        filter(GenoGroups != 'NA' & PhenoGroups != 'NA'),
                      aes(x = Latitude,
                          y = Elevation,
                          fill = as.factor(GenoGroups),
                          shape = as.factor(PhenoGroups)),
                      size = 3,
                      width = 1.5,
                      height =1.5) +
  scale_fill_manual(values = myCols)
#g3

###### create the plot and save to an object
plotForLegend = ggplot(typesOfSpp,
                       aes(x = Latitude,
                           y = Elevation,
                           fill = as.factor(GenoGroups),
                           shape = as.factor(PhenoGroups))) +
  geom_point() +
  scale_fill_manual(values = myCols,
                    name = 'Genogroups',
                    labels=c('1','2','3','4','unknown')) +
  scale_shape_manual(values = myShapes,
                     name='Phenogroups',
                     labels=c('1','2','3','unknown')) +
  guides(fill = guide_legend(override.aes = list(fill = c(myCols[1:4],
                                                        'white'),
                                                 size = 3,
                                                 shape = 22,
                                                 color='black'))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21,22,23,8),
                                                  size = 3))) +
  theme(legend.key = element_rect(fill = NA),
        text = element_text(size = 8))

legend = cowplot::get_legend(plotForLegend)

#g3 + legend


##########
# GK TAU # ----------------------------------------------------------------------------------------
##########
fullOverlap = typesOfSpp %>%
  filter(PhenoGroups != 'NA' & GenoGroups != 'NA')

gktau_morph2molecules = GKtau(fullOverlap$PhenoGroups,
                              fullOverlap$GenoGroups)
gktau_molecules2morph = GKtau(fullOverlap$GenoGroups,
                              fullOverlap$PhenoGroups)


####################################
# PLOT both together # --------------------------------------------
####################################
# one row
f2 + g3 + legend + plot_layout(widths=c(4,3,1))


```
Return to [Clade II Data integration](#data-integration-2)
\pagebreak

## Clade III
### Fig S25: Taxon sampling
```{r CladeIIIVennDiagram, echo=FALSE, dpi=300, fig.width=4, fig.align='center', fig.cap="Specimens sampled according to three types of data. Specimens outside the Flowers category represent sterile specimens."}

CladeIII_Venn = escalloniaSampling %>%
  filter(CladeID == "III") %>%
  mutate(Flowers = if_else(is.na(`Floral and Leaf`), 0, 1))

CladeIII_Venn = CladeIII_Venn %>%
  mutate(Leaves = if_else(is.na(`Leaf only`) & is.na(`Floral and Leaf`), 0,1))

CladeIII_Venn = CladeIII_Venn %>%
  mutate(Genes = if_else(is.na(libraryNumber), 0, 1))

CladeIII_Venn = CladeIII_Venn %>%
  select(Flowers,
         Leaves,
         Genes)

VennForCladeIII = venn(CladeIII_Venn)

plot(VennForCladeIII,
     fills = list(fill =c('#c2a5cf',
                         '#d9f0d3',
                         '#f7f7f7')),
     quantities = list(TRUE,
                       cex = 1.5),
     legend = list(labels,
                   cex = 1))

```
Return to [Clade III Sampling](#sampling-3)
\pagebreak

### Fig S26: Geographic distribution
```{r CladeIIIsamplingMaps, echo=FALSE, dpi = 300, fig.align='center', fig.cap="Geographic distribution of specimens sampled for this clade according to taxonomic species. Filled symbols indicate specimens used in phenotypic analyses and empty symbols specimens used in genomic analyses."}
myColors = "#cccccc"
myShapes = c(21,22,23,24,25,10)
discolorMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'discolor') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon=='discolor') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-100, -55),  
              ylim = c(-30, 20)) +
  theme_bw() +
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label='Escallonia discolor')


paniculataMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'paniculata') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'paniculata') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-100, -55),  
              ylim = c(-30, 20)) +
  theme_bw() +
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face='italic'),
        text = element_text(size=6)) +
  ggtitle(label='Escallonia paniculata')

piurensisMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'piurensis') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex=2,
              alpha=0.8,
              width=0.5,
              height=0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'piurensis') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-100, -55),  
              ylim = c(-30, 20)) +
  theme_bw() +
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face='italic'),
        text = element_text(size=6)) +
  ggtitle(label='Escallonia piurensis')

resinosaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'resinosa') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'resinosa') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-100, -55),
              ylim = c(-30, 20)) +
  theme_bw() +
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label='Escallonia resinosa')

reticulataMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'reticulata') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'reticulata') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-100, -55),
              ylim = c(-30, 20)) +
  theme_bw() +
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label='Escallonia reticulata')

schreiteriMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'schreiteri') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'schreiteri') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-100, -55),
              ylim = c(-30, 20)) +
  theme_bw() +
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label='Escallonia schreiteri')

discolorMap + paniculataMap + piurensisMap +
  resinosaMap + reticulataMap + schreiteriMap + plot_layout(ncol = 3, guides='collect')

```
Return to [Clade III Sampling](#sampling-3)
\pagebreak

```{r currentStateOfTaxonomyCladeIIIMakePlots, echo=FALSE, warning=FALSE}

pal = c('#636363')
names(pal) = c('assignment')

# plot
CladeIII_propOverlap_All = ggplot(melted_CladeIII_overlapProp_All,
                                aes(x = X1,
                                    y = X2)) +
  geom_tile(aes(fill = value)) +
  geom_text(aes(label = round(value,
                              digits = 3)),
            color = 'darkgrey',
            size = 1) +
  scale_fill_gradient(low = 'white', 
                      high = "black") +
  theme(axis.title = element_blank(),
        axis.text = element_text(size = 6),
        axis.text.x = element_text(angle = -45,
                                   hjust = 0.1)) +
  theme(legend.text = element_text(size = 4),
        legend.margin = margin(2,2,6,2),
        legend.title = element_text(size = 6),
        legend.background = element_rect(color="#cccccc")) +
  guides(fill = guide_colourbar(barwidth = 0.5,
                                barheight = 2))


CladeIII_inHullAssignments_All = ggplot(melted_CladeIII_hullResults_All,
                                      aes(x = uniqueID,
                                          y = variable,
                                          fill = col)) +
  geom_tile(aes(width = 0.99,
                height = 0.99),
            color = 'black') +
  scale_fill_manual(values = c("assignment" = '#636363',
                               "no assignment" = "white"),
                    name = NULL,
                    labels = c('yes','no'),
                    drop = FALSE) +
  scale_y_discrete(labels=c("In E. discolor",
                            "In E. paniculata",
                            "In E. piurensis",
                            "In E. resinosa",
                            "In E. reticulata",
                            "In E. schreiteri")) +
  scale_x_discrete(limits = factor(1:130)) + # limits equal to number of samples
  theme_bw() +
  theme(panel.ontop = TRUE,
        panel.background = element_rect(fill='transparent'),
        panel.border = element_blank(),
        panel.grid.major.y = element_blank(),
        panel.grid.major.x = element_blank(),
        axis.title.y = element_blank(),
        axis.title.x = element_blank(),
        axis.ticks.x = element_blank(),
        axis.text.x = element_blank(),
        legend.position = 'top',
        text = element_text(size = 6),
        plot.margin = margin(1,20,30,10),
        legend.key.height = unit(0.25,'cm'),
        legend.key.width = unit(0.25,'cm')) +
  coord_cartesian(ylim = c(1.1,6), clip = 'off') +
  annotate(geom = 'text',
           x = (5/2),
           y = 0.1,
           label = 'As E. discolor',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom='text',
           x = (6 + (66-6)/2),
           y = 0.1,
           label = 'As E. paniculata',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (67 + (73-67)/2),
           y = 0.1,
           label = 'As E. piurensis',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom= 'text',
           x = (74 + (106-74)/2),
           y = 0.1,
           label = 'As E. resinosa',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (107 + (117-107)/2),
           y = 0.1,
           label = 'As E. reticulata',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (118 + (130-118)/2),
           y = 0.1,
           label= 'As E. schreiteri',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'segment',
           x = 1,
           xend = 5,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 6,
           xend = 66,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 67,
           xend = 73,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 74,
           xend = 106,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 107,
           xend = 117,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 118,
           xend = 130,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75)

```

### Fig S27: Current state of taxonomic species with phenotypic data
```{r currentStateOfTaxonomyCladeIIIPlots, fig.width=7, fig.height = 5, dpi=200, echo=FALSE, fig.cap="Assessment of current state of taxonomic species with phenotypic data. Top panel: Pairwise overlap among 10-cubes describing geometrically each taxonomic species. Bottom panel: Matching-prediction analysis with each cell along the x-axis representing specimens sorted according to taxonomic species and the 10-cubes corresponding to each taxonomic species along the y-axis. If a specimen matches the prediction of the monograph (i.e., it is inside a 10-cube), the corresponding cell is shaded. If the specimen does not match the prediction, the cell is empty."}

(((CladeIII_propOverlap_All + plot_spacer())/ CladeIII_inHullAssignments_All) + plot_layout(tag_level='new'))


```
Return to [Clade III Current state of taxonomic species](#current-state-of-taxonomic-species-2)
\pagebreak

### Fig S28: Phenogroup delimitation: Gaussian finite mixture modeling
```{r cladeIIImorphologicalDelimitationPlots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=6.5, dpi=300, fig.cap='Gaussian finite mixture modeling (GFMM) for phenogroup delimitation and model selection using the Bayesian information criterion (BIC). Top panel: empirical support (ordinate) for Gaussian mixture models (GMM) assuming distinct number of phenogroups (abscissa). Each GMM specifies different number of phenogroups (shapes). Empirical support was measured as difference in BIC relative to the best model ($\\Delta$BIC = $0$). Bottom panel: Visualization of the phenogroups (shapes) identified by the best fit GMM; left panel shows phenogroups in the space defined by two axes obtained by linear discriminat analysis (to maximize separation and visualization), middle panel shows phenogroups in the space defined by discriminant axis 1 and latitude, and right panel shows phenogroups in the space defined by discriminant axis 1 and elevation.'}

CladeIII_modelsRanked = read.csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_modelsRanked.csv")
CladeIII_morphDR = read.csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_morphDR.csv")
max.BIC = 387.1528

#plot
modelsRanked = ggplot(CladeIII_modelsRanked,
                      aes(x = numMorphGroups,
                          y = deltaBIC_naive)) +
  geom_point(aes(x = numMorphGroups,
                 y = deltaBIC_naive,
                 shape = "A_naive_models"),
             cex = 3,
             col = "black",
             bg = 'white') +
  geom_point(aes(x = numMorphGroups[order(deltaBIC_naive)[1]],
                 y = deltaBIC_naive[order(deltaBIC_naive)[1]],
                 shape = "B_best_naive_model"),
             cex = 3,
             col = 'black',
             bg = 'black') +
  # geom_point(aes(x=numMorphGroups[order(deltaBIC_naive)[2]],
  #                y=deltaBIC_naive[order(deltaBIC_naive)[2]],
  #                shape ="C_second_naive_model"), cex=3, col='black',bg='grey') +
  geom_point(aes(x = numMorphGroups[taxonBIC != 'NA'],
                 y = max.BIC-taxonBIC[taxonBIC != 'NA'],
                 shape="D_currentTax"),
             cex = 3,
             col = 'black',
             bg ='red') +
  geom_point(aes(x = numMorphGroups[morphoGroupsBIC != 'NA'],
                 y = max.BIC-morphoGroupsBIC[morphoGroupsBIC != 'NA'],
                 shape = "E_morphoGroups"),
             cex = 3,
             col = 'black',
             bg = 'red') +
  scale_shape_manual(values = c(21,21,25,24),
                     labels = c("Best fit model",
                              "Naive models",
                              "Taxonomy model",
                              "Taxonomy unaware model"),
                     name = NULL) +
  guides(shape = guide_legend(override.aes = list(shape = c(21,21,25,24),
                                                  bg = c('black','white','red','red'),
                                                  color = c('black','black','black','black'),
                                                  size = 3))) +
  scale_x_discrete(limits=c(1,2,3,4,5,6,7,8,9)) +
  theme_bw() +
  theme(panel.grid = element_blank()) +
  xlab('Number of phenogroups') +
  ylab(expression(paste("Empirical support (", Delta, "BIC)", sep="")))



# plot ordination of best naive model
myShapes = c(21,22,23,24,25)
Q2 = ggplot(CladeIII_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = MclustDR_Dir2,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                    name = 'Classification',
                    labels = c('group1',
                               'group2',
                               'group3',
                               'group4',
                               'group5')) +
  xlab("Dimension 1") +
  ylab("Dimension 2") +
  theme(legend.key = element_rect(fill=NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size=2)))

Q3 = ggplot(CladeIII_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = Latitude,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group1',
                              'group2',
                              'group3',
                              'group4',
                              'group5')) +
  xlab("Dimension 1") +
  ylab("Latitude") +
  theme(legend.key = element_rect(fill=NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

Q4 = ggplot(CladeIII_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = Elevation,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values=myShapes,
                     name='Classification',
                     labels=c('group1',
                              'group2',
                              'group3',
                              'group4',
                              'group5')) +
  xlab("Dimension 1") +
  ylab("Elevation") +
  theme(legend.key = element_rect(fill=NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

(modelsRanked + plot_spacer() + plot_layout(widths = c(3,1))) /
  (Q2 + Q3 + Q4 + plot_layout(guides = 'collect'))

```
Return to [Clade III Phenomics: model-based species discovery](#model-based-species-discovery-4)
\pagebreak

```{r, genomicSensitivityAnalyses_largest, echo=FALSE, warning = FALSE, message=FALSE, include=FALSE}

# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeIII_resultsFiles/CladeIII_largestMiss25_concat.contree")
load("supplementary_material_files/CladeIII_resultsFiles/CladeIII_largestMiss25_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeIII_resultsFiles/CladeIII_largestMiss25_ADMIX.8.Q")
testData = read_delim("supplementary_material_files/CladeIII_resultsFiles/CladeIII_largestMiss25_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_popFile_withLocality_ADMIXordered.csv", header=TRUE)
sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                  'toss1',
                  'taxon',
                  'libraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,toss1,taxon),
        sep = "_",
        remove = FALSE) %>%
  select(MolecularSampleName,
         SampleName,
         SpecimenID,
         libraryNumber,
         taxon,
         country,
         Elevation,
         Latitude,
         Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3","ADMIXgroup4","ADMIXgroup5","ADMIXgroup6","ADMIXgroup7","ADMIXgroup8")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:17)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, genomicSensitivityAnalyses_largest2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}
#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(brewer.pal(8, 'Greys'))
plot1 = ggtree(myTree_midRoot)
plot2 = plot1 +
  geom_tiplab(geom = 'text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)
#plot2
plot3 = facet_plot(plot2,
                   data = admixMelt,
                   panel = 'Admixture proportions',
                   geom = geom_barh,
                   mapping = aes(x = as.numeric(value),
                                 y = as.character(label),
                                 fill = variable),
                   color = 'grey90',
                   stat = 'identity') +
  xlim_tree(0.02) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")
#plot3
plot4 = facet_widths(plot3, c(6,3))
#plot4

#########################
#####      PCA      #####
#########################
pch_v = c(21,22,23,24,25,10)
fastPCA_largest = ggplot(tab,
                         aes(x = EV1,
                             y = EV2,
                             shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.discolor",
                                "E.paniculata",
                                "E.piurensis",
                                "E.resinosa",
                                "E.reticulata",
                                "E.schreiteri"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size = unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_largest = as_tibble(testData %>%
                            dplyr::group_by(clusterNum) %>%
                            summarise_at("error", list(mean, sd)))

colnames(step1_largest) = c("clusterNum",
                            "mean",
                            "stdev")

step1_largest$me = sapply(step1_largest$stdev,
                          function(x) qt(.95,9)*x/sqrt(10))

testCI_largest = mutate(as_tibble(step1_largest),
                        lower = mean-me,
                        upper = mean+me)

ADMIX_CI_largest = ggplot(testCI_largest,
                          aes(x = clusterNum,
                              y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:9)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

#plot4 + (ADMIX_CI_largest / fastPCA_largest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation( title = "Largest data matrix: ~75% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))


```

```{r, genomicSensitivityAnalyses_middle, echo=FALSE, warning = FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeIII_resultsFiles/CladeIII_middleMiss50_concat.contree")
load("supplementary_material_files/CladeIII_resultsFiles/CladeIII_middleMiss50_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeIII_resultsFiles/CladeIII_middleMiss50_ADMIX.8.Q")
testData = read_delim("supplementary_material_files/CladeIII_resultsFiles/CladeIII_middleMiss50_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_popFile_withLocality_ADMIXordered.csv", header=TRUE)
sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep = "_",
        remove = FALSE) %>%
  select(MolecularSampleName,
         SampleName,
         SpecimenID,
         libraryNumber,
         taxon,
         country,
         Elevation,
         Latitude,
         Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3","ADMIXgroup4","ADMIXgroup5","ADMIXgroup6","ADMIXgroup7","ADMIXgroup8")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:17)],
                            id = c("SpecimenID", "taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, genomicSensitivityAnalyses_middle2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}
#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(brewer.pal(8, 'Greys'))

plot5 = ggtree(myTree_midRoot)

plot6 = plot5 +
  geom_tiplab(geom = 'text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot7 = facet_plot(plot6,
                   data = admixMelt,
                   panel = 'Admixture proportions',
                   geom = geom_barh,
                   mapping = aes(x = as.numeric(value),
                                 y = as.character(label),
                                 fill = variable),
                    color = 'grey90',
                    stat = 'identity') +
  xlim_tree(0.02) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")

plot8 = facet_widths(plot7, c(6,3))

#########################
#####      PCA      #####
#########################
pch_v = c(21,22,23,24,25,10)
fastPCA_middle = ggplot(tab,
                        aes(x = EV1,
                            y = EV2,
                            shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.discolor",
                                "E.paniculata",
                                "E.piurensis",
                                "E.resinosa",
                                "E.reticulata",
                                "E.schreiteri"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size = unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_middle = as_tibble(testData %>%
                           dplyr::group_by(clusterNum) %>%
                           summarise_at("error", list(mean, sd)))

colnames(step1_middle) = c("clusterNum",
                           "mean",
                           "stdev")
step1_middle$me = sapply(step1_middle$stdev,
                         function(x) qt(.95,9)*x/sqrt(10))

testCI_middle = mutate(as_tibble(step1_middle),
                       lower = mean-me,
                       upper = mean+me)

ADMIX_CI_middle = ggplot(testCI_middle,
                         aes(x = clusterNum,
                             y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:9)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

#plot8 + (ADMIX_CI_middle / fastPCA_middle) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(title = "Middle-sized data matrix: ~50% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))


```

```{r, genomicSensitivityAnalyses_smallest, echo=FALSE, warning = FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeIII_resultsFiles/CladeIII_smallestMiss75_concat.contree")
load("supplementary_material_files/CladeIII_resultsFiles/CladeIII_smallestMiss75_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeIII_resultsFiles/CladeIII_smallestMiss75_ADMIX.9.Q")
testData = read_delim("supplementary_material_files/CladeIII_resultsFiles/CladeIII_smallestMiss75_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_popFile_withLocality_ADMIXordered.csv", header=TRUE)

sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep = "_",
        remove = FALSE) %>%
  select(MolecularSampleName,
         SampleName,
         SpecimenID,
         libraryNumber,
         taxon,
         country,
         Elevation,
         Latitude,
         Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3","ADMIXgroup4","ADMIXgroup5","ADMIXgroup6","ADMIXgroup7","ADMIXgroup8","ADMIXgroup9")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:18)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, genomicSensitivityAnalyses_smallest2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}

#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(brewer.pal(9, 'Greys'))

plot9 = ggtree(myTree_midRoot)

plot10 = plot9 +
  geom_tiplab(geom = 'text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot11 = facet_plot(plot10,
                    data = admixMelt,
                    panel = 'Admixture proportions',
                    geom = geom_barh,
                    mapping = aes(x = as.numeric(value),
                                  y = as.character(label),
                                  fill = variable),
                    color = 'grey90',
                    stat = 'identity') +
  xlim_tree(0.02) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")

plot12 = facet_widths(plot11, c(6,3))

#########################
#####      PCA      #####
#########################
pch_v = c(21,22,23,24,25,10)
fastPCA_smallest = ggplot(tab,
                          aes(x = EV1,
                              y = EV2,
                              shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.discolor",
                                "E.paniculata",
                                "E.piurensis",
                                "E.resinosa",
                                "E.reticulata",
                                "E.schreiteri"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size = unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_smallest = as_tibble(testData %>%
                             dplyr::group_by(clusterNum) %>%
                             summarise_at("error", list(mean, sd)))

colnames(step1_smallest) = c("clusterNum", "mean", "stdev")

step1_smallest$me = sapply(step1_smallest$stdev,
                           function(x) qt(.95,9)*x/sqrt(10))

testCI_smallest = mutate(as_tibble(step1_smallest),
                         lower = mean-me,
                         upper = mean+me)

ADMIX_CI_smallest = ggplot(testCI_smallest,
                           aes(x = clusterNum,
                               y = mean)) +
  geom_point(cex=0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:9)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))


#plot12 + (ADMIX_CI_smallest / fastPCA_smallest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(title = "Smallest data matrix: ~25% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))

```

### Fig S29: Sensitivity tests with $75\%$ missing data
```{r, cladeIIIgenomicSensitivityAnalysesLargestplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($75\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot4 + (ADMIX_CI_largest / fastPCA_largest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade III Genomics: sensitivity tests](#sensitivity-tests-2)
\pagebreak

### Fig S30: Sensitivity tests with $50\%$ missing data
```{r, cladeIIIgenomicSensitivityAnalysesMiddleplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($50\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot8 + (ADMIX_CI_middle / fastPCA_middle) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade III Genomics: sensitivity tests](#sensitivity-tests-2)
\pagebreak

### Fig S31: Sensitivity tests with $25\%$ missing data
```{r, cladeIIIgenomicSensitivityAnalysesSmallestplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($25\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot12 + (ADMIX_CI_smallest / fastPCA_smallest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade III Genomics: sensitivity tests](#sensitivity-tests-2)
\pagebreak

### Fig S32: Genogroup delimitation: Genotypic cluster model
```{r cladeIIImolecularDelimitationsNmms, echo=FALSE, message=FALSE, fig.width=7, fig.height=2.5, dpi=300, fig.cap='Gaussian finite mixture modeling (GFMM) for genogroup delimitation. Visualization of the genogroups (shades) identified by the best fit Gaussian mixture model (GMM). A) genogroups in the space defined by two axes obtained by non-metric multidimensional scaling (NMDS); B) genogroups in the space defined by NMDS axis 1 and latitude; C) genogroups in the space defined by NMDS axis 1 and elevation.'}

# read in results of NMM delimitation
CladeIII_nmms_temp = read.csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_smallestMiss75_NMMs.csv")
# select classification and dimensions 1 and 2 from best model
CladeIII_nmms = CladeIII_nmms_temp %>%
  select(SpecimenID,
         Latitude,
         Longitude,
         Elevation,
         class.dim2.n0,
         NMDS1.dim2.n0,
         NMDS2.dim2.n0)

colnames(CladeIII_nmms)[5:7] = c('Classification','Dim1','Dim2')
# melt in prep for plotting
CladeIII_nmms_melted = reshape::melt(CladeIII_nmms, by='SpecimenID')

myColors = c('#f0f0f0','#bdbdbd','#636363')
# plot groups in first two dimensions
p1 = ggplot(CladeIII_nmms,
             aes(x = Dim1,
                 y = Dim2,
                 fill = as.factor(Classification))) +
  geom_jitter(pch = 21,
              height = 0.05,
              width = 0.05,
              size = 3,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3')) +
  xlab('Dimension 1') +
  ylab('Dimension 2') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))
# plot groups in dimension 1 against latitude
p2 = ggplot(CladeIII_nmms,
             aes(x = Dim1,
                 y = Latitude)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              size = 3,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3')) +
  xlab('Dimension 1') +
  ylab('Latitude') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))
# plot groups in dimension1 against elevation
p3 = ggplot(CladeIII_nmms,
             aes(x = Dim1,
                 y = Elevation)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              size = 3,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3')) +
  xlab('Dimension 1') +
  ylab('Elevation') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))

p1 + p2 + p3 + plot_layout(guides = 'collect') + plot_annotation(tag_levels = 'A') & theme(plot.tag = element_text(size = 8))

```
Return to [Clade III Genomics: model-based species discovery](#model-based-species-discovery-5)
\pagebreak

```{r cladeIIImolecularDelimitationsPhylogeny, echo=FALSE, include=FALSE}

myTree_midRoot = paintSubTree(myTree_midRoot, node=c(97), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(94), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(82), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(77), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(74), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(65), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(57), state='2', stem=FALSE)

```

### Fig S33: Genogroup delimitation. Cladogenesis to anagenesis model
```{r cladeIIImolecularDelimitationsPhylogeny2, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, out.height='.6\\textwidth', dpi=300, fig.show='asis', fig.cap='Phylogenetic modeling for genogroup delimitation. Midpoint-rooted phylogenetic tree showing genogroups in red. Values correspond to nodes at the transition point between cladogenesis (between species) to anagenesis (within species). Values closer to 0 indicate that the node was identified as a transition to anagenesis summarized over $500$ delimitations.'}
# at this point, the 'myTree_midRoot' is now of class 'simmap', so the plotting is a bit different
# because we want our trees without tip labels, i've set the fsize to be very small (essentially label-less)
# after plotting the tree with the 'painted' clades, add the custom node labels
colors = setNames(c('black','red'),c('1','2'))
plotSimmap(myTree_midRoot, setEnv = TRUE, fsize=0.001, lwd=1, colors=colors)
nodelabels(text='0.004', node=97, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.001', node=94, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.35', node=82, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.005', node=88, frame = 'none', adj=c(1.1,-0.50), cex=1)
nodelabels(text='0', node=83, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0', node=77, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0', node=74, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.29', node=65, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.004', node=66, frame = 'none', adj=c(1.1,-0.50), cex=1)
nodelabels(text='0.09', node=57, frame = 'none', adj=c(1.5,-0.50), cex=1)

# internal nodes
nodelabels(text='1.0', node=75, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='1.0', node=76, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.99', node=81, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='1.0', node=55, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='1.0', node=56, frame = 'none', adj=c(1.5,-0.50), cex=1)
```
Return to [Clade III Genomics: model-based species discovery](#model-based-species-discovery-5)
\pagebreak

### Fig S34: Genogroup delimitation: Reproductive isolation model
```{r cladeIIImolecularDelimitationPopGenBpp, echo=FALSE, fig.width=6.5, dpi=300, fig.cap='Population genetic modeling for genogroup delimitation. Top panel: assignment of specimens to demes according to \\texttt{STRUCTURE} and posterior probability of species delimitation modeling according to \\texttt{BPP} using these demes. Bottom panel: assignment of specimens to demes according to \\texttt{MAVERICK} and posterior probability of species delimitation modeling according to \\texttt{BPP} using these demes. Specimens are sorted from north (left) to south (right) according to locality of collection.'}

mySTR = read.csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_smallestMiss75_STRUCTURE.csv", header=TRUE)
myRMav = read.csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_smallestMiss75_RMav.csv", header=TRUE)


# attach assignments
# IMPORTANT: make sure to confirm number of columns you need to attach (corresponds with number of groups to be plotted)
# number of STRUCTURE groups = 3
# number of RMaverick groups = 5
mergedGroupInfo = cbind(sampleInfo, mySTR[match(sampleInfo$MolecularSampleName, mySTR[,'MolecularSampleName']),2:4])
mergedGroupInfo = cbind(mergedGroupInfo, myRMav[match(sampleInfo$MolecularSampleName, myRMav[,'MolecularSampleName']),2:6])

# determine majority group assignment for each individual and attach these to the group information
# STRUCTURE groups are columns 11:12
# RMaverick groups are columns 13:15
mergedGroupInfo$STRassignment = as.factor(colnames(mergedGroupInfo[,10:12]) [apply(mergedGroupInfo[,10:12],1,which.max)])
mergedGroupInfo$RMavassignment = as.factor(colnames(mergedGroupInfo[,13:17]) [apply(mergedGroupInfo[,13:17],1,which.max)])


myGreys = c("#f7f7f7","#cccccc","#969696", "#636363", "#252525")
myBlues = c("#eff3ff","#bdd7e7","#6baed6","#3182bd","#08519c")

STRplot = reshape::melt(mergedGroupInfo,
                        id.vars = c("SpecimenID","Latitude"),
                        measure.vars = c("group1",
                                         "group2",
                                         "group3")) %>%
  ggplot(aes(x = reorder(as.factor(SpecimenID), -Latitude),
             y = value,
             fill = as.factor(variable))) +
  geom_bar(stat = 'identity',
           show.legend = FALSE,
           color = 'black') +
  scale_fill_manual(values = myGreys[c(1,2,3)]) +
  ylab("Probability") +
  theme_bw() +
  #theme(axis.text.x = element_text(size=8, angle=90, vjust = 0.5, hjust=1)) +
  theme(axis.text.x = element_blank(),
        axis.title.x = element_blank()) +
  theme(text = element_text(size = 10))

RMavplot = reshape::melt(mergedGroupInfo,
                         id.vars = c("SpecimenID",
                                   "Latitude"),
                         measure.vars = c("deme1",
                                          "deme2",
                                          "deme3",
                                          "deme4",
                                          "deme5")) %>%
  ggplot(aes(x = reorder(as.factor(SpecimenID), -Latitude),
             y = value,
             fill = as.factor(variable))) +
  geom_bar(stat = 'identity',
           show.legend = FALSE,
           color = 'black') +
  scale_fill_manual(values = myBlues[c(1,2,3,4,5)]) +
  ylab("Probability") +
  xlab("Specimens sorted by Latitudinal position") +
  theme_bw() +
  theme(axis.text.x = element_text(size = 4,
                                   angle = 90,
                                   vjust = 0.5,
                                   hjust = 1)) +
  theme(text = element_text(size = 10))
  #theme(axis.text.x = element_blank(), axis.title.x = element_blank())

(STRplot + grid::textGrob("BPP post. probability\n for 3 species = 1.0") + plot_layout(widths = c(3,1))) / (RMavplot + grid::textGrob("BPP post. probability\n for 5 species = 1.0") + plot_layout(widths = c(3,1)))
```
Return to [Clade III Genomics: model-based species discovery](#model-based-species-discovery-5)
\pagebreak

```{r cladeIIIfinalDelimitation, echo=FALSE, include=FALSE}
# need to use the output of the morphometrics analysis, since it only includes samples that have BOTH flowers and leaves
morphDelim = read.csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_morphDR.csv")

molecularDelim = read.csv("supplementary_material_files/CladeIII_resultsFiles/CladeIII_molecularDelimitation.csv")
# if it doesn't have a SpecimenID column, you'll need to make one
molecularDelim = molecularDelim %>%
  separate(col = SampleName,
           into = c('SpecimenID',
                  'toss1',
                  'taxon',
                  'LibraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,toss1,taxon),
        sep="_",
        remove = FALSE) %>%
  unite(col = 'MolecularSampleName',
        c(SpecimenID,
          toss1,
          taxon,
          LibraryNumber),
        sep ="_",
        remove = FALSE)

# join datasets, coalesce duplicate columns, select needed data
typesOfSpp_temp = full_join(morphDelim, molecularDelim, by="SpecimenID") %>%
  mutate(
    Latitude = coalesce(Latitude.x, Latitude.y),
    Longitude = coalesce(Longitude.x, Longitude.y),
    Elevation = coalesce(Elevation.x, Elevation.y)
  ) %>%
  select(MolecularSampleName,
         SpecimenID,
         Elevation,
         Latitude,
         Longitude,
         BestModel_Classification, #name of the morphoDelim column
         mPTPgroups) # name of the molecularDelim column
# confirmation
#colnames(typesOfSpp_temp)
#view(typesOfSpp_temp) # confirm number of rows matches that of sampling table


# create plotting df
typesOfSpp = typesOfSpp_temp
colnames(typesOfSpp) = c("MolecularSampleName",
                         "SpecimenID",
                         "Elevation",
                         "Latitude",
                         "Longitude",
                         "PhenoGroups",
                         "GenoGroups")

# plot tree and match dataframe to tip names - ensures order of plotting is correct
myTree = read.tree("supplementary_material_files/CladeIII_resultsFiles/CladeIII_smallestMiss75_concat.contree")
myTree_midRoot = midpoint.root(myTree)
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()
typesOfSpp = typesOfSpp[order(match(typesOfSpp$MolecularSampleName, myTipNames)),]

```

### Fig S35: Data integration
```{r cladeIIIfinalDelimitation2, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, fig.height=7, dpi=300, fig.cap="Integration of phenotypic and genomic data with spatial information and evolutionary history. All specimens are assigned to their corresponding best fit phenogroup (shapes) and genogroup (colors). Specimens without phenotypic or genomic data (unknown specimens) are shown as asterisks and empty shapes, accordingly. Specimens are shown as tips of the maximum likelihood tree (left) used in the CA model analysis and mapped along latitude and elevation (right). Specimens assigned to a single phenogroup and a single genogroup delineate species that we determined as 'good species'. Specimens assigned to a single phenogroup across multiple genogroups delineate species that we determined as 'phenotypic cryptic species'. Specimens assigned to a single genogroup across multiple phenogroups delineate species that we determined as 'genetic cryptic species'."}
########################
# Phylogenetic context # ----------------------------------------------------------------------------------------
########################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups

# get number of colors and shapes that you'll need for plotting
#unique(typesOfSpp$PhenoGroups)
#unique(typesOfSpp$GenoGroups)

# set values for colors and shapes
myShapes = c(21,22,23,24,25,8) #the 8 is for unknown
names(myShapes) = c('1','2','3','4','5','NA')
myCols = c(brewer.pal(n=7,'Dark2'),"#ffffff")
names(myCols) = c('group1','group2','group3','group4','group5','group6','group7','NA')


f = ggtree(myTree_midRoot) %<+% typesOfSpp + xlim_tree(0.015)
#f
f2 = f + geom_tippoint(aes(x = x,
                           fill = as.factor(GenoGroups),
                           shape = as.factor(PhenoGroups)),
                       size = 3) +
  scale_shape_manual(values = myShapes) +
  scale_fill_manual(values = myCols) +
  geom_tiplab(aes(label = SpecimenID),
              hjust = -0.3,
              size = 2) +
  geom_point2(aes(subset=is.na(PhenoGroups) & isTip,
                  color=as.factor(GenoGroups)),
              pch = 8,
              size = 2,
              stroke = 0.5,
              alpha = 0.5) +
  scale_color_manual(values = myCols) +
  theme(legend.position = "none")
#f2

f3 = viewClade(f2,76)
f4 = viewClade(f2,97)
f5 = viewClade(f2,55)

###################
# Spatial context # ----------------------------------------------------------------------------------------
###################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups


##### plot individuals with GenoGroup assignment, but no PhenoGroup assignment
g = ggplot(typesOfSpp,
           aes(x = Latitude,
               y = Elevation)) +
  geom_jitter(data = typesOfSpp %>%
                filter(is.na(PhenoGroups)),
              aes(x = Latitude,
                  y = Elevation,
                  color = as.factor(GenoGroups)),
              size = 2,
              alpha = 0.5,
              width = 0.75,
              height = 1.5,
              pch = 8,
              stroke = 1) +
  scale_color_manual(values = myCols) +
  scale_x_reverse() +
  theme_bw() +
  theme(legend.position = 'none') +
  theme(panel.grid.major = element_line(size = 0.55,
                                        color = '#f0f0f0'))
#g

##### plot individuals with PhenoGroup assignment, but no GenoGroup assignment
g2 = g + geom_jitter(data = typesOfSpp %>%
                       filter(is.na(GenoGroups)),
                     aes(x = Latitude,
                         y = Elevation,
                         shape = as.factor(PhenoGroups)),
                     fill = 'white',
                     size = 2,
                     width = 0.75,
                     height = 1.5) +
  scale_shape_manual(values = myShapes)
#g2

###### plots fully overlapping data (has both Geno- and Pheno- group assignments)
g3 = g2 + geom_jitter(data = typesOfSpp %>%
                        filter(GenoGroups != 'NA' & PhenoGroups != 'NA'),
                      aes(x = Latitude,
                          y = Elevation,
                          fill = as.factor(GenoGroups),
                          shape = as.factor(PhenoGroups)),
                      size = 3,
                      width = 0.75,
                      height = 1.5) +
  scale_fill_manual(values = myCols)
#g3

###### create the plot and save to an object
plotForLegend = ggplot(typesOfSpp,
                       aes(x = Latitude,
                           y = Elevation,
                           fill = as.factor(GenoGroups),
                           shape = as.factor(PhenoGroups))) +
  geom_point() +
  scale_fill_manual(values = myCols,
                    name = 'Genogroups',
                    labels = c('1','2','3','4','5','6','7','unknown')) +
  scale_shape_manual(values = myShapes,
                     name='Phenogroups',
                     labels = c('1','2','3','4','5','unknown')) +
  guides(fill = guide_legend(override.aes = list(fill = c(myCols[1:7],
                                                          'white'),
                                                 size = 3,
                                                 shape = 22,
                                                 color = 'black'))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21,22,23,24,25,8),
                                                  size = 3))) +
  theme(legend.key = element_rect(fill = NA),
        text = element_text(size = 8))

legend = cowplot::get_legend(plotForLegend)

#g3 + legend


##########
# GK TAU # ----------------------------------------------------------------------------------------
##########
fullOverlap = typesOfSpp %>%
  filter(PhenoGroups != 'NA' & GenoGroups != 'NA')

gktau_morph2molecules = GKtau(fullOverlap$PhenoGroups,
                              fullOverlap$GenoGroups)
gktau_molecules2morph = GKtau(fullOverlap$GenoGroups,
                              fullOverlap$PhenoGroups)

####################################
# PLOT both together # --------------------------------------------
####################################
# one row
f2 + g3 + legend + plot_layout(widths=c(4,3,1))

```
Return to [Clade III Data integration](#data-integration-3)
\pagebreak

## Clade IV
### Fig S36: Taxon sampling
```{r CladeIVVennDiagram, echo=FALSE,warning=FALSE, dpi=300, fig.width=4, fig.align='center', fig.cap="Specimens sampled according to three types of data. Specimens outside the Flowers category represent sterile specimens."}

CladeIV_Venn = escalloniaSampling %>%
  filter(CladeID == "IV") %>%
  mutate(Flowers = if_else(is.na(`Floral and Leaf`), 0, 1))

CladeIV_Venn = CladeIV_Venn %>%
  mutate(Leaves = if_else(is.na(`Leaf only`) & is.na(`Floral and Leaf`), 0,1))
CladeIV_Venn = CladeIV_Venn %>%
  mutate(Genes = if_else(is.na(libraryNumber), 0,1))

CladeIV_Venn = CladeIV_Venn %>%
  select(Flowers,Leaves, Genes)

VennForCladeIV = venn(CladeIV_Venn)

plot(VennForCladeIV,
     fills=list(fill = c('#c2a5cf',
                         '#d9f0d3',
                         '#f7f7f7')),
     quantities = list(TRUE,
                       cex = 1.5),
     legend=list(labels,
                 cex = 1))

```
Return to [Clade IV Sampling](#sampling-4)
\pagebreak

### Fig S37: Geographic distribution
```{r CladeIVsamplingMaps, echo=FALSE, dpi = 300, fig.align='center', fig.cap="Geographic distribution of specimens sampled for this clade according to taxonomic species. Filled symbols indicate specimens used in phenotypic analyses and empty symbols specimens used in genomic analyses."}
 #
myColors = "#cccccc"
myShapes = c(21,22,23,24,25,10)
# plot -------------------------------------------------------------
#
myrtilloidesMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'myrtilloides') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 3,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon=='myrtilloides') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 2,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-100, -55),  
              ylim = c(-30, 20)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label='Escallonia myrtilloides')


polifoliaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'polifolia') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 3,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon=='polifolia') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 2,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-100, -55),
              ylim = c(-30, 20)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label='Escallonia polifolia')


myrtilloidesMap + polifoliaMap + plot_layout(ncol = 2, guides = 'collect')

```
Return to [Clade IV Sampling](#sampling-4)
\pagebreak

```{r currentStateOfTaxonomyCladeIVMakePlots, echo=FALSE}

pal = c('#636363')
names(pal) = c('assignment')

# plot
CladeIV_propOverlap_All = ggplot(melted_CladeIV_overlapProp_All,
                                aes(x = X1, y = X2)) +
  geom_tile(aes(fill = value)) +
  geom_text(aes(label = round(value,
                              digits = 3)),
            color = 'darkgrey') +
  scale_fill_gradient(low = 'white',
                      high = "black") +
  theme(axis.title = element_blank(),
        axis.text = element_text(size = 6),
        axis.text.x = element_text(angle = -45,
                                   hjust = 0.1)) +
  theme(legend.text = element_text(size = 4),
        legend.margin = margin(2,2,6,2),
        legend.title = element_text(size = 6),
        legend.background = element_rect(color = "#cccccc")) +
  guides(fill = guide_colourbar(barwidth = 0.5,
                                barheight = 2))


CladeIV_inHullAssignments_All = ggplot(melted_CladeIV_hullResults_All,
                                      aes(x = uniqueID,
                                          y = variable,
                                          fill = col)) +
  geom_tile(aes(width = 0.99,
                height = 0.99),
            color = 'black') +
  scale_fill_manual(values = c("assignment" = '#636363',
                               "no assignment" = "white"),
                    name = NULL,
                    labels = c('yes','no'),
                    drop = FALSE) +
  scale_y_discrete(labels = c("In E. myrtilloides",
                              "In E. polifolia")) +
  scale_x_discrete(limits = factor(1:74)) + # limits equal to number of samples
  theme_bw() +
  theme(panel.ontop = TRUE,
        panel.background = element_rect(fill='transparent'),
        panel.border = element_blank(),
        panel.grid.major.y = element_blank(),
        panel.grid.major.x = element_blank(),
        axis.title.y = element_blank(),
        axis.title.x = element_blank(),
        axis.ticks.x = element_blank(),
        axis.text.x = element_blank(),
        legend.position = 'top',
        text = element_text(size = 6),
        plot.margin = margin(1,20,50,10),
        legend.key.height = unit(0.25,'cm'),
        legend.key.width = unit(0.25,'cm')) +
  coord_cartesian(ylim = c(1.1,2),
                  clip = 'off') +
  annotate(geom = 'text',
           x = (65/2),
           y = 0.1,
           label = 'As E. myrtilloides',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (65 + (74-65)/2),
           y = 0.1,
           label = 'As E. polifolia',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'segment',
           x = 1,
           xend = 65,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 66,
           xend = 74,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75)

```

### Fig S38: Current state of taxonomic species with phenotypic data
```{r currentStateOfTaxonomyCladeIVPlots, fig.width=7, fig.height = 5, dpi=200, echo=FALSE, fig.cap="Assessment of current state of taxonomic species with phenotypic data. Top panel: Pairwise overlap among 10-cubes describing geometrically each taxonomic species. Bottom panel: Matching-prediction analysis with each cell along the x-axis representing specimens sorted according to taxonomic species and the 10-cubes corresponding to each taxonomic species along the y-axis. If a specimen matches the prediction of the monograph (i.e., it is inside a 10-cube), the corresponding cell is shaded. If the specimen does not match the prediction, the cell is empty."}

(((CladeIV_propOverlap_All + plot_spacer())/ CladeIV_inHullAssignments_All) + plot_layout(tag_level='new'))

```
Return to [Clade IV Current state of taxonomic species](#current-state-of-taxonomic-species-3)
\pagebreak

### Fig S39: Phenogroup delimitation: Gaussian finite mixture modeling
```{r cladeIVmorphologicalDelimitationPlots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=6.5, fig.height=7, dpi=300, fig.cap='Gaussian finite mixture modeling (GFMM) for phenogroup delimitation and model selection using the Bayesian information criterion (BIC). Top panel: empirical support (ordinate) for Gaussian mixture models (GMM) assuming distinct number of phenogroups (abscissa). Each GMM specifies different number of phenogroups (shapes). Empirical support was measured as difference in BIC relative to the best model ($\\Delta$BIC = $0$). Bottom panel: Visualization of the phenogroups (shapes) identified by the best fit GMM; left panel shows phenogroups in the space defined by two axes obtained by linear discriminat analysis (to maximize separation and visualization), middle panel shows phenogroups in the space defined by discriminant axis 1 and latitude, and right panel shows phenogroups in the space defined by discriminant axis 1 and elevation.'}

CladeIV_modelsRanked = read.csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_modelsRanked.csv")
CladeIV_morphDR = read.csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_morphDR.csv")

max.BIC = -115.0039 # recorded during analysis


#plot
modelsRanked = ggplot(CladeIV_modelsRanked,
                      aes(x = numMorphGroups,
                          y = deltaBIC_naive)) +
  geom_point(aes(x = numMorphGroups,
                 y = deltaBIC_naive,
                 shape = "A_naive_models"),
             cex = 3,
             col = "black",
             bg = 'white') +
  # geom_point(aes(x=numMorphGroups[order(deltaBIC_naive)[1]],
  #                y=deltaBIC_naive[order(deltaBIC_naive)[1]],
  #                shape="B_best_naive_model"), cex=3.5, col='black', bg = 'black') +
  # geom_point(aes(x=numMorphGroups[order(deltaBIC_naive)[2]],
  #                y=deltaBIC_naive[order(deltaBIC_naive)[2]],
  #                shape ="C_second_naive_model"), cex=3.5, col='black',bg='grey') +
  geom_point(aes(x = numMorphGroups[taxonBIC != 'NA'],
                 y = max.BIC-taxonBIC[taxonBIC != 'NA'],
                 shape = "D_currentTax"),
             cex = 2.5,
             col = 'black',
             bg = 'red') +
  geom_point(aes(x = numMorphGroups[morphoGroupsBIC != 'NA'],
                 y = max.BIC-morphoGroupsBIC[morphoGroupsBIC != 'NA'],
                 shape = "E_morphoGroups"),
             cex = 2.5,
             col = 'black',
             bg = 'red') +
  scale_shape_manual(values = c(21,25,24),
                     labels = c("Naive models",
                              "Taxonomy model",
                              "Taxonomy unaware model"),
                     name = NULL) +
  guides(shape = guide_legend(override.aes = list(shape = c(21,25,24),
                                                  bg = c('white','red','red'),
                                                  color = c('black','black','black'),
                                                  size = 3))) +
  scale_x_discrete(limits=c(1,2,3)) +
  theme_bw() +
  theme(panel.grid = element_blank()) +
  xlab('Number of phenogroups') +
  ylab(expression(paste("Empirical support (", Delta, "BIC)", sep="")))


# plot ordination of best model
myShapes = c(21,22)
Q2 = ggplot(CladeIV_morphDR) +
  geom_point(aes(x = taxonomyDR_Dir1,
                 y = taxonomyDR_Dir2,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2')) +
  xlab("Dimension 1") +
  ylab("Dimension 2") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10),
        axis.ticks.y = element_blank()) +
  guides(shape = guide_legend(override.aes = list(size = 2)))


Q3 = ggplot(CladeIV_morphDR) +
  geom_point(aes(x = taxonomyDR_Dir1,
                 y = Latitude,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group 1',
                                'group 2')) +
  xlab("Dimension 1") +
  ylab("Latitude") +
  theme(legend.key = element_rect(fill=NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

Q4 = ggplot(CladeIV_morphDR) +
  geom_point(aes(x = taxonomyDR_Dir1,
                 y = Elevation, shape=as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name= 'Classification',
                     labels = c('group 1',
                                'group 2')) +
  xlab("Dimension 1") +
  ylab("Elevation") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))


# plot ordination of best model
myShapes = c(21,22,23)
Q5 = ggplot(CladeIV_morphDR) +
  geom_point(aes(x = MClustDR_Dir1,
                 y = MClustDR_Dir2,
                 shape = as.factor(BestNaiveModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3')) +
  xlab("Dimension 1") +
  ylab("Dimension 2") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10),
        axis.ticks.y = element_blank()) +
  guides(shape = guide_legend(override.aes = list(size = 2)))


Q6 = ggplot(CladeIV_morphDR) +
  geom_point(aes(x = MClustDR_Dir1,
                 y = Latitude,
                 shape = as.factor(BestNaiveModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group 1',
                                'group 2',
                                'group 3')) +
  xlab("Dimension 1") +
  ylab("Latitude") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

Q7 = ggplot(CladeIV_morphDR) +
  geom_point(aes(x = MClustDR_Dir1,
                 y = Elevation,
                 shape = as.factor(BestNaiveModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 2,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group 1',
                                'group 2',
                                'group 3')) +
  xlab("Dimension 1") +
  ylab("Elevation") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

(modelsRanked + plot_spacer() + plot_layout(widths = c(3,1))) /
  (Q2 + Q3 + Q4 + plot_layout(guides='collect')) /
  (Q5 + Q6 + Q7 + plot_layout(guides='collect'))

```
Return to [Clade IV Phenomics: model-based species discovery](#model-based-species-discovery-6)
\pagebreak

```{r, CladeIVgenomicSensitivityAnalyses_largest, echo=FALSE, warning = FALSE, message=FALSE, include=FALSE}

# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeIV_resultsFiles/CladeIV_largestMiss25_concat.contree")
load("supplementary_material_files/CladeIV_resultsFiles/CladeIV_largestMiss25_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeIV_resultsFiles/CladeIV_largestMiss25_ADMIX.3.Q")
testData = read_delim("supplementary_material_files/CladeIV_resultsFiles/CladeIV_largestMiss25_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_popFile_withLocality_ADMIXordered.csv", header=TRUE)
sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
   unite(col = 'SampleName',
         c(SpecimenID,
           toss1,
           taxon),
         sep = "_",
         remove = FALSE) %>%
  select(MolecularSampleName,
         SampleName,
         SpecimenID,
         libraryNumber,
         taxon,
         country,
         Elevation,
         Latitude,
         Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex=1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:12)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeIVgenomicSensitivityAnalyses_largest2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}
#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(brewer.pal(3, 'Greys'))

plot1 = ggtree(myTree_midRoot)

plot2 = plot1 + geom_tiplab(geom = 'text',
                            size = 1.0) +
  geom_treescale(fontsize = 1.0)

#plot2
plot3 = facet_plot(plot2,
                   data = admixMelt,
                   panel = 'Admixture proportions',
                   geom = geom_barh,
                   mapping = aes(x = as.numeric(value),
                               y = as.character(label),
                               fill = variable),
                    color='grey90',
                    stat='identity') +
  xlim_tree(0.02) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values=cols) +
  scale_y_discrete(limits=rev(myTipNames)) +
  theme(strip.text.x = element_text(size=4),
        legend.position = "none")
#plot3
plot4 = facet_widths(plot3, c(6,3))
#plot4

#########################
#####      PCA      #####
#########################
pch_v = c(21,22)
fastPCA_largest = ggplot(tab,
                         aes(x = EV1,
                             y = EV2,
                             shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.myrtilloides",
                                "E.polifolia"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size=unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_largest = as_tibble(testData %>%
                            dplyr::group_by(clusterNum) %>%
                            summarise_at("error", list(mean, sd)))

colnames(step1_largest) = c("clusterNum",
                            "mean",
                            "stdev")

step1_largest$me = sapply(step1_largest$stdev,
                          function(x) qt(.95,9)*x/sqrt(10))

testCI_largest = mutate(as_tibble(step1_largest),
                        lower = mean-me,
                        upper = mean+me)

ADMIX_CI_largest = ggplot(testCI_largest,
                          aes(x = clusterNum,
                              y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:3)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

#plot4 + (ADMIX_CI_largest / fastPCA_largest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation( title = "Largest data matrix: ~75% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))


```

```{r, CladeIVgenomicSensitivityAnalyses_middle, echo=FALSE, warning = FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeIV_resultsFiles/CladeIV_middleMiss50_concat.contree")
load("supplementary_material_files/CladeIV_resultsFiles/CladeIV_middleMiss50_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeIV_resultsFiles/CladeIV_middleMiss50_ADMIX.3.Q")
testData = read_delim("supplementary_material_files/CladeIV_resultsFiles/CladeIV_middleMiss50_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_popFile_withLocality_ADMIXordered.csv", header=TRUE)

sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep="_",
        remove = FALSE) %>%
   select(MolecularSampleName,
          SampleName,
          SpecimenID,
          libraryNumber,
          taxon,
          country,
          Elevation,
          Latitude,
          Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:12)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeIVgenomicSensitivityAnalyses_middle2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}
#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(brewer.pal(3, 'Greys'))

plot5 = ggtree(myTree_midRoot)

plot6 = plot5 +
  geom_tiplab(geom = 'text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot7 = facet_plot(plot6,
                   data = admixMelt,
                   panel = 'Admixture proportions',
                   geom = geom_barh,
                   mapping = aes(x = as.numeric(value),
                                 y = as.character(label),
                                 fill = variable),
                    color = 'grey90',
                    stat = 'identity') +
  xlim_tree(0.02) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")

plot8 = facet_widths(plot7, c(6,3))

#########################
#####      PCA      #####
#########################
pch_v = c(21,22)
fastPCA_middle = ggplot(tab,
                        aes(x = EV1,
                            y = EV2,
                            shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.myrtilloides",
                                "E.polifolia"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size=unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_middle = as_tibble(testData %>%
                           dplyr::group_by(clusterNum) %>%
                           summarise_at("error", list(mean, sd)))

colnames(step1_middle) = c("clusterNum", "mean", "stdev")

step1_middle$me = sapply(step1_middle$stdev,
                         function(x) qt(.95,9)*x/sqrt(10))

testCI_middle = mutate(as_tibble(step1_middle),
                       lower = mean-me,
                       upper = mean+me)

ADMIX_CI_middle = ggplot(testCI_middle,
                         aes(x = clusterNum,
                             y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:3)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

#plot8 + (ADMIX_CI_middle / fastPCA_middle) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(title = "Middle-sized data matrix: ~50% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))


```

```{r, CladeIVgenomicSensitivityAnalyses_smallest, echo=FALSE, warning = FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeIV_resultsFiles/CladeIV_smallestMiss75_concat.contree")
load("supplementary_material_files/CladeIV_resultsFiles/CladeIV_smallestMiss75_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeIV_resultsFiles/CladeIV_smallestMiss75_ADMIX.3.Q")
testData = read_delim("supplementary_material_files/CladeIV_resultsFiles/CladeIV_smallestMiss75_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_popFile_withLocality_ADMIXordered.csv", header=TRUE)

sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
   unite(col = 'SampleName',
         c(SpecimenID,
           toss1,
           taxon),
         sep = "_",
         remove = FALSE) %>%
   select(MolecularSampleName,
          SampleName,
          SpecimenID,
          libraryNumber,
          taxon,
          country,
          Elevation,
          Latitude,
          Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:12)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeIVgenomicSensitivityAnalyses_smallest2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}

#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(brewer.pal(3, 'Greys'))

plot9 = ggtree(myTree_midRoot)

plot10 = plot9 +
  geom_tiplab(geom = 'text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot11 = facet_plot(plot10,
                    data = admixMelt,
                    panel = 'Admixture proportions',
                    geom = geom_barh,
                    mapping = aes(x = as.numeric(value),
                                  y = as.character(label),
                                  fill = variable),
                    color = 'grey90',
                    stat = 'identity') +
  xlim_tree(0.02) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")

plot12 = facet_widths(plot11, c(6,3))

#########################
#####      PCA      #####
#########################
pch_v = c(21,22)
fastPCA_smallest = ggplot(tab,
                          aes(x = EV1,
                              y = EV2,
                              shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values=pch_v,
                     labels = c("E.myrtilloides",
                                "E.polifolia"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size=unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_smallest = as_tibble(testData %>%
                             dplyr::group_by(clusterNum) %>%
                             summarise_at("error", list(mean, sd)))

colnames(step1_smallest) = c("clusterNum", "mean", "stdev")

step1_smallest$me = sapply(step1_smallest$stdev,
                           function(x) qt(.95,9)*x/sqrt(10))

testCI_smallest = mutate(as_tibble(step1_smallest),
                         lower = mean-me,
                         upper = mean+me)

ADMIX_CI_smallest = ggplot(testCI_smallest,
                           aes(x = clusterNum,
                               y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:3)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))


#plot12 + (ADMIX_CI_smallest / fastPCA_smallest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(title = "Smallest data matrix: ~25% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))

```

### Fig S40: Sensitivity tests with $75\%$ missing data
```{r, cladeIVgenomicSensitivityAnalysesLargestplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($75\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot4 + (ADMIX_CI_largest / fastPCA_largest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade IV Genomics: sensitivity tests](#sensitivity-tests-3)
\pagebreak

### Fig S41: Sensitivity tests with $50\%$ missing data
```{r, cladeIVgenomicSensitivityAnalysesMiddleplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($50\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot8 + (ADMIX_CI_middle / fastPCA_middle) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade IV Genomics: sensitivity tests](#sensitivity-tests-3)
\pagebreak

### Fig S42: Sensitivity tests with $25\%$ missing data
```{r, cladeIVgenomicSensitivityAnalysesSmallestplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($25\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot12 + (ADMIX_CI_smallest / fastPCA_smallest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade IV Genomics: sensitivity tests](#sensitivity-tests-3)
\pagebreak

### Fig S43: Genogroup delimitation: Genotypic cluster model
```{r cladeIVmolecularDelimitationsNmms, echo=FALSE, message=FALSE, fig.width=7, fig.height=2.5, dpi=300, fig.cap='Gaussian finite mixture modeling (GFMM) for genogroup delimitation. Visualization of the genogroups (shades) identified by the best fit Gaussian mixture model (GMM). A) genogroups in the space defined by two axes obtained by non-metric multidimensional scaling (NMDS); B) genogroups in the space defined by NMDS axis 1 and latitude; C) genogroups in the space defined by NMDS axis 1 and elevation.'}

# read in results of NMM delimitation
CladeIV_nmms_temp = read.csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_smallestMiss75_NMMs_30Jul2020.csv")
# select classification and dimensions 1 and 2 from best model
CladeIV_nmms = CladeIV_nmms_temp %>% select(SpecimenID, Latitude, Longitude, Elevation, class.dim2.n0, NMDS1.dim2.n0, NMDS2.dim2.n0)
colnames(CladeIV_nmms)[5:7] = c('Classification','Dim1','Dim2')
# melt in prep for plotting
CladeIV_nmms_melted = reshape::melt(CladeIV_nmms, by='SpecimenID')

myColors = c('#f0f0f0','#bdbdbd','#636363')
# plot groups in first two dimensions
p1 = ggplot(CladeIV_nmms,
            aes(x = Dim1,
                y = Dim2)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              color = 'black',
              size = 3) +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3')) +
  xlab('Dimension 1') +
  ylab('Dimension 2') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))
# plot groups in dimension 1 against latitude
p2 = ggplot(CladeIV_nmms,
            aes(x = Dim1,
                y = Latitude)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              size = 3,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3')) +
  xlab('Dimension 1') +
  ylab('Latitude') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))
# plot groups in dimension1 against elevation
p3 = ggplot(CladeIV_nmms,
             aes(x = Dim1,
                 y = Elevation)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              size = 3,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3')) +
  xlab('Dimension 1') +
  ylab('Elevation') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))

p1 + p2 + p3 + plot_layout(guides = 'collect') + plot_annotation(tag_levels = 'A') & theme(plot.tag = element_text(size = 8))

```
Return to [Clade IV Genomics: model-based species discovery](#model-based-species-discovery-7)
\pagebreak

```{r cladeIVmolecularDelimitationsPhylogeny, echo=FALSE, include=FALSE}

myTree_midRoot = paintSubTree(myTree_midRoot, node=c(45), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(60), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(65), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(70), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(81), state='2', stem=FALSE)
```

### Fig S44: Genogroup delimitation. Cladogenesis to anagenesis model
```{r cladeIVmolecularDelimitationsPhylogeny2, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, out.height='.6\\textwidth', dpi=300, fig.show='asis', fig.cap='Phylogenetic modeling for genogroup delimitation. Midpoint-rooted phylogenetic tree showing genogroups in red. Values correspond to nodes at the transition point between cladogenesis (between species) to anagenesis (within species). Values closer to 0 indicate that the node was identified as a transition to anagenesis summarized over $500$ delimitations.'}
# at this point, the 'myTree_midRoot' is now of class 'simmap', so the plotting is a bit different
# because we want our trees without tip labels, i've set the fsize to be very small (essentially label-less)
# after plotting the tree with the 'painted' clades, add the custom node labels
colors = setNames(c('black','red'),c('1','2'))

plotSimmap(myTree_midRoot, setEnv = TRUE, fsize=0.001, lwd=1, colors=colors)

nodelabels(text='0.45', node=45, frame = 'none', adj=c(1.0,-0.50), cex=1)
nodelabels(text='0.48', node=60, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.003', node=65, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.07', node=70, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.04', node=81, frame = 'none', adj=c(1.5,-0.50), cex=1)

nodelabels(text='0.806', node=69, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.96', node=68, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='1.0', node=43, frame = 'none', adj=c(1.0,-0.50), cex=1)
nodelabels(text='0.99', node=44, frame = 'none', adj=c(1.5,-0.50), cex=1)
nodelabels(text='0.99', node=59, frame = 'none', adj=c(1.5,-0.50), cex=1)

# internal nodes

```
Return to [Clade IV Genomics: model-based species discovery](#model-based-species-discovery-7)
\pagebreak

### Fig S45: Genogroup delimitation: Reproductive isolation model
```{r cladeIVmolecularDelimitationPopGenBpp, echo=FALSE, fig.width=6.5, dpi=300, fig.cap='Population genetic modeling for genogroup delimitation. Top panel: assignment of specimens to demes according to \\texttt{STRUCTURE} and posterior probability of species delimitation modeling according to \\texttt{BPP} using these demes. Bottom panel: assignment of specimens to demes according to \\texttt{MAVERICK} and posterior probability of species delimitation modeling according to \\texttt{BPP} using these demes. Specimens are sorted from north (left) to south (right) according to locality of collection.'}

mySTR = read.csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_smallestMiss75_STRUCTURE.csv", header=TRUE)
myRMav = read.csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_smallestMiss75_RMav.csv", header=TRUE)


# attach assignments
# IMPORTANT: make sure to confirm number of columns you need to attach (corresponds with number of groups to be plotted)
# number of STRUCTURE groups = 2
# number of RMaverick groups = 2
mergedGroupInfo = cbind(sampleInfo, mySTR[match(sampleInfo$MolecularSampleName, mySTR[,'MolecularSampleName']),2:3])
mergedGroupInfo = cbind(mergedGroupInfo, myRMav[match(sampleInfo$MolecularSampleName, myRMav[,'MolecularSampleName']),2:4])

# determine majority group assignment for each individual and attach these to the group information
# STRUCTURE groups are columns 11:12
# RMaverick groups are columns 13:15
mergedGroupInfo$STRassignment = as.factor(colnames(mergedGroupInfo[,10:11]) [apply(mergedGroupInfo[,10:11],1,which.max)])
mergedGroupInfo$RMavassignment = as.factor(colnames(mergedGroupInfo[,12:14]) [apply(mergedGroupInfo[,12:14],1,which.max)])


myGreys = c("#f7f7f7","#cccccc")
myBlues = c("#deebf7","#9ecae1","#3182bd")

STRplot = reshape::melt(mergedGroupInfo,
                        id.vars=c("SpecimenID",
                                  "Latitude"),
                        measure.vars=c("group1",
                                       "group2")) %>%
  ggplot(aes(x = reorder(as.factor(SpecimenID), -Latitude),
             y = value,
             fill=as.factor(variable))) +
  geom_bar(stat = 'identity',
           show.legend = FALSE,
           color = 'black') +
  scale_fill_manual(values = myGreys[c(1,2)]) +
  ylab("Probability") +
  theme_bw() +
  #theme(axis.text.x = element_text(size=8, angle=90, vjust = 0.5, hjust=1)) +
  theme(axis.text.x = element_blank(),
        axis.title.x = element_blank()) +
  theme(text = element_text(size = 10))

RMavplot = reshape::melt(mergedGroupInfo,
                         id.vars=c("SpecimenID",
                                   "Latitude"),
                         measure.vars = c("deme3",
                                          "deme2",
                                          "deme1")) %>%
  ggplot(aes(x = reorder(as.factor(SpecimenID), -Latitude),
             y = value,
             fill = as.factor(variable))) +
  geom_bar(stat = 'identity',
           show.legend = FALSE,
           color = 'black') +
  scale_fill_manual(values = myBlues[c(1,2,3)]) +
  ylab("Probability") +
  xlab("Specimens sorted by Latitudinal position") +
  theme_bw() +
  theme(axis.text.x = element_text(size = 4,
                                   angle = 90,
                                   vjust = 0.5,
                                   hjust = 1)) +
  theme(text = element_text(size = 10))
  #theme(axis.text.x = element_blank(), axis.title.x = element_blank())

(STRplot + grid::textGrob("BPP post. probability\n for 2 species = 1.0") + plot_layout(widths = c(3,1))) / (RMavplot + grid::textGrob("BPP post. probability\n for 2 species = 1.0") + plot_layout(widths = c(3,1)))
```
Return to [Clade IV Genomics: model-based species discovery](#model-based-species-discovery-7)
\pagebreak

```{r cladeIVfinalDelimitation, echo=FALSE, include=FALSE}
# need to use the output of the morphometrics analysis, since it only includes samples that have BOTH flowers and leaves
morphDelim = read.csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_morphDR.csv")

molecularDelim = read.csv("supplementary_material_files/CladeIV_resultsFiles/CladeIV_molecularDelimitation.csv")
# if it doesn't have a SpecimenID column, you'll need to make one
molecularDelim = molecularDelim %>%
  separate(col = SampleName,
           into = c('SpecimenID',
                  'toss1',
                  'taxon',
                  'LibraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep = "_",
        remove = FALSE) %>%
  unite(col = 'MolecularSampleName',
        c(SpecimenID,
          toss1,
          taxon,
          LibraryNumber),
        sep = "_",
        remove = FALSE)

# join datasets, coalesce duplicate columns, select needed data
typesOfSpp_temp = full_join(morphDelim, molecularDelim, by = "SpecimenID") %>%
  mutate(
    Latitude = coalesce(Latitude.x, Latitude.y),
    Longitude = coalesce(Longitude.x, Longitude.y),
    Elevation = coalesce(Elevation.x, Elevation.y)
  ) %>%
  select(MolecularSampleName,
         SpecimenID,
         Elevation,
         Latitude,
         Longitude,
         BestModel_Classification, #name of the morphoDelim column
         mPTP) # name of the molecularDelim column
# confirmation
#colnames(typesOfSpp_temp)
#view(typesOfSpp_temp) # confirm number of rows matches that of sampling table


# create plotting df
typesOfSpp = typesOfSpp_temp
colnames(typesOfSpp) = c("MolecularSampleName",
                         "SpecimenID",
                         "Elevation",
                         "Latitude",
                         "Longitude",
                         "PhenoGroups",
                         "GenoGroups")


# plot tree and match dataframe to tip names - ensures order of plotting is correct
myTree = read.tree("supplementary_material_files/CladeIV_resultsFiles/CladeIV_smallestMiss75_concat.contree")
myTree_midRoot = midpoint.root(myTree)
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()
typesOfSpp = typesOfSpp[order(match(typesOfSpp$MolecularSampleName, myTipNames)),]

```

### Fig S46: Data integration
```{r cladeIVfinalDelimitation2, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, fig.height=5, dpi=300, fig.cap="Integration of phenotypic and genomic data with spatial information and evolutionary history. All specimens are assigned to their corresponding best fit phenogroup (shapes) and genogroup (colors). Specimens without phenotypic or genomic data (unknown specimens) are shown as asterisks and empty shapes, accordingly. Specimens are shown as tips of the maximum likelihood tree (left) used in the CA model analysis and mapped along latitude and elevation (right). Specimens assigned to a single phenogroup and a single genogroup delineate species that we determined as 'good species'. Specimens assigned to a single phenogroup across multiple genogroups delineate species that we determined as 'phenotypic cryptic species'. Specimens assigned to a single genogroup across multiple phenogroups delineate species that we determined as 'genetic cryptic species'."}
########################
# Phylogenetic context # ----------------------------------------------------------------------------------------
########################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups

# get number of colors and shapes that you'll need for plotting
#unique(typesOfSpp$PhenoGroups)
#unique(typesOfSpp$GenoGroups)

# set values for colors and shapes
myShapes = c(21,22,8) #the 8 is for unknown
names(myShapes) = c('5','6','NA')
myCols = c(brewer.pal(n = 6,'Dark2'),"#ffffff")
names(myCols) = c('group1','group2','group3','group4','group5','group6','NA')



f = ggtree(myTree_midRoot) %<+% typesOfSpp + xlim_tree(0.005)
#f
f2 = f + geom_tippoint(aes(x = x,
                           fill = as.factor(GenoGroups),
                           shape = as.factor(PhenoGroups)),
                       size = 3) +
  scale_shape_manual(values = myShapes) +
  scale_fill_manual(values = myCols) +
  geom_tiplab(aes(label = SpecimenID),
              hjust = -0.3,
              size = 2) +
  geom_point2(aes(subset = is.na(PhenoGroups) & isTip,
                  color = as.factor(GenoGroups)),
              pch = 8,
              size = 2,
              stroke = 0.5,
              alpha = 0.5) +
  scale_color_manual(values = myCols) +
  theme(legend.position = "none")
#f2


###################
# Spatial context # ----------------------------------------------------------------------------------------
###################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups


##### plot individuals with GenoGroup assignment, but no PhenoGroup assignment
g = ggplot(typesOfSpp,
           aes(x = Latitude,
               y = Elevation)) +
  geom_jitter(data = typesOfSpp %>%
                filter(is.na(PhenoGroups)),
              aes(x = Latitude,
                  y = Elevation,
                  color = as.factor(GenoGroups)),
              size = 2,
              alpha = 0.5,
              width = 1.5,
              height = 1.5,
              pch = 8,
              stroke = 1) +
  scale_color_manual(values = myCols) +
  scale_x_reverse() +
  theme_bw() +
  theme(legend.position = 'none') +
  theme(panel.grid.major = element_line(size = 0.55, color = '#f0f0f0'))
#g

##### plot individuals with PhenoGroup assignment, but no GenoGroup assignment
g2 = g +
  geom_jitter(data=typesOfSpp %>%
                filter(is.na(GenoGroups)),
              aes(x = Latitude,
                  y = Elevation,
                  shape = as.factor(PhenoGroups)),
              fill = 'white',
              size = 2,
              width = 1.5,
              height = 1.5) +
  scale_shape_manual(values = myShapes)
#g2

###### plots fully overlapping data (has both Geno- and Pheno- group assignments)
g3 = g2 +
  geom_jitter(data = typesOfSpp %>%
                filter(GenoGroups != 'NA' & PhenoGroups != 'NA'),
              aes(x = Latitude,
                  y = Elevation,
                  fill = as.factor(GenoGroups),
                  shape = as.factor(PhenoGroups)),
              size = 3,
              width = 1.5,
              height = 1.5) +
  scale_fill_manual(values = myCols)
#g3

###### create the plot and save to an object
plotForLegend = ggplot(typesOfSpp,
                       aes(x = Latitude,
                           y = Elevation,
                           fill = as.factor(GenoGroups),
                           shape = as.factor(PhenoGroups))) +
  geom_point() +
  scale_fill_manual(values = myCols,
                    name = 'Genogroups',
                    labels = c('1','2','3','4','5','6','unknown')) +
  scale_shape_manual(values = myShapes,
                     name = 'Phenogroups',
                     labels = c('1','2','unknown')) +
  guides(fill = guide_legend(override.aes = list(fill=c(myCols[1:6],
                                                        'white'),
                                                 size = 3,
                                                 shape = 22,
                                                 color = 'black'))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21,22,8),
                                                  size = 3))) +
  theme(legend.key = element_rect(fill = NA),
        text = element_text(size = 8))

legend = cowplot::get_legend(plotForLegend)

#g3 + legend


##########
# GK TAU # ----------------------------------------------------------------------------------------
##########
fullOverlap = typesOfSpp %>%
  filter(PhenoGroups != 'NA' & GenoGroups != 'NA')

gktau_morph2molecules = GKtau(fullOverlap$PhenoGroups,
                              fullOverlap$GenoGroups)
gktau_molecules2morph = GKtau(fullOverlap$GenoGroups,
                              fullOverlap$PhenoGroups)


####################################
# PLOT both together # --------------------------------------------
####################################
# one row
f2 + g3 + legend + plot_layout(widths = c(4,3,1))


```
Return to [Clade IV Data integration](#data-integration-4)
\pagebreak

## Clade V
### Fig S47: Taxon sampling
```{r CladeVVennDiagram, echo=FALSE, dpi=300, fig.width=4, fig.align='center', fig.cap="Specimens sampled according to three types of data. Specimens outside the Flowers category represent sterile specimens."}

CladeV_Venn = escalloniaSampling %>%
  filter(CladeID == "V") %>%
  mutate(Flowers = if_else(is.na(`Floral and Leaf`), 0, 1))

CladeV_Venn = CladeV_Venn %>%
  mutate(Leaves = if_else(is.na(`Leaf only`) & is.na(`Floral and Leaf`), 0,1))

CladeV_Venn = CladeV_Venn %>%
  mutate(Genes = if_else(is.na(libraryNumber), 0,1))

CladeV_Venn = CladeV_Venn %>%
  select(Flowers,Leaves, Genes)

VennForCladeV = venn(CladeV_Venn)

plot(VennForCladeV,
     fills = list(fill=c('#c2a5cf','#d9f0d3','#f7f7f7')),
     quantities = list(TRUE,
                       cex = 1.5),
     legend = list(labels,
                   cex = 1))

```
Return to [Clade V Sampling](#sampling-5)
\pagebreak

### Fig S48: Geographic distribution
```{r CladeVsamplingMaps, echo=FALSE, dpi = 300, fig.width=7, fig.align='center',fig.cap="Geographic distribution of specimens sampled for this clade according to taxonomic species. Filled symbols indicate specimens used in phenotypic analyses and empty symbols specimens used in genomic analyses."}
myColors = "#cccccc"
myShapes = c(21,22,23,24,25,10)
alpinaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'alpina') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'alpina') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -55),  
              ylim = c(-55, -28)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face='italic'),
        text = element_text(size = 6)) +
  ggtitle(label='Escallonia alpina')


floridaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'florida') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'florida') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -55),
              ylim = c(-55, -28)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia florida')

leucanthaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'leucantha') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'leucantha') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -55),  
              ylim = c(-55, -28)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia leucantha')

myrtoideaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'myrtoidea') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color='black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'myrtoidea') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -55),
              ylim = c(-55, -28)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia myrtoidea')

revolutaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'revoluta') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'revoluta') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -55),  
              ylim = c(-55, -28)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia revoluta')

roseaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'rosea') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon=='rosea') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -55),
              ylim = c(-55, -28)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia rosea')

rubraMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'rubra') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'rubra') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -55),  
              ylim = c(-55, -28)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia rubra')


alpinaMap + floridaMap + leucanthaMap +
  myrtoideaMap + revolutaMap + roseaMap + rubraMap + plot_layout(ncol = 4, guides='collect')

```
Return to [Clade V Sampling](#sampling-5)
\pagebreak

```{r currentStateOfTaxonomyCladeVMakePlots, echo=FALSE}

pal = c('#636363')
names(pal) = c('assignment')

# plot
CladeV_propOverlap_All = ggplot(melted_CladeV_overlapProp_All,
                                aes(x = X1,
                                    y = X2)) +
  geom_tile(aes(fill = value)) +
  geom_text(aes(label = round(value,
                              digits = 3)),
            color = 'darkgrey',
            size = 1) +
  scale_fill_gradient(low = 'white',
                      high = "black") +
  theme(axis.title = element_blank(),
        axis.text = element_text(size = 6),
        axis.text.x = element_text(angle = -45,
                                   hjust = 0.1)) +
  theme(legend.text = element_text(size = 4),
        legend.margin = margin(2,2,6,2),
        legend.title = element_text(size = 6),
        legend.background = element_rect(color = "#cccccc")) +
  guides(fill = guide_colourbar(barwidth = 0.5,
                                barheight = 2))


CladeV_inHullAssignments_All = ggplot(melted_CladeV_hullResults_All,
                                      aes(x = uniqueID,
                                          y = variable,
                                          fill = col)) +
  geom_tile(aes(width = 0.99,
                height = 0.99),
            color = 'black') +
  scale_fill_manual(values = c("assignment" = '#636363',
                               "no assignment" = "white"),
                    name = NULL,
                    labels = c('yes','no'),
                    drop = FALSE) +
  scale_y_discrete(labels = c("In E. alpina",
                            "In E. florida",
                            "In E. leucantha",
                            "In E. myrtoidea",
                            "In E. revoluta",
                            "In E. rosea",
                            "In E. rubra")) +
  scale_x_discrete(limits = factor(1:214)) + # limits equal to number of samples
  theme_bw() +
  theme(panel.ontop = TRUE,
        panel.background = element_rect(fill='transparent'),
        panel.border = element_blank(),
        panel.grid.major.y = element_blank(),
        panel.grid.major.x = element_blank(),
        axis.title.y = element_blank(),
        axis.title.x = element_blank(),
        axis.ticks.x = element_blank(),
        axis.text.x = element_blank(),
        legend.position = 'top',
        text = element_text(size = 6),
        plot.margin = margin(1,20,30,10),
        legend.key.height = unit(0.25,'cm'),
        legend.key.width = unit(0.25,'cm')) +
  coord_cartesian(ylim = c(1.1,7),
                  clip = 'off') +
  annotate(geom = 'text',
           x = (43/2),
           y = 0.1,
           label = 'As E. alpina',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (44 + (65-44)/2),
           y = 0.1,
           label = 'As E. florida',
           size = 2,
           angle = 320,
           hjust = 0) +  
  annotate(geom = 'text',
           x = (66 + (86-66)/2),
           y = 0.1,
           label = 'As E. leucantha',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (87 + (103-87)/2),
           y = 0.1,
           label = 'As E. myrtoidea',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (104 + (125-104)/2),
           y = 0.1,
           label = 'As E. revoluta',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (126 + (159-126)/2),
           y = 0.1,
           label = 'As E. rosea',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (160 + (214-160)/2),
           y = 0.1,
           label = 'As E. rubra',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'segment',
           x = 1,
           xend = 43,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 44,
           xend = 65,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 66,
           xend = 86,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 87,
           xend = 103,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 104,
           xend = 125,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 126,
           xend = 159,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 160,
           xend = 214,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75)

```
\pagebreak

### Fig S49: Current state of taxonomic species with phenotypic data
```{r currentStateOfTaxonomyCladeVPlots, fig.width=7, fig.height = 5, dpi=200, echo=FALSE, fig.cap="Assessment of current state of taxonomic species with phenotypic data. Top panel: Pairwise overlap among 10-cubes describing geometrically each taxonomic species. Bottom panel: Matching-prediction analysis with each cell along the x-axis representing specimens sorted according to taxonomic species and the 10-cubes corresponding to each taxonomic species along the y-axis. If a specimen matches the prediction of the monograph (i.e., it is inside a 10-cube), the corresponding cell is shaded. If the specimen does not match the prediction, the cell is empty."}

(((CladeV_propOverlap_All + plot_spacer())/ CladeV_inHullAssignments_All) + plot_layout(tag_level='new'))

```
Return to [Clade V Current state of taxonomic species](#current-state-of-taxonomic-species-4)
\pagebreak

### Fig S50: Phenogroup delimitation: Gaussian finite mixture modeling
```{r cladeVmorphologicalDelimitationPlots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=6.5, dpi=300, fig.cap='Gaussian finite mixture modeling (GFMM) for phenogroup delimitation and model selection using the Bayesian information criterion (BIC). Top panel: empirical support (ordinate) for Gaussian mixture models (GMM) assuming distinct number of phenogroups (abscissa). Each GMM specifies different number of phenogroups (shapes). Empirical support was measured as difference in BIC relative to the best model ($\\Delta$BIC = $0$). Bottom panel: Visualization of the phenogroups (shapes) identified by the best fit GMM; left panel shows phenogroups in the space defined by two axes obtained by linear discriminat analysis (to maximize separation and visualization), middle panel shows phenogroups in the space defined by discriminant axis 1 and latitude, and right panel shows phenogroups in the space defined by discriminant axis 1 and elevation.'}

CladeV_modelsRanked = read.csv("supplementary_material_files/CladeV_resultsFiles/CladeV_modelsRanked.csv")
CladeV_morphDR = read.csv("supplementary_material_files/CladeV_resultsFiles/CladeV_morphDR.csv")
max.BIC = -516.7234

#plot
modelsRanked = ggplot(CladeV_modelsRanked,
                      aes(x = numMorphGroups,
                          y = deltaBIC_naive)) +  
  geom_point(aes(x = numMorphGroups,
                 y = deltaBIC_naive,
                 shape = "A_naive_models"),
             cex = 3,
             col = "black",
             bg = 'white') +
  geom_point(aes(x = numMorphGroups[order(deltaBIC_naive)[1]],
                 y = deltaBIC_naive[order(deltaBIC_naive)[1]],
                 shape = "B_best_naive_model"),
             cex = 3,
             col = 'black',
             bg = 'black') +
  # geom_point(aes(x=numMorphGroups[order(deltaBIC_naive)[2]],
  #                y=deltaBIC_naive[order(deltaBIC_naive)[2]],
  #                shape ="C_second_naive_model"), cex=3, col='black',bg='grey') +
  geom_point(aes(x = numMorphGroups[taxonBIC != 'NA'],
                 y = max.BIC-taxonBIC[taxonBIC != 'NA'],
                 shape = "D_currentTax"),
             cex = 3,
             col = 'black',
             bg = 'red') +
  geom_point(aes(x = numMorphGroups[morphoGroupsBIC != 'NA'],
                 y = max.BIC-morphoGroupsBIC[morphoGroupsBIC != 'NA'],
                 shape = "E_morphoGroups"),
             cex = 3,
             col = 'black',
             bg = 'red') +
  scale_shape_manual(values = c(21,21,25,24),
                     labels = c("Best fit model",
                              "Naive models",
                              "Taxonomy model",
                              "Taxonomy unaware model"),
                     name = NULL) +
  guides(shape = guide_legend(override.aes = list(shape = c(21,21,25,24),
                                                  bg = c('black',
                                                         'white',
                                                         'red',
                                                         'red'),
                                                  color = c('black',
                                                            'black',
                                                            'black',
                                                            'black'),
                                                  size = 3))) +
  scale_x_discrete(limits = c(1,2,3,4,5,6,7,8,9,10)) +
  theme_bw() +
  theme(panel.grid = element_blank()) +
  xlab('Number of phenogroups') +
  ylab(expression(paste("Empirical support (", Delta, "BIC)", sep="")))



# plot ordination of best naive model
myShapes = c(21,22,23,24,25,10,12,9)
Q2 = ggplot(CladeV_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = MclustDR_Dir2,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 1,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group1',
                                'group2',
                                'group3',
                                'group4',
                                'group5',
                                'group6',
                                'group7',
                                'group8')) +
  xlab("Dimension 1") +
  ylab("Dimension 2") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

Q3 = ggplot(CladeV_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = Latitude, shape=as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 1,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group1',
                                'group2',
                                'group3',
                                'group4',
                                'group5',
                                'group6',
                                'group7',
                                'group8')) +
  xlab("Dimension 1") +
  ylab("Latitude") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

Q4 = ggplot(CladeV_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = Elevation,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg= '#cccccc',
             size = 1,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group1',
                                'group2',
                                'group3',
                                'group4',
                                'group5',
                                'group6',
                                'group7',
                                'group8')) +
  xlab("Dimension 1") +
  ylab("Elevation") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

(modelsRanked + plot_spacer() + plot_layout(widths=c(3,1))) /
  (Q2 + Q3 + Q4 + plot_layout(guides='collect'))

```
Return to [Clade V Phenomics: model-based species discovery](#model-based-species-discovery-8)
\pagebreak

```{r, CladeVgenomicSensitivityAnalyses_largest, echo=FALSE, warning = FALSE, message=FALSE, include=FALSE}

# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeV_resultsFiles/CladeV_largestMiss25_concat.contree")
load("supplementary_material_files/CladeV_resultsFiles/CladeV_largestMiss25_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeV_resultsFiles/CladeV_largestMiss25_ADMIX.11.Q")
testData = read_delim("supplementary_material_files/CladeV_resultsFiles/CladeV_largestMiss25_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeV_resultsFiles/CladeV_popFile_withLocality_ADMIXordered.csv", header=TRUE)

sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
   unite(col = 'SampleName',
         c(SpecimenID,
           toss1,
           taxon),
         sep="_",
         remove = FALSE) %>%
   select(MolecularSampleName,
          SampleName,
          SpecimenID,
          libraryNumber,
          taxon,
          country,
          Elevation,
          Latitude,
          Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3","ADMIXgroup4","ADMIXgroup5","ADMIXgroup6","ADMIXgroup7","ADMIXgroup8","ADMIXgroup9","ADMIXgroup10","ADMIXgroup11")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:20)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeVgenomicSensitivityAnalyses_largest2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}
#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(c(brewer.pal(6, 'Greys')),c(brewer.pal(5, 'Blues')))

plot1 = ggtree(myTree_midRoot)

plot2 = plot1 + geom_tiplab(geom = 'text',
                            size = 1.0) +
  geom_treescale(fontsize = 1.0)
#plot2

plot3 = facet_plot(plot2,
                   data = admixMelt,
                   panel = 'Admixture proportions',
                   geom = geom_barh,
                   mapping = aes(x = as.numeric(value),
                                 y = as.character(label),
                                 fill = variable),
                   color = 'grey90',
                   stat = 'identity') +
  xlim_tree(0.01) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")
#plot3
plot4 = facet_widths(plot3, c(6,3))
#plot4

#########################
#####      PCA      #####
#########################
pch_v = c(21,22,23,24,25,10,12)
fastPCA_largest = ggplot(tab,
                         aes(x = EV1,
                             y = EV2,
                             shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.alpina",
                                "E.florida",
                                "E.leucantha",
                                "E.myrtoidea",
                                "E.revoluta",
                                "E.rosea",
                                "E.rubra"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size=unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_largest = as_tibble(testData %>%
                            dplyr::group_by(clusterNum) %>%
                            summarise_at("error", list(mean, sd)))

colnames(step1_largest) = c("clusterNum", "mean", "stdev")

step1_largest$me = sapply(step1_largest$stdev,
                          function(x) qt(.95,9)*x/sqrt(10))

testCI_largest = mutate(as_tibble(step1_largest),
                        lower = mean-me,
                        upper = mean+me)

ADMIX_CI_largest = ggplot(testCI_largest,
                          aes(x = clusterNum,
                              y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:11)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

#plot4 + (ADMIX_CI_largest / fastPCA_largest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation( title = "Largest data matrix: ~75% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))


```

```{r, CladeVgenomicSensitivityAnalyses_middle, echo=FALSE, warning = FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeV_resultsFiles/CladeV_middleMiss50_concat.contree")
load("supplementary_material_files/CladeV_resultsFiles/CladeV_middleMiss50_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeV_resultsFiles/CladeV_middleMiss50_ADMIX.9.Q")
testData = read_delim("supplementary_material_files/CladeV_resultsFiles/CladeV_middleMiss50_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeV_resultsFiles/CladeV_popFile_withLocality_ADMIXordered.csv", header=TRUE)
sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into=c('SpecimenID',
                  'toss1',
                  'taxon',
                  'libraryNumber'),
           remove = FALSE) %>%
   unite(col = 'SampleName',
         c(SpecimenID,
           toss1,
           taxon),
         sep = "_",
         remove = FALSE) %>%
  select(MolecularSampleName,
         SampleName,
         SpecimenID,
         libraryNumber,
         taxon,
         country,
         Elevation,
         Latitude,
         Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3","ADMIXgroup4","ADMIXgroup5","ADMIXgroup6","ADMIXgroup7","ADMIXgroup8","ADMIXgroup9")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:18)], id =c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeVgenomicSensitivityAnalyses_middle2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}
#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(c(brewer.pal(5, 'Greys')),c(brewer.pal(4, 'Blues')))  
plot5 = ggtree(myTree_midRoot)

plot6 = plot5 + geom_tiplab(geom = 'text',
                            size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot7 = facet_plot(plot6,
                   data = admixMelt,
                   panel = 'Admixture proportions',
                   geom = geom_barh,
                   mapping = aes(x = as.numeric(value),
                                 y = as.character(label),
                                 fill = variable),
                   color = 'grey90',
                   stat = 'identity') +
  xlim_tree(0.01) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")

plot8 = facet_widths(plot7, c(6,3))

#########################
#####      PCA      #####
#########################
pch_v = c(21,22,23,24,25,10,12)

fastPCA_middle = ggplot(tab,
                        aes(x = EV1,
                            y = EV2,
                            shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.alpina",
                                "E.florida",
                                "E.leucantha",
                                "E.myrtoidea",
                                "E.revoluta",
                                "E.rosea",
                                "E.rubra"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size=unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_middle = as_tibble(testData %>%
                           dplyr::group_by(clusterNum) %>%
                           summarise_at("error", list(mean, sd)))

colnames(step1_middle) = c("clusterNum",
                           "mean",
                           "stdev")

step1_middle$me = sapply(step1_middle$stdev,
                         function(x) qt(.95,9)*x/sqrt(10))

testCI_middle = mutate(as_tibble(step1_middle),
                       lower = mean-me,
                       upper = mean+me)

ADMIX_CI_middle = ggplot(testCI_middle,
                         aes(x = clusterNum,
                             y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:11)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

#plot8 + (ADMIX_CI_middle / fastPCA_middle) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(title = "Middle-sized data matrix: ~50% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))


```

```{r, CladeVgenomicSensitivityAnalyses_smallest, echo=FALSE, warning = FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeV_resultsFiles/CladeV_smallestMiss75_concat.contree")
load("supplementary_material_files/CladeV_resultsFiles/CladeV_smallestMiss75_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeV_resultsFiles/CladeV_smallestMiss75_ADMIX.11.Q")
testData = read_delim("supplementary_material_files/CladeV_resultsFiles/CladeV_smallestMiss75_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeV_resultsFiles/CladeV_popFile_withLocality_ADMIXordered.csv", header=TRUE)

sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
   unite(col = 'SampleName',
         c(SpecimenID,
           toss1,
           taxon),
         sep = "_",
         remove = FALSE) %>%
  select(MolecularSampleName,
         SampleName,
         SpecimenID,
         libraryNumber,
         taxon,
         country,
         Elevation,
         Latitude,
         Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3","ADMIXgroup4","ADMIXgroup5","ADMIXgroup6","ADMIXgroup7","ADMIXgroup8","ADMIXgroup9","ADMIXgroup10","ADMIXgroup11")

mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:20)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeVgenomicSensitivityAnalyses_smallest2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}

#########################

##### PHYLO + ADMIX BARS #####

###########################

cols = c(c(brewer.pal(6, 'Greys')),
         c(brewer.pal(5, 'Blues')))

plot9 = ggtree(myTree_midRoot)

plot10 = plot9 + geom_tiplab(geom = 'text',
                             size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot11 = facet_plot(plot10,
                    data = admixMelt,
                    panel = 'Admixture proportions',
                    geom = geom_barh,
                    mapping = aes(x = as.numeric(value),
                                  y = as.character(label),
                                  fill = variable),
                    color = 'grey90',
                    stat = 'identity') +
  xlim_tree(0.01) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits=rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")
plot12 = facet_widths(plot11, c(6,3))

#########################
#####      PCA      #####
#########################
pch_v = c(21,22,23,24,25,10,12)

fastPCA_smallest = ggplot(tab,
                          aes(x = EV1,
                              y = EV2,
                              shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.alpina",
                                "E.florida",
                                "E.leucantha",
                                "E.myrtoidea",
                                "E.revoluta",
                                "E.rosea",
                                "E.rubra"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size = unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################
step1_smallest = as_tibble(testData %>%
                             dplyr::group_by(clusterNum) %>%
                             summarise_at("error", list(mean, sd)))

colnames(step1_smallest) = c("clusterNum", "mean", "stdev")

step1_smallest$me = sapply(step1_smallest$stdev,
                           function(x) qt(.95,9)*x/sqrt(10))

testCI_smallest = mutate(as_tibble(step1_smallest),
                         lower = mean-me,
                         upper = mean+me)

ADMIX_CI_smallest = ggplot(testCI_smallest,
                           aes(x = clusterNum,
                               y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:11)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))


#plot12 + (ADMIX_CI_smallest / fastPCA_smallest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(title = "Smallest data matrix: ~25% missing data", tag_levels = 'A') & theme(plot.tag = element_text(size=6), plot.title = element_text(size=8))

```

### Fig S51: Sensitivity tests with $75\%$ missing data
```{r, cladeVgenomicSensitivityAnalysesLargestplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($75\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot4 + (ADMIX_CI_largest / fastPCA_largest) + plot_layout(widths = c(4,2), heights = c(50,0.25)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade V Genomics: sensitivity tests](#sensitivity-tests-4)
\pagebreak

### Fig S52: Sensitivity tests with $50\%$ missing data
```{r, cladeVgenomicSensitivityAnalysesMiddleplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($50\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot8 + (ADMIX_CI_middle / fastPCA_middle) + plot_layout(widths = c(4,2), heights = c(50,0.25)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade V Genomics: sensitivity tests](#sensitivity-tests-4)
\pagebreak

### Fig S53: Sensitivity tests with $25\%$ missing data
```{r, cladeVgenomicSensitivityAnalysesSmallestplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($25\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot12 + (ADMIX_CI_smallest / fastPCA_smallest) + plot_layout(widths = c(4,2), heights = c(50,0.25)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))

```
Return to [Clade V Genomics: sensitivity tests](#sensitivity-tests-4)
\pagebreak

### Fig S54: Genogroup delimitation: Genotypic cluster model
```{r cladeVmolecularDelimitationsNmms, echo=FALSE, message=FALSE, fig.width=7, fig.height=2.5, dpi=300, fig.cap='Gaussian finite mixture modeling (GFMM) for genogroup delimitation. Visualization of the genogroups (shades) identified by the best fit Gaussian mixture model (GMM). A) genogroups in the space defined by two axes obtained by non-metric multidimensional scaling (NMDS); B) genogroups in the space defined by NMDS axis 1 and latitude; C) genogroups in the space defined by NMDS axis 1 and elevation.'}
# read in results of NMM delimitation
CladeV_nmms_temp = read.csv("supplementary_material_files/CladeV_resultsFiles/CladeV_smallestMiss75_NMMs.csv")
# select classification and dimensions 1 and 2 from best model
CladeV_nmms = CladeV_nmms_temp %>%
  select(SpecimenID,
         Latitude,
         Longitude,
         Elevation,
         class.dim2.n0,
         NMDS1.dim2.n0,
         NMDS2.dim2.n0)

colnames(CladeV_nmms)[5:7] = c('Classification','Dim1','Dim2')
# melt in prep for plotting
CladeV_nmms_melted = reshape::melt(CladeV_nmms, by='SpecimenID')

myColors = c(brewer.pal(6, 'Greys'))
# plot groups in first two dimensions
p1 = ggplot(CladeV_nmms,
             aes(x = Dim1,
                 y = Dim2,
                 fill = as.factor(Classification))) +
  geom_jitter(pch = 21,
              height = 0.05,
              width = 0.05,
              size = 3,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3',
                               'group 4',
                               'group 5',
                               'group 6')) +
  xlab('Dimension 1') +
  ylab('Dimension 2') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))
# plot groups in dimension 1 against latitude
p2 = ggplot(CladeV_nmms,
            aes(x = Dim1,
                y = Latitude)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              size = 3,
              color = 'black') +
  scale_fill_manual(values =myColors,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3',
                               'group 4',
                               'group 5',
                               'group 6')) +
  xlab('Dimension 1') +
  ylab('Latitude') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))
# plot groups in dimension1 against elevation
p3 = ggplot(CladeV_nmms,
            aes(x = Dim1,
                y = Elevation)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              size = 3,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3',
                               'group 4',
                               'group 5',
                               'group 6')) +
  xlab('Dimension 1') +
  ylab('Elevation') +
  theme_bw() +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))

p1 + p2 + p3 + plot_layout(guides = 'collect') + plot_annotation(tag_levels = 'A') & theme(plot.tag = element_text(size = 8))

```
Return to [Clade V Genomics: model-based species discovery](#model-based-species-discovery-9)
\pagebreak

```{r cladeVmolecularDelimitationsPhylogeny, echo=FALSE, include=FALSE}

myTree_midRoot = paintSubTree(myTree_midRoot, node=c(111), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(143), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(149), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(169), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(174), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(178), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(184), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(190), state='2', stem=FALSE)

```

### Fig S55: Genogroup delimitation. Cladogenesis to anagenesis model
```{r cladeVmolecularDelimitationsPhylogeny2, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, out.height='.6\\textwidth', dpi=300, fig.show='asis', fig.cap='Phylogenetic modeling for genogroup delimitation. Midpoint-rooted phylogenetic tree showing genogroups in red. Values correspond to nodes at the transition point between cladogenesis (between species) to anagenesis (within species). Values closer to 0 indicate that the node was identified as a transition to anagenesis summarized over $500$ delimitations.'}
# at this point, the 'myTree_midRoot' is now of class 'simmap', so the plotting is a bit different
# because we want our trees without tip labels, i've set the fsize to be very small (essentially label-less)
# after plotting the tree with the 'painted' clades, add the custom node labels
colors = setNames(c('black','red'),c('1','2'))
plotSimmap(myTree_midRoot, setEnv = TRUE, fsize=0.001, lwd=1, colors=colors)
# coalescent nodes
nodelabels(text='0.266', node=111, frame = 'none', adj=c(1.5,-0.50), cex=0.7)
nodelabels(text='0.074', node=143, frame = 'none', adj=c(1.5,-0.50), cex=0.7)
nodelabels(text='0.011', node=149, frame = 'none', adj=c(1.5,-0.50), cex=0.7)
nodelabels(text='0.041', node=169, frame = 'none', adj=c(1.15,-0.50), cex=0.7)
nodelabels(text='0.216', node=174, frame = 'none', adj=c(1.5,-0.50), cex=0.7)
nodelabels(text='0.003', node=178, frame = 'none', adj=c(1.5,-0.50), cex=0.7)
nodelabels(text='0.005', node=184, frame = 'none', adj=c(1.15,-0.50), cex=0.7)
nodelabels(text='0.051', node=190, frame = 'none', adj=c(1.25,-0.50), cex=0.7)

# species nodes
nodelabels(text='1.0', node=140, frame = 'none', adj=c(1.5,-0.50), cex=0.7)
nodelabels(text='1.0', node=139, frame = 'none', adj=c(1,-0.50), cex=0.7)
nodelabels(text='0.971', node=141, frame = 'none', adj=c(1.5,-0.50), cex=0.7)
nodelabels(text='0.969', node=142, frame = 'none', adj=c(1,-0.50), cex=0.7)
nodelabels(text='0.638', node=148, frame = 'none', adj=c(1.5,-0.50), cex=0.7)
nodelabels(text='0.999', node=171, frame = 'none', adj=c(1.5,-0.50), cex=0.7)
nodelabels(text='0.999', node=172, frame = 'none', adj=c(1,-0.50), cex=0.7)
nodelabels(text='0.999', node=173, frame = 'none', adj=c(1.5,-0.50), cex=0.7)
```
Return to [Clade V Genomics: model-based species discovery](#model-based-species-discovery-9)
\pagebreak

### Fig S56: Genogroup delimitation: Reproductive isolation model
```{r cladeVmolecularDelimitationPopGenBpp, echo=FALSE, fig.width=7, dpi=300, fig.cap='Population genetic modeling for genogroup delimitation. Top panel: assignment of specimens to demes according to \\texttt{STRUCTURE} and posterior probability of species delimitation modeling according to \\texttt{BPP} using these demes. Bottom panel: assignment of specimens to demes according to \\texttt{MAVERICK} and posterior probability of species delimitation modeling according to \\texttt{BPP} using these demes. Specimens are sorted from north (left) to south (right) according to locality of collection.'}

mySTR = read.csv("supplementary_material_files/CladeV_resultsFiles/CladeV_smallestMiss75_STRUCTURE.csv", header=TRUE)
myRMav = read.csv("supplementary_material_files/CladeV_resultsFiles/CladeV_smallestMiss75_RMav.csv", header=TRUE)
# attach assignments
# IMPORTANT: make sure to confirm number of columns you need to attach (corresponds with number of groups to be plotted)
# number of STRUCTURE groups = 2
# number of RMaverick groups = 3
mergedGroupInfo = cbind(sampleInfo, mySTR[match(sampleInfo$MolecularSampleName, mySTR[,'MolecularSampleName']),2:3])
mergedGroupInfo = cbind(mergedGroupInfo, myRMav[match(sampleInfo$MolecularSampleName, myRMav[,'MolecularSampleName']),2:5])

# determine majority group assignment for each individual and attach these to the group information
# STRUCTURE groups are columns 10:11
# RMaverick groups are columns 12:15
mergedGroupInfo$STRassignment = as.factor(colnames(mergedGroupInfo[,10:11]) [apply(mergedGroupInfo[,10:11],1,which.max)])
mergedGroupInfo$RMavassignment = as.factor(colnames(mergedGroupInfo[,12:15]) [apply(mergedGroupInfo[,12:15],1,which.max)])

myGreys = c(brewer.pal(3,'Greys'))
myBlues = c(brewer.pal(4, 'Blues'))
STRplot = reshape::melt(mergedGroupInfo, id.vars=c("SpecimenID","Latitude"), measure.vars=c("group1","group2")) %>%
  ggplot(aes(x = reorder(as.factor(SpecimenID), -Latitude),
             y = value,
             fill = as.factor(variable))) +
  geom_bar(stat = 'identity',
           show.legend = FALSE,
           color = 'black') +
  scale_fill_manual(values = myGreys[c(1,2,3)]) +
  ylab("Probability") +
  theme_bw() +
  #theme(axis.text.x = element_text(size=8, angle=90, vjust = 0.5, hjust=1)) +
  theme(axis.text.x = element_blank(),
        axis.title.x = element_blank()) +
  theme(text = element_text(size = 8))

RMavplot = reshape::melt(mergedGroupInfo,
                         id.vars = c("SpecimenID",
                                     "Latitude"),
                         measure.vars = c("deme1",
                                          "deme2",
                                          "deme3",
                                          "deme4")) %>%
  ggplot(aes(x = reorder(as.factor(SpecimenID), -Latitude),
             y = value,
             fill = as.factor(variable))) +
  geom_bar(stat = 'identity',
           show.legend = FALSE,
           color = 'black') +
  scale_fill_manual(values = myBlues[c(1,2,3,4,5)]) +
  ylab("Probability") +
  xlab("Specimens sorted by Latitudinal position") +
  theme_bw() +
  theme(axis.text.x = element_text(size = 4,
                                   angle = 90,
                                   vjust = 0.5,
                                   hjust = 1)) +
  theme(text = element_text(size = 8))
  #theme(axis.text.x = element_blank(), axis.title.x = element_blank())

(STRplot + grid::textGrob("BPP post. probability\nfor 2 species = 1.0") + plot_layout(widths = c(1.5,0.5))) / (RMavplot + grid::textGrob("BPP post. probability\nfor 3 species = \n0.58-0.876") + plot_layout(widths = c(1.5,0.5)))
```
Return to [Clade V Genomics: model-based species discovery](#model-based-species-discovery-9)
\pagebreak

```{r cladeVfinalDelimitation, echo=FALSE, include=FALSE}
# need to use the output of the morphometrics analysis, since it only includes samples that have BOTH flowers and leaves
morphDelim = read.csv("supplementary_material_files/CladeV_resultsFiles/CladeV_morphDR.csv")

molecularDelim = read.csv("supplementary_material_files/CladeV_resultsFiles/CladeV_molecularDelimitation.csv")
# if it doesn't have a SpecimenID column, you'll need to make one
molecularDelim = molecularDelim %>%
  separate(col = SampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'LibraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep = "_",
        remove = FALSE) %>%
  unite(col='MolecularSampleName',
        c(SpecimenID,
          toss1,
          taxon,
          LibraryNumber),
        sep = "_",
        remove = FALSE)

# join datasets, coalesce duplicate columns, select needed data
typesOfSpp_temp = full_join(morphDelim, molecularDelim, by="SpecimenID") %>%
  mutate(
    Latitude = coalesce(Latitude.x, Latitude.y),
    Longitude = coalesce(Longitude.x, Longitude.y),
    Elevation = coalesce(Elevation.x, Elevation.y)
  ) %>%
  select(MolecularSampleName,
         SpecimenID,
         Elevation,
         Latitude,
         Longitude,
         BestModel_Classification, #name of the morphoDelim column
         mPTPgroups) # name of the molecularDelim column
# confirmation
#colnames(typesOfSpp_temp)
#view(typesOfSpp_temp) # confirm number of rows matches that of sampling table


# create plotting df
typesOfSpp = typesOfSpp_temp
colnames(typesOfSpp) = c("MolecularSampleName",
                         "SpecimenID",
                         "Elevation",
                         "Latitude",
                         "Longitude",
                         "PhenoGroups",
                         "GenoGroups")


# plot tree and match dataframe to tip names - ensures order of plotting is correct
myTree = read.tree("supplementary_material_files/CladeV_resultsFiles/CladeV_smallestMiss75_concat.contree")
myTree_midRoot = midpoint.root(myTree)

ggtree(myTree_midRoot)

myTipNames = get_taxa_name()
typesOfSpp = typesOfSpp[order(match(typesOfSpp$MolecularSampleName, myTipNames)),]

```

### Fig S57: Data integration
```{r cladeVfinalDelimitation2, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, fig.height=7, dpi=300, fig.cap="Integration of phenotypic and genomic data with spatial information and evolutionary history. All specimens are assigned to their corresponding best fit phenogroup (shapes) and genogroup (colors). Specimens without phenotypic or genomic data (unknown specimens) are shown as asterisks and empty shapes, accordingly. Specimens are shown as tips of the maximum likelihood tree (left) used in the CA model analysis and mapped along latitude and elevation (right). Specimens assigned to a single phenogroup and a single genogroup delineate species that we determined as 'good species'. Specimens assigned to a single phenogroup across multiple genogroups delineate species that we determined as 'phenotypic cryptic species'. Specimens assigned to a single genogroup across multiple phenogroups delineate species that we determined as 'genetic cryptic species'."}
########################
# Phylogenetic context # ----------------------------------------------------------------------------------------
########################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups

# get number of colors and shapes that you'll need for plotting
#unique(typesOfSpp$PhenoGroups)
#unique(typesOfSpp$GenoGroups)

# set values for colors and shapes
myShapes = c(21,22,23,24,25,21,22,23,8) #the 8 is for unknown
names(myShapes) = c('1','2','3','4','5','6','7','8','NA')
myCols = c(brewer.pal(10,'Paired'),"#ffffff")
names(myCols) = c('group1','group2','group3','group4','group5','group6','group7','group8','group9','group10','NA')

# # GENO groups
# clade1 = typesOfSpp %>%
#   filter(GenoGroups=='group1' | GenoGroups=='group2' | GenoGroups=='group3' | GenoGroups=='group4')
# clade2 = typesOfSpp %>%
#   filter(GenoGroups=='group5' | GenoGroups=='group6' | GenoGroups=='group7' | GenoGroups=='group8')
# clade3 = typesOfSpp %>%
#   filter(GenoGroups=='group9')
# clade4 = typesOfSpp %>%
#   filter(GenoGroups=='group10')
#  # PHENO groups
# pheno1 = typesOfSpp %>%
#   filter(PhenoGroups=='1')
# pheno2 = typesOfSpp %>%
#   filter(PhenoGroups=='2')
# pheno3 = typesOfSpp %>%
#   filter(PhenoGroups=='3')
# pheno4 = typesOfSpp %>%
#   filter(PhenoGroups=='4')
# pheno5 = typesOfSpp %>%
#   filter(PhenoGroups=='5')
# pheno6 = typesOfSpp %>%
#   filter(PhenoGroups=='6')
# pheno7 = typesOfSpp %>%
#   filter(PhenoGroups=='7')
# pheno8 = typesOfSpp %>%
#   filter(PhenoGroups=='8')




###################
# Phylogenetic context # --------------------------------------------------------------------------------
###################

f = ggtree(myTree_midRoot) %<+% typesOfSpp + xlim_tree(0.005)
#f
f2 = f + geom_tippoint(aes(x = x,
                           fill = as.factor(GenoGroups),
                           shape = as.factor(PhenoGroups)),
                       size = 2.5,
                       stroke = 0.25) +
  scale_shape_manual(values = myShapes) +
  scale_fill_manual(values = myCols) +
  geom_tiplab(aes(label = SpecimenID),
              hjust = -0.5,
              size = 1) +
  geom_point2(aes(subset=PhenoGroups=='6'),
              pch = 3,
              size = 1,
              col = 'black') +
  geom_point2(aes(subset=PhenoGroups=='7'),
              pch = 3,
              size = 1,
              col = 'black') +
  geom_point2(aes(subset=PhenoGroups=='8'),
              pch = 3,
              size = 1,
              col = 'black') +
  geom_point2(aes(subset = is.na(PhenoGroups) & isTip,
                  color=as.factor(GenoGroups)),
              pch = 8,
              size = 1,
              stroke = 0.5,
              alpha = 0.5)+
  scale_color_manual(values = myCols) +
  theme(legend.position = "none")
#f2

# subset by major clades; clade numbers come from mPTP figure above
f3 = viewClade(f2,141) #blue/green
f4 = viewClade(f2,171) #oranges
f5 = viewClade(f2,190) #light purple clade
f6 = viewClade(f2,111) #dark purple clade

###################
# Spatial context # ----------------------------------------------------------------------------------------
###################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups

##### plot individuals with GenoGroup assignment, but no PhenoGroup assignment
g = ggplot(typesOfSpp,
           aes(x = Latitude,
               y = Elevation)) +
  geom_jitter(data = typesOfSpp %>%
                filter(is.na(PhenoGroups)),
              aes(x = Latitude,
                  y = Elevation,
                  color = as.factor(GenoGroups)),
              size = 2,
              alpha = 0.5,
              width = 0.75,
              height = 1.5,
              pch = 8,
              stroke = 1) +
  scale_color_manual(values = myCols) +
  scale_x_reverse(breaks=c(-25,-27,-29,-31,-33,-35,-37,-39,-41,-43,-45,-47,-49,-51,-53,-55)) +
  theme_bw() +
  theme(legend.position = 'none') +
  theme(panel.grid.major = element_line(size = 0.55,
                                        color = '#f0f0f0'))
#g

##### plot individuals with PhenoGroup assignment, but no GenoGroup assignment
g2 = g + geom_point(data = typesOfSpp %>%
                      filter(is.na(GenoGroups)),
                    aes(x = Latitude,
                        y = Elevation,
                        shape = as.factor(PhenoGroups)),
                    fill = 'white',
                    size = 2) +
  scale_shape_manual(values = myShapes) +
  geom_point(data = typesOfSpp %>%
               filter(is.na(GenoGroups) & PhenoGroups=='6'),
             aes(x = Latitude,
                 y = Elevation),
             shape = 3,
             size = 1) +
  geom_point(data = typesOfSpp %>%
               filter(is.na(GenoGroups) & PhenoGroups=='7'),
             aes(x = Latitude,
                 y = Elevation),
             shape = 3,
             size = 1) +
  geom_point(data = typesOfSpp %>%
               filter(is.na(GenoGroups) & PhenoGroups=='8'),
             aes(x = Latitude,
                 y = Elevation),
             shape = 3,
             size = 1)
#g2

###### plots fully overlapping data (has both Geno- and Pheno- group assignments)
g3 = g2 + geom_point(data=typesOfSpp %>%
                       filter(GenoGroups != 'NA' & PhenoGroups != 'NA'),
                       aes(x = Latitude,
                           y = Elevation,
                           fill = as.factor(GenoGroups),
                           shape = as.factor(PhenoGroups)),
                     size = 3.5,
                     width = 0.75,
                     height = 1.5) +
  scale_fill_manual(values = myCols) +
  geom_point(data = typesOfSpp %>%
               filter(GenoGroups != 'NA' & PhenoGroups == '6'),
             aes(x = Latitude,
                 y = Elevation),
             shape = 3,
             size = 1.5) +
  geom_point(data = typesOfSpp %>%
               filter(GenoGroups != 'NA' & PhenoGroups == '7'),
             aes(x = Latitude,
                 y = Elevation),
             shape = 3,
             size = 1.5) +
  geom_point(data = typesOfSpp %>%
               filter(GenoGroups != 'NA' & PhenoGroups == '8'),
             aes(x = Latitude,
                 y = Elevation),
             shape = 4,
             size = 1.5)
#g3

###### create the plot and save to an object
plotForLegend = ggplot(typesOfSpp,
                       aes(x = Latitude,
                           y = Elevation,
                           fill = as.factor(GenoGroups),
                           shape = as.factor(PhenoGroups))) +
  geom_point() +
  scale_fill_manual(values = myCols,
                    name = 'Genogroups',
                    labels = c('1','2','3','4','5','6','7','8','9','10','unknown')) +
  scale_shape_manual(values = myShapes,
                     name = 'Phenogroups',
                     labels = c('1','2','3','4','5','6','7','8','unknown')) +
  guides(fill = guide_legend(override.aes = list(fill = c(myCols[1:10],
                                                          'white'),
                                                 size = 3,
                                                 shape = 22,
                                                 color = 'black'))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21,22,23,24,25,10,12,9,8),
                                                  size = 3))) +
  theme(legend.key = element_rect(fill = NA),
        text = element_text(size = 8))

legend = cowplot::get_legend(plotForLegend)
#g3 + legend


##########
# GK TAU # ----------------------------------------------------------------------------------------
##########
fullOverlap = typesOfSpp %>%
  filter(PhenoGroups != 'NA' & GenoGroups != 'NA')

gktau_morph2molecules = GKtau(fullOverlap$PhenoGroups,
                              fullOverlap$GenoGroups)
gktau_molecules2morph = GKtau(fullOverlap$GenoGroups,
                              fullOverlap$PhenoGroups)


####################################
# PLOT both together # --------------------------------------------
####################################
# one row
f2 + g3 + legend + plot_layout(widths=c(4,4,1))


```
Return to [Clade V Data integration](#data-integration-5)
\pagebreak

## Clade VI
### Fig S58: Taxon sampling
```{r CladeVIVennDiagram, echo=FALSE, dpi=300, fig.width=4, fig.align='center', fig.cap="Specimens sampled according to three types of data. Specimens outside the Flowers category represent sterile specimens."}

CladeVI_Venn = escalloniaSampling %>%
  filter(CladeID == "VI") %>%
  mutate(Flowers = if_else(is.na(`Floral and Leaf`), 0, 1))

CladeVI_Venn = CladeVI_Venn %>%
  mutate(Leaves = if_else(is.na(`Leaf only`) & is.na(`Floral and Leaf`), 0,1))

CladeVI_Venn = CladeVI_Venn %>%
  mutate(Genes = if_else(is.na(libraryNumber), 0, 1))

CladeVI_Venn = CladeVI_Venn %>%
  select(Flowers,
         Leaves,
         Genes)

VennForCladeVI = venn(CladeVI_Venn)

plot(VennForCladeVI,
     fills = list(fill=c('#c2a5cf',
                         '#d9f0d3',
                         '#f7f7f7')),
     quantities = list(TRUE,
                       cex = 1.5),
     legend = list(labels,
                   cex = 1))

```
Return to [Clade VI Sampling](#sampling-6)
\pagebreak

### Fig S59: Geographic distribution
```{r CladeVIsamplingMaps, echo=FALSE, dpi = 300, fig.width=7, fig.align='center', fig.cap="Geographic distribution of specimens sampled for this clade according to taxonomic species. Filled symbols indicate specimens used in phenotypic analyses and empty symbols specimens used in genomic analyses."}
myColors = "#cccccc"
myShapes = c(21,22,23,24,25,10)
angustifoliaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'angustifolia') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'angustifolia') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name ='Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape=c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -35),
              ylim = c(-40, 0)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia angustifolia')


bifidaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'bifida') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'bifida') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -35),  
              ylim = c(-40, 0)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia bifida')

farinaceaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'farinacea') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'farinacea') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -35),
              ylim = c(-40, 0)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia farinacea')

hypoglaucaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'hypoglauca') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'hypoglauca') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -35),
              ylim = c(-40, 0)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia hypoglauca')

illinitaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'illinita') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'illinita') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -35),  
              ylim = c(-40, 0)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia illinita')

laevisMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'laevis') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'laevis') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -35),
              ylim = c(-40, 0)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia laevis')

ledifoliaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'ledifolia') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'ledifolia') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -35),
              ylim = c(-40, 0)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia ledifolia')


megapotamicaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'megapotamica') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'megapotamica') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -35),  
              ylim = c(-40, 0)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia megapotamica')


petrophilaaMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'petrophila') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'petrophila') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -35),
              ylim = c(-40, 0)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia petrophila')


tucumanensisMap = escalloniaSampling %>%
  dplyr::filter(Taxon == 'tucumanensis') %>%
  ggplot() +
  mapa +
  geom_jitter(aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon),
              bg = myColors,
              color = 'black',
              cex = 2,
              alpha = 0.8,
              width = 0.5,
              height = 0.5) +
  geom_jitter(data = escalloniaSampling %>%
                dplyr::filter(Taxon == 'tucumanensis') %>%
                dplyr::filter(libraryNumber != 'NA'),
              aes(x = Longitude,
                  y = Latitude,
                  shape = Taxon,
                  fill = Taxon),
              cex = 1.5,
              width = 0.5,
              height = 0.5) +
  scale_shape_manual(values = myShapes[1],
                     name = 'Phenotype',
                     labels = NULL) +
  scale_fill_manual(values = 'white',
                    name = 'Genomics',
                    labels = NULL) +
  guides(fill = guide_legend(override.aes = list(shape = c(21),
                                                 cex = 4))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21),
                                                  cex = 4))) +
  coord_fixed(xlim = c(-80, -35),  
              ylim = c(-40, 0)) +
  theme_bw() +  
  theme(panel.border = element_blank(),
        panel.grid.major = element_line(colour = "#f0f0f0"),
        panel.grid.minor = element_blank(),
        axis.title = element_blank(),
        title = element_text(face = 'italic'),
        text = element_text(size = 6)) +
  ggtitle(label = 'Escallonia tucumanensis')

angustifoliaMap + bifidaMap + farinaceaMap + hypoglaucaMap +
  illinitaMap + laevisMap + ledifoliaMap + megapotamicaMap +
  petrophilaaMap + tucumanensisMap + plot_layout(ncol = 4, guides = 'collect')

```
Return to [Clade VI Sampling](#sampling-6)
\pagebreak

```{r currentStateOfTaxonomyCladeVIMakePlots, echo=FALSE}

pal = c('#636363')
names(pal) = c('assignment')

# plot
CladeVI_propOverlap_All = ggplot(melted_CladeVI_overlapProp_All,
                                aes(x = X1,
                                    y = X2)) +
  geom_tile(aes(fill = value)) +
  geom_text(aes(label = round(value, digits = 3)),
            color = 'darkgrey',
            size = 0.75) +
  scale_fill_gradient(low = 'white',
                      high = "black") +
  theme(axis.title = element_blank(),
        axis.text = element_text(size = 6),
        axis.text.x = element_text(angle = -45,
                                   hjust = 0.1)) +
  theme(legend.text = element_text(size = 4),
        legend.margin = margin(2,2,6,2),
        legend.title = element_text(size = 6),
        legend.background = element_rect(color="#cccccc")) +
  guides(fill = guide_colourbar(barwidth = 0.5,
                                barheight = 2))


CladeVI_inHullAssignments_All = ggplot(melted_CladeVI_hullResults_All,
                                      aes(x = uniqueID,
                                          y = variable,
                                          fill = col)) +
  geom_tile(aes(width = 0.99,
                height = 0.99),
            color = 'black') +
  scale_fill_manual(values = c("assignment" = '#636363',
                               "no assignment" = "white"),
                    name = NULL,
                    labels = c('yes','no'),
                    drop = FALSE) +
  scale_y_discrete(labels = c("In E. angustifolia",
                            "In E. bifida",
                            "In E. farinacea",
                            "In E. hypoglauca",
                            "In E. illinita",
                            "In E. laevis",
                            "In E. ledifolia",
                            "In E. megapotamica",
                            "In E. petrophila",
                            "In E. tucumanensis")) +
  scale_x_discrete(limits=factor(1:195)) + # limits equal to number of samples
  theme_bw() +
  theme(panel.ontop = TRUE,
        panel.background = element_rect(fill='transparent'),
        panel.border = element_blank(),
        panel.grid.major.y = element_blank(),
        panel.grid.major.x = element_blank(),
        axis.title.y = element_blank(),
        axis.title.x = element_blank(),
        axis.ticks.x = element_blank(),
        axis.text.x = element_blank(),
        legend.position = 'top',
        text = element_text(size = 6),
        plot.margin = margin(1,20,35,10),
        legend.key.height = unit(0.25,'cm'),
        legend.key.width = unit(0.25,'cm')) +
  coord_cartesian(ylim = c(1.1,10),
                  clip = 'off') +
  annotate(geom = 'text',
           x = (14/2),
           y = 0.1,
           label = 'As E. angustifolia',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (15 + (58-15)/2),
           y = 0.1,
           label = 'As E. bifida',
           size = 2,
           angle = 320,
           hjust = 0) +  
  annotate(geom = 'text',
           x = (59 + (75-59)/2),
           y = 0.1,
           label ='As E. farinacea',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (76 + (97-76)/2),
           y = 0.1,
           label = 'As E. hypoglauca',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (98 + (118-98)/2),
           y = 0.1,
           label = 'As E. illinita',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (119 + (145-119)/2),
           y = 0.1,
           label = 'As E. laevis',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (146 + (147-146)/2),
           y = 0.1,
           label = 'As E. ledifolia',
           size = 2,
           angle = 320,
           hjust=0) +
  annotate(geom = 'text',
           x = (148 + (174-148)/2),
           y = 0.1,
           label = 'As E. megapotamica',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (175 + (176-175)/2),
           y = 0.1,
           label = 'As E. petrophila',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'text',
           x = (177 + (195-177)/2),
           y = 0.1,
           label = 'As E. tucumanensis',
           size = 2,
           angle = 320,
           hjust = 0) +
  annotate(geom = 'segment',
           x = 1,
           xend = 14,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 15,
           xend = 58,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 59,
           xend = 75,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 76,
           xend = 97,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 98,
           xend = 118,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 119,
           xend = 145,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 146,
           xend = 147,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 148,
           xend = 174,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 175,
           xend = 176,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75) +
  annotate(geom = 'segment',
           x = 177,
           xend = 195,
           y = 0.3,
           yend = 0.3,
           color = 'black',
           size = 0.75)

```

### Fig S60: Current state of taxonomic species with phenotypic data
```{r currentStateOfTaxonomyCladeVIPlots, fig.width=7, fig.height = 5, dpi=200, echo=FALSE, fig.cap="Assessment of current state of taxonomic species with phenotypic data. Top panel: Pairwise overlap among 10-cubes describing geometrically each taxonomic species. Bottom panel: Matching-prediction analysis with each cell along the x-axis representing specimens sorted according to taxonomic species and the 10-cubes corresponding to each taxonomic species along the y-axis. If a specimen matches the prediction of the monograph (i.e., it is inside a 10-cube), the corresponding cell is shaded. If the specimen does not match the prediction, the cell is empty."}

(((CladeVI_propOverlap_All + plot_spacer())/ CladeVI_inHullAssignments_All) + plot_layout(tag_level='new'))

```
Return to [Clade VI Current state of taxonomic species](#current-state-of-taxonomic-species-5)
\pagebreak

### Fig S61: Phenogroup delimitation: Gaussian finite mixture modeling
```{r cladeVImorphologicalDelimitationPlots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=6.5, dpi=300, fig.cap='Gaussian finite mixture modeling (GFMM) for phenogroup delimitation and model selection using the Bayesian information criterion (BIC). Top panel: empirical support (ordinate) for Gaussian mixture models (GMM) assuming distinct number of phenogroups (abscissa). Each GMM specifies different number of phenogroups (shapes). Empirical support was measured as difference in BIC relative to the best model ($\\Delta$BIC = $0$). Bottom panel: Visualization of the phenogroups (shapes) identified by the best fit GMM; left panel shows phenogroups in the space defined by two axes obtained by linear discriminat analysis (to maximize separation and visualization), middle panel shows phenogroups in the space defined by discriminant axis 1 and latitude, and right panel shows phenogroups in the space defined by discriminant axis 1 and elevation.'}


CladeVI_modelsRanked = read.csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_modelsRanked.csv")
CladeVI_morphDR = read.csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_morphDR.csv")
max.BIC = 231.2478

#plot
modelsRanked = ggplot(CladeVI_modelsRanked,
                      aes(x = numMorphGroups,
                          y = deltaBIC_naive)) +  
  geom_point(aes(x = numMorphGroups,
                 y = deltaBIC_naive,
                 shape = "A_naive_models"),
             cex = 2,
             col = "black",
             bg = 'white') +
  geom_point(aes(x = numMorphGroups[order(deltaBIC_naive)[1]],
                 y = deltaBIC_naive[order(deltaBIC_naive)[1]],
                 shape = "B_best_naive_model"),
             cex = 2,
             col = 'black',
             bg = 'black') +
  # geom_point(aes(x=numMorphGroups[order(deltaBIC_naive)[2]],
  #                y=deltaBIC_naive[order(deltaBIC_naive)[2]],
  #                shape ="C_second_naive_model"), cex=2, col='black',bg='grey') +
  geom_point(aes(x = numMorphGroups[taxonBIC != 'NA'],
                 y = max.BIC-taxonBIC[taxonBIC != 'NA'],
                 shape = "D_currentTax"),
             cex = 2,
             col = 'black',
             bg = 'red') +
  geom_point(aes(x = numMorphGroups[morphoGroupsBIC != 'NA'],
                 y = max.BIC-morphoGroupsBIC[morphoGroupsBIC != 'NA'],
                 shape = "E_morphoGroups"),
             cex = 2,
             col = 'black',
             bg = 'red') +
  scale_shape_manual(values = c(21,21,25,24),
                     labels = c("Best fit model",
                              "Naive models",
                              "Taxonomy model",
                              "Taxonomy unaware model"),
                     name = NULL) +
  guides(shape = guide_legend(override.aes = list(shape = c(21,21,25,24),
                                                  bg = c('black','white','red','red'),
                                                  color = c('black','black','black','black'),
                                                  size = 3))) +
  scale_x_discrete(limits=c(1,2,3,4,5,6,7,8,9,10,11,12,13,14,15)) +
  theme_bw() +
  theme(panel.grid = element_blank()) +
  xlab('Number of phenogroups') +
  ylab(expression(paste("Empirical support (", Delta, "BIC)", sep="")))



# plot ordination of best naive model
myShapes = c(21,22,23,24,25,10,12,9)
Q2 = ggplot(CladeVI_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = MclustDR_Dir2,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 1,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group1',
                                'group2',
                                'group3',
                                'group4',
                                'group5',
                                'group6',
                                'group7',
                                'group8')) +
  xlab("Dimension 1") +
  ylab("Dimension 2") +
  theme(legend.key = element_rect(fill=NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size=10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

Q3 = ggplot(CladeVI_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = Latitude,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 1,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group1',
                                'group2',
                                'group3',
                                'group4',
                                'group5',
                                'group6',
                                'group7',
                                'group8')) +
  xlab("Dimension 1") +
  ylab("Latitude") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

Q4 = ggplot(CladeVI_morphDR) +
  geom_point(aes(x = MclustDR_Dir1,
                 y = Elevation,
                 shape = as.factor(BestModel_Classification)),
             color = 'black',
             bg = '#cccccc',
             size = 1,
             stroke = 0.25) +
  scale_shape_manual(values = myShapes,
                     name = 'Classification',
                     labels = c('group1',
                                'group2',
                                'group3',
                                'group4',
                                'group5',
                                'group6',
                                'group7',
                                'group8')) +
  xlab("Dimension 1") +
  ylab("Elevation") +
  theme(legend.key = element_rect(fill = NA),
        panel.grid.major = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(fill = NA,
                                    color = 'black'),
        text = element_text(size = 10)) +
  guides(shape = guide_legend(override.aes = list(size = 2)))

(modelsRanked + plot_spacer() + plot_layout(widths = c(3,1))) /
  (Q2 + Q3 + Q4 + plot_layout(guides = 'collect'))

```
Return to [Clade VI Phenomics: model-based species discovery](#model-based-species-discovery-10)
\pagebreak

```{r, CladeVIgenomicSensitivityAnalyses_largest, echo=FALSE, warning = FALSE, message=FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeVI_resultsFiles/CladeVI_largestMiss25_concat.contree")
load("supplementary_material_files/CladeVI_resultsFiles/CladeVI_largestMiss25_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeVI_resultsFiles/CladeVI_largestMiss25_ADMIX.15.Q")
testData = read_delim("supplementary_material_files/CladeVI_resultsFiles/CladeVI_largestMiss25_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_popFile_withLocality_ADMIXordered.csv", header=TRUE)
sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
   unite(col = 'SampleName',
         c(SpecimenID,
           toss1,
           taxon),
         sep="_",
         remove = FALSE) %>%
   select(MolecularSampleName,
          SampleName,
          SpecimenID,
          libraryNumber,
          taxon,
          country,
          Elevation,
          Latitude,
          Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3","ADMIXgroup4","ADMIXgroup5","ADMIXgroup6","ADMIXgroup7","ADMIXgroup8","ADMIXgroup9","ADMIXgroup10","ADMIXgroup11","ADMIXgroup12","ADMIXgroup13","ADMIXgroup14","ADMIXgroup15")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:24)], id =c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeVIgenomicSensitivityAnalyses_largest2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}
cols = c(c(brewer.pal(5, 'Greys')),c(brewer.pal(5, 'Greens')),c(brewer.pal(5, 'Purples')))

#########################

##### PHYLO + ADMIX BARS #####

###########################

plot1 = ggtree(myTree_midRoot)

plot2 = plot1 +
  geom_tiplab(geom = 'text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)
#plot2

plot3 = facet_plot(plot2,
                   data = admixMelt,
                   panel = 'Admixture proportions',
                   geom = geom_barh,
                   mapping = aes(x = as.numeric(value),
                                 y = as.character(label),
                                 fill = variable),
                   color = 'grey90',
                   stat = 'identity') +
  xlim_tree(0.01) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values=cols) +
  scale_y_discrete(limits=rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")
#plot3

plot4 = facet_widths(plot3, c(6,3))
#plot4

#########################
#####      PCA      #####
#########################

pch_v = c(21,22,23,24,25,10,12,8,9,11)
fastPCA_largest = ggplot(tab,
                         aes(x = EV1,
                             y = EV2,
                             shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.angustifolia",
                                "E.bifida",
                                "E.farinacea",
                                "E.hypoglauca",
                                "E.illinita",
                                "E.laevis",
                                "E.ledifolia",
                                "E.megapotamica",
                                "E.petrophila",
                                "E.tucumanensis"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size = unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################

step1_largest = as_tibble(testData %>%
                            dplyr::group_by(clusterNum) %>%
                            summarise_at("error", list(mean, sd)))

colnames(step1_largest) = c("clusterNum", "mean", "stdev")

step1_largest$me = sapply(step1_largest$stdev,
                          function(x) qt(.95,9)*x/sqrt(10))

testCI_largest = mutate(as_tibble(step1_largest),
                        lower = mean-me,
                        upper = mean+me)

ADMIX_CI_largest = ggplot(testCI_largest, aes(x=clusterNum, y=mean)) + geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:15)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

```

```{r, CladeVIgenomicSensitivityAnalyses_middle, echo=FALSE, warning = FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeVI_resultsFiles/CladeVI_middleMiss50_concat.contree")
load("supplementary_material_files/CladeVI_resultsFiles/CladeVI_middleMiss50_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeVI_resultsFiles/CladeVI_middleMiss50_ADMIX.15.Q")
testData = read_delim("supplementary_material_files/CladeVI_resultsFiles/CladeVI_middleMiss50_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_popFile_withLocality_ADMIXordered.csv", header=TRUE)
sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
   unite(col = 'SampleName',
         c(SpecimenID,
           toss1,
           taxon),
         sep = "_",
         remove = FALSE) %>%
   select(MolecularSampleName,
          SampleName,
          SpecimenID,
          libraryNumber,
          taxon,
          country,
          Elevation,
          Latitude,
          Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3","ADMIXgroup4","ADMIXgroup5","ADMIXgroup6","ADMIXgroup7","ADMIXgroup8","ADMIXgroup9","ADMIXgroup10","ADMIXgroup11","ADMIXgroup12","ADMIXgroup13","ADMIXgroup14","ADMIXgroup15")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:24)],
                            id = c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeVIgenomicSensitivityAnalyses_middle2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}

cols = c(c(brewer.pal(5, 'Greys')), c(brewer.pal(5, 'Greens')), c(brewer.pal(5, 'Purples')))

#########################

##### PHYLO + ADMIX BARS #####

###########################

plot5 = ggtree(myTree_midRoot)

plot6 = plot5 +
  geom_tiplab(geom = 'text',
              size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot7 = facet_plot(plot6,
                   data = admixMelt,
                   panel = 'Admixture proportions',
                   geom = geom_barh,
                   mapping = aes(x = as.numeric(value),
                                 y = as.character(label),
                                 fill = variable),
                   color = 'grey90',
                   stat = 'identity') +
  xlim_tree(0.01) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")

plot8 = facet_widths(plot7, c(6,3))

#########################
#####      PCA      #####
#########################

pch_v = c(21,22,23,24,25,10,12,8,9,11)
fastPCA_middle = ggplot(tab,
                        aes(x = EV1,
                            y = EV2,
                            shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.angustifolia",
                                "E.bifida",
                                "E.farinacea",
                                "E.hypoglauca",
                                "E.illinita",
                                "E.laevis",
                                "E.ledifolia",
                                "E.megapotamica",
                                "E.petrophila",
                                "E.tucumanensis"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size = unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################

step1_middle = as_tibble(testData %>%
                           dplyr::group_by(clusterNum) %>%
                           summarise_at("error", list(mean, sd)))

colnames(step1_middle) = c("clusterNum", "mean", "stdev")

step1_middle$me = sapply(step1_middle$stdev,
                         function(x) qt(.95,9)*x/sqrt(10))

testCI_middle = mutate(as_tibble(step1_middle),
                       lower = mean-me,
                       upper = mean+me)

ADMIX_CI_middle = ggplot(testCI_middle,
                         aes(x = clusterNum,
                             y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:15)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

```

```{r, CladeVIgenomicSensitivityAnalyses_smallest, echo=FALSE, warning = FALSE, include=FALSE}
# read phylogeny, fast PCA, and results from ADMIX (both assignments and results from replicate runs)
myTree = read.tree("supplementary_material_files/CladeVI_resultsFiles/CladeVI_smallestMiss75_concat.contree")
load("supplementary_material_files/CladeVI_resultsFiles/CladeVI_smallestMiss75_PCA.Rdata")
tbl = read.table("supplementary_material_files/CladeVI_resultsFiles/CladeVI_smallestMiss75_ADMIX.15.Q")
testData = read_delim("supplementary_material_files/CladeVI_resultsFiles/CladeVI_smallestMiss75_ADMIX_results.csv", delim=",")

# read in sample information and modify for plotting
sampleInfo = read.csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_popFile_withLocality_ADMIXordered.csv", header=TRUE)
sampleInfo = sampleInfo %>%
  separate(col = MolecularSampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'libraryNumber'),
           remove = FALSE) %>%
   unite(col = 'SampleName',
         c(SpecimenID,
           toss1,
           taxon),
         sep="_",
         remove = FALSE) %>%
   select(MolecularSampleName,
          SampleName,
          SpecimenID,
          libraryNumber,
          taxon,
          country,
          Elevation,
          Latitude,
          Longitude)

# assign column names to ADMIX assignments, and bind to sample information
colnames(tbl) = c("ADMIXgroup1","ADMIXgroup2","ADMIXgroup3","ADMIXgroup4","ADMIXgroup5","ADMIXgroup6","ADMIXgroup7","ADMIXgroup8","ADMIXgroup9","ADMIXgroup10","ADMIXgroup11","ADMIXgroup12","ADMIXgroup13","ADMIXgroup14","ADMIXgroup15")
mergedAdmixtureTable = cbind(sampleInfo, tbl)

# midpoint root the tree and replace extended taxon names with SpecimenID
myTree_midRoot = midpoint.root(myTree)

plot.phylo(myTree_midRoot,
           show.tip.label = FALSE,
           edge.width = 4)

myTree_midRoot$tip.label = as.character(sampleInfo[,'SpecimenID'][match(myTree_midRoot$tip.label, sampleInfo[,'MolecularSampleName'])])

plot.phylo(myTree_midRoot,
           edge.width = 3,
           cex = 1,
           align.tip.label = TRUE)

# plot tree to get tip names
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()

# order merged admixture table according to the tip names
mergedAdmixtureTable = mergedAdmixtureTable[order(match(mergedAdmixtureTable$SpecimenID, myTipNames)),]
# melt the table to plot alongside the phylogeny
admixMelt = reshape2:::melt(mergedAdmixtureTable[,c(3,5,10:24)], id =c("SpecimenID","taxon")) # SpecimenID, taxon, ADMIX groups

```

```{r, CladeVIgenomicSensitivityAnalyses_smallest2, echo=FALSE, warning=FALSE, message=FALSE, include=FALSE}

cols = c(c(brewer.pal(5, 'Greys')),c(brewer.pal(5, 'Greens')),c(brewer.pal(5, 'Purples')))

#########################

##### PHYLO + ADMIX BARS #####

###########################

plot9 = ggtree(myTree_midRoot)

plot10 = plot9 + geom_tiplab(geom = 'text',
                             size = 1.0) +
  geom_treescale(fontsize = 1.0)

plot11 = facet_plot(plot10,
                    data = admixMelt,
                    panel = 'Admixture proportions',
                    geom = geom_barh,
                    mapping = aes(x = as.numeric(value),
                                  y = as.character(label),
                                  fill = variable),
                    color = 'grey90',
                    stat = 'identity') +
  xlim_tree(0.01) +
  #xlim_expand(c(0,5), 'Tree') +
  scale_fill_manual(values = cols) +
  scale_y_discrete(limits = rev(myTipNames)) +
  theme(strip.text.x = element_text(size = 4),
        legend.position = "none")

plot12 = facet_widths(plot11, c(6,3))

#########################
#####      PCA      #####
#########################

pch_v = c(21,22,23,24,25,10,12,8,9,11)
fastPCA_smallest = ggplot(tab,
                          aes(x = EV1,
                              y = EV2,
                              shape = pop)) +
  geom_point(cex = 1,
             col = 'black',
             bg = "grey90") +
  scale_shape_manual(values = pch_v,
                     labels = c("E.angustifolia",
                                "E.bifida",
                                "E.farinacea",
                                "E.hypoglauca",
                                "E.illinita",
                                "E.laevis",
                                "E.ledifolia",
                                "E.megapotamica",
                                "E.petrophila",
                                "E.tucumanensis"),
                     name = 'Taxon') +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        text = element_text(size = 6),
        legend.key.size = unit(0.25,'cm')) +
  xlab('Eigenvector 1') +
  ylab('Eigenvector 2')

#########################
####     ADMIX CI   #####
#########################

step1_smallest = as_tibble(testData %>%
                             dplyr::group_by(clusterNum) %>%
                             summarise_at("error", list(mean, sd)))

colnames(step1_smallest) = c("clusterNum", "mean", "stdev")

step1_smallest$me = sapply(step1_smallest$stdev,
                           function(x) qt(.95,9)*x/sqrt(10))

testCI_smallest = mutate(as_tibble(step1_smallest),
                         lower = mean-me,
                         upper = mean+me)

ADMIX_CI_smallest = ggplot(testCI_smallest,
                           aes(x = clusterNum,
                               y = mean)) +
  geom_point(cex = 0.25) +
  geom_errorbar(aes(ymin = lower,
                    ymax = upper),
                width = 0.5,
                size = 0.25) +
  scale_x_discrete(name = "Number of Groups (K)",
                   limits = c(1:15)) +
  scale_y_reverse() +
  theme(text = element_text(size = 6))

```

### Fig S62: Sensitivity tests with $75\%$ missing data
```{r, cladeVIgenomicSensitivityAnalysesLargestplots, echo=FALSE, warning=FALSE, message=FALSE, results='asis', fig.width=7, dpi=300, fig.cap='Impact of missing data ($75\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes.'}

plot4 + (ADMIX_CI_largest/fastPCA_largest)+plot_layout(widths=c(4,2),heights=c(50,0.5))+plot_annotation(tag_levels = 'A') & theme(text=element_text(size = 8))

```
Return to [Clade VI Genomics: sensitivity tests](#sensitivity-tests-5)
\pagebreak

### Fig S63: Sensitivity tests with $50\%$ missing data
```{r, cladeVIgenomicSensitivityAnalysesMiddleplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($50\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot8 + (ADMIX_CI_middle / fastPCA_middle) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))
```
Return to [Clade VI Genomics: sensitivity tests](#sensitivity-tests-5)
\pagebreak

### Fig S64: Sensitivity tests with $25\%$ missing data
```{r, cladeVIgenomicSensitivityAnalysesSmallestplots, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, dpi=300, fig.cap="Impact of missing data ($25\\%$) on data analysis. A) maximum likelihood phylogenetic tree of specimens assigned to demes according to best \\texttt{ADMIXTURE} run. B) Mean cross validation error with $95\\%$ confidence interval (ordinate) for ten replicate runs of \\texttt{ADMIXTURE} assuming differnt number of demes (K) (abscissa); C) Scatterplot of Principal Component Analysis (PCA) projected along the first two axes."}

plot12 + (ADMIX_CI_smallest / fastPCA_smallest) + plot_layout(widths = c(4,2), heights = c(50,0.5)) + plot_annotation(tag_levels = 'A') & theme(text = element_text(size = 8))
```
Return to [Clade VI Genomics: sensitivity tests](#sensitivity-tests-5)
\pagebreak

### Fig S65: Genogroup delimitation: Genotypic cluster model
```{r cladeVImolecularDelimitationsNmms, echo=FALSE, message=FALSE, fig.width=7, fig.height=2.5, dpi=300, fig.cap='Gaussian finite mixture modeling (GFMM) for genogroup delimitation. Visualization of the genogroups (shades) identified by the best fit Gaussian mixture model (GMM). A) genogroups in the space defined by two axes obtained by non-metric multidimensional scaling (NMDS); B) genogroups in the space defined by NMDS axis 1 and latitude; C) genogroups in the space defined by NMDS axis 1 and elevation.'}
# read in results of NMM delimitation
CladeVI_nmms_temp = read.csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_smallestMiss75_NMMs.csv")
# select classification and dimensions 1 and 2 from best model
CladeVI_nmms = CladeVI_nmms_temp %>% select(SpecimenID, Latitude, Longitude, Elevation, class.dim2.n0, NMDS1.dim2.n0, NMDS2.dim2.n0)
colnames(CladeVI_nmms)[5:7] = c('Classification','Dim1','Dim2')
# melt in prep for plotting
CladeVI_nmms_melted = reshape::melt(CladeVI_nmms, by='SpecimenID')

myColors = c(c(brewer.pal(4, 'Greys')), c(brewer.pal(3, 'Blues')))
# plot groups in first two dimensions
p1 = ggplot(CladeVI_nmms,
             aes(x = Dim1,
                 y = Dim2,
                 fill = as.factor(Classification))) +
  geom_jitter(pch = 21,
              height = 0.05,
              width = 0.05,
              size = 2,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3',
                               'group 4',
                               'group 5',
                               'group 6',
                               'group 7')) +
  xlab('Dimension 1') +
  ylab('Dimension 2') +
  theme_bw() +
  theme(axis.text = element_text(size=6)) +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))
# plot groups in dimension 1 against latitude
p2 = ggplot(CladeVI_nmms,
             aes(x = Dim1,
                 y = Latitude)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              size = 2,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3',
                               'group 4',
                               'group 5',
                               'group 6',
                               'group 7')) +
  xlab('Dimension 1') +
  ylab('Latitude') +
  theme_bw() +
  theme(axis.text = element_text(size=6)) +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))
# plot groups in dimension1 against elevation
p3 = ggplot(CladeVI_nmms,
             aes(x = Dim1,
                 y = Elevation)) +
  geom_jitter(aes(fill = as.factor(Classification)),
              pch = 21,
              height = 0.05,
              width = 0.05,
              size = 2,
              color = 'black') +
  scale_fill_manual(values = myColors,
                    name = 'Classification',
                    labels = c('group 1',
                               'group 2',
                               'group 3',
                               'group 4',
                               'group 5',
                               'group 6',
                               'group 7')) +
  xlab('Dimension 1') +
  ylab('Elevation') +
  theme_bw() +
  theme(axis.text = element_text(size=6)) +
  theme(legend.position = 'right',
        legend.direction = 'vertical',
        text = element_text(size = 10))

p1 + p2 + p3 + plot_layout(guides = 'collect') + plot_annotation(tag_levels = 'A') & theme(plot.tag = element_text(size = 8))

```
Return to [Clade VI Genomics: model-based species discovery](#model-based-species-discovery-11)
\pagebreak

```{r cladeVImolecularDelimitationsPhylogeny, echo=FALSE, include=FALSE}

myTree_midRoot = paintSubTree(myTree_midRoot, node=c(161), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(156), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(154), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(144), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(135), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(129), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(120), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(101), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(93), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(91), state='2', stem=FALSE)
myTree_midRoot = paintSubTree(myTree_midRoot, node=c(86), state='2', stem=FALSE)

```

### Fig S66: Genogroup delimitation. Cladogenesis to anagenesis model
```{r cladeVImolecularDelimitationsPhylogeny2, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, out.height='.6\\textwidth', dpi=300, fig.show='asis', fig.cap='Phylogenetic modeling for genogroup delimitation. Midpoint-rooted phylogenetic tree showing genogroups in red. Values correspond to nodes at the transition point between cladogenesis (between species) to anagenesis (within species). Values closer to 0 indicate that the node was identified as a transition to anagenesis summarized over $500$ delimitations.'}
# at this point, the 'myTree_midRoot' is now of class 'simmap', so the plotting is a bit different
# because we want our trees without tip labels, i've set the fsize to be very small (essentially label-less)
# after plotting the tree with the 'painted' clades, add the custom node labels
colors = setNames(c('black','red'),c('1','2'))
plotSimmap(myTree_midRoot, setEnv = TRUE, fsize=0.001, lwd=1, colors=colors)
# coalescent nodes
nodelabels(text='0.001', node=161, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='0.001', node=156, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='0.001', node=154, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='0.012', node=144, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='0.295', node=135, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='0.006', node=129, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='0.105', node=120, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='0.241', node=101, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='0.002', node=93, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='0.002', node=91, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='0.001', node=86, frame = 'none', adj=c(1.2,-0.25), cex=0.7)

# species nodes
nodelabels(text='1.0', node=152, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='0.999', node=153, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='1.0', node=134, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='1.0', node=119, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='1.0', node=100, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='1.0', node=84, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='1.0', node=85, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='1.0', node=83, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='1.0', node=99, frame = 'none', adj=c(1.2,-0.25), cex=0.7)
nodelabels(text='1.0', node=98, frame = 'none', adj=c(1.2,-0.25), cex=0.8)

```
Return to [Clade VI Genomics: model-based species discovery](#model-based-species-discovery-11)
\pagebreak

### Fig S67: Genogroup delimitation: Reproductive isolation model
```{r cladeVImolecularDelimitationPopGenBpp, echo=FALSE, fig.width=6.5, dpi=300, fig.cap='Population genetic modeling for genogroup delimitation. Top panel: assignment of specimens to demes according to \\texttt{STRUCTURE} and posterior probability of species delimitation modeling according to \\texttt{BPP} using these demes. Bottom panel: assignment of specimens to demes according to \\texttt{MAVERICK} and posterior probability of species delimitation modeling according to \\texttt{BPP} using these demes. Specimens are sorted from north (left) to south (right) according to locality of collection.'}

mySTR = read.csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_smallestMiss75_STRUCTURE.csv", header=TRUE)
myRMav = read.csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_smallestMiss75_RMav.csv", header=TRUE)
# attach assignments
# IMPORTANT: make sure to confirm number of columns you need to attach (corresponds with number of groups to be plotted)
# number of STRUCTURE groups = 4
# number of RMaverick groups = 4
mergedGroupInfo = cbind(sampleInfo, mySTR[match(sampleInfo$MolecularSampleName, mySTR[,'MolecularSampleName']),2:5])
mergedGroupInfo = cbind(mergedGroupInfo, myRMav[match(sampleInfo$MolecularSampleName, myRMav[,'MolecularSampleName']),2:6])

# determine majority group assignment for each individual and attach these to the group information
# STRUCTURE groups are columns 12:13
# RMaverick groups are columns 14:18
mergedGroupInfo$STRassignment = as.factor(colnames(mergedGroupInfo[,12:13]) [apply(mergedGroupInfo[,12:13],1,which.max)])
mergedGroupInfo$RMavassignment = as.factor(colnames(mergedGroupInfo[,14:18]) [apply(mergedGroupInfo[,14:18],1,which.max)])

myGreys = c(brewer.pal(4,'Greys'))
myBlues = c(brewer.pal(5, 'Blues'))
STRplot = reshape::melt(mergedGroupInfo,
                        id.vars = c("SpecimenID","Latitude"),
                        measure.vars = c("group1",
                                         "group2",
                                         "group3",
                                         "group4")) %>%
  ggplot(aes(x = reorder(as.factor(SpecimenID), -Latitude),
             y = value,
             fill = as.factor(variable))) +
  geom_bar(stat = 'identity',
           show.legend = FALSE,
           color = 'black') +
  scale_fill_manual(values = myGreys[c(1,2,3,4)]) +
  ylab("Probability") +
  theme_bw() +
  #theme(axis.text.x = element_text(size=8, angle=90, vjust = 0.5, hjust=1)) +
  theme(axis.text.x = element_blank(),
        axis.title.x = element_blank()) +
  theme(text = element_text(size = 8))

RMavplot = reshape::melt(mergedGroupInfo,
                         id.vars = c("SpecimenID","Latitude"),
                         measure.vars = c("deme1",
                                          "deme2",
                                          "deme3",
                                          "deme4",
                                          "deme5")) %>%
  ggplot(aes(x = reorder(as.factor(SpecimenID), -Latitude),
             y = value,
             fill = as.factor(variable))) +
  geom_bar(stat = 'identity',
           show.legend = FALSE,
           color = 'black') +
  scale_fill_manual(values = myBlues[c(1,2,3,4,5)]) +
  ylab("Probability") +
  xlab("Specimens sorted by Latitudinal position") +
  theme_bw() +
  theme(axis.text.x = element_text(size = 4,
                                   angle = 90,
                                   vjust = 0.5,
                                   hjust = 1)) +
  theme(text = element_text(size = 8))
  #theme(axis.text.x = element_blank(), axis.title.x = element_blank())

mypatchwork = (STRplot + grid::textGrob("BPP post. probability\nfor 4 species = 1.0") + plot_layout(widths = c(1.5,0.5))) / (RMavplot + grid::textGrob("BPP post. probability\nfor 4 species = 1.0") + plot_layout(widths = c(1.5,0.5)))

mypatchwork + theme(text=element_text(size = 8))
```
Return to [Clade VI Genomics: model-based species discovery](#model-based-species-discovery-11)
\pagebreak

```{r cladeVIfinalDelimitation, echo=FALSE, include=FALSE}
# need to use the output of the morphometrics analysis, since it only includes samples that have BOTH flowers and leaves
morphDelim = read.csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_morphDR.csv")
molecularDelim = read.csv("supplementary_material_files/CladeVI_resultsFiles/CladeVI_molecularDelimitation.csv")
# if it doesn't have a SpecimenID column, you'll need to make one
molecularDelim = molecularDelim %>%
  separate(col = SampleName,
           into = c('SpecimenID',
                    'toss1',
                    'taxon',
                    'LibraryNumber'),
           remove = FALSE) %>%
  unite(col = 'SampleName',
        c(SpecimenID,
          toss1,
          taxon),
        sep = "_",
        remove = FALSE) %>%
  unite(col = 'MolecularSampleName',
        c(SpecimenID,
          toss1,
          taxon,
          LibraryNumber),
        sep = "_",
        remove = FALSE)

# join datasets, coalesce duplicate columns, select needed data
typesOfSpp_temp = full_join(morphDelim, molecularDelim, by="SpecimenID") %>%
  mutate(
    Latitude = coalesce(Latitude.x, Latitude.y),
    Longitude = coalesce(Longitude.x, Longitude.y),
    Elevation = coalesce(Elevation.x, Elevation.y)
  ) %>%
  select(MolecularSampleName,
         SpecimenID,
         Elevation,
         Latitude,
         Longitude,
         BestModel_Classification, #name of the morphoDelim column
         mPTP) # name of the molecularDelim column
# confirmation
#colnames(typesOfSpp_temp)
#view(typesOfSpp_temp) # confirm number of rows matches that of sampling table


# create plotting df
typesOfSpp = typesOfSpp_temp
colnames(typesOfSpp) = c("MolecularSampleName",
                         "SpecimenID",
                         "Elevation",
                         "Latitude",
                         "Longitude",
                         "PhenoGroups",
                         "GenoGroups")


# plot tree and match dataframe to tip names - ensures order of plotting is correct
myTree = read.tree("supplementary_material_files/CladeVI_resultsFiles/CladeVI_smallestMiss75_concat.contree")
myTree_midRoot = midpoint.root(myTree)
ggtree(myTree_midRoot)
myTipNames = get_taxa_name()
typesOfSpp = typesOfSpp[order(match(typesOfSpp$MolecularSampleName, myTipNames)),]
```

### Fig S68: Data integration
```{r cladeVIfinalDelimitation2, echo=FALSE, warning=FALSE, message=FALSE, fig.width=7, fig.height=7, dpi=300, fig.cap="Integration of phenotypic and genomic data with spatial information and evolutionary history. All specimens are assigned to their corresponding best fit phenogroup (shapes) and genogroup (colors). Specimens without phenotypic or genomic data (unknown specimens) are shown as asterisks and empty shapes, accordingly. Specimens are shown as tips of the maximum likelihood tree (left) used in the CA model analysis and mapped along latitude and elevation (right). Specimens assigned to a single phenogroup and a single genogroup delineate species that we determined as 'good species'. Specimens assigned to a single phenogroup across multiple genogroups delineate species that we determined as 'phenotypic cryptic species'. Specimens assigned to a single genogroup across multiple phenogroups delineate species that we determined as 'genetic cryptic species'."}
########################
# Phylogenetic context # ----------------------------------------------------------------------------------------
########################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups

# get number of colors and shapes that you'll need for plotting
#unique(typesOfSpp$PhenoGroups)
#unique(typesOfSpp$GenoGroups)

# set values for colors and shapes
myShapes = c(21,22,23,24,25,21,22,23,8)
names(myShapes) = c('1','2','3','4','5','6','7','8','NA')
myCols = c(brewer.pal(12,'Paired')[-11],"#ffffff")
names(myCols) = c('species1',
                  'species2',
                  'species3',
                  'species4',
                  'species5',
                  'species6',
                  'species7',
                  'species8',
                  'species9',
                  'species10',
                  'species11',
                  'NA')


###################
# Phylogenetic context # --------------------------------------------------------------------------------
###################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups

f = ggtree(myTree_midRoot) %<+% typesOfSpp + xlim_tree(0.005)
#f
f2 = f + geom_tippoint(aes(x = x,
                           fill = as.factor(GenoGroups),
                           shape = as.factor(PhenoGroups)),
                       size = 2.5,
                       stroke = 0.25) +
  scale_shape_manual(values = myShapes) +
  scale_fill_manual(values = myCols) +
  geom_tiplab(aes(label = SpecimenID),
              hjust = -0.5,
              size = 1) +
  geom_point2(aes(subset = PhenoGroups=='6'),
              pch = 3,
              size = 1,
              col = 'black') +
  geom_point2(aes(subset = PhenoGroups=='7'),
              pch = 3,
              size = 1,
              col = 'black') +
  geom_point2(aes(subset = PhenoGroups=='8'),
              pch = 3,
              size = 1,
              col = 'black') +
  geom_point2(aes(subset = is.na(PhenoGroups) & isTip,
                  color = as.factor(GenoGroups)),
              pch = 8,
              size = 1,
              stroke = 0.5,
              alpha = 0.5) +
  scale_color_manual(values = myCols) +
  theme(legend.position = "none")
#f2

# subset by major clades; clade numbers come from mPTP figure above
f3 = viewClade(f2,101) #green
f4 = viewClade(f2,119) #pinks
f5 = viewClade(f2,134) #oranges
f6 = viewClade(f2,152) #purple/yellow
f7 = viewClade(f2,84) #blue/green

###################
# Spatial context # ----------------------------------------------------------------------------------------
###################

# shape = PhenoGroups
# fill = GenoGroups
# color = matches morphologically unsampled (but genetically sampled) to genogroups

##### plot individuals with GenoGroup assignment, but no PhenoGroup assignment
g = ggplot(typesOfSpp,
           aes(x = Longitude,
               y = Elevation)) +
  geom_jitter(data = typesOfSpp %>%
                filter(is.na(PhenoGroups)),
              aes(x = Longitude,
                  y = Elevation,
                  color = as.factor(GenoGroups)),
              size = 2,
              alpha = 0.5,
              width = 0.75,
              height = 1.5,
              pch = 8,
              stroke = 1) +
  scale_color_manual(values = myCols) +
  scale_x_reverse(breaks=c(-30, -40, -50, -60, -70, -80)) +
  theme_bw() +
  theme(legend.position = 'none') +
  theme(panel.grid.major = element_line(size = 0.55,
                                        color = '#f0f0f0'))
#g

##### plot individuals with PhenoGroup assignment, but no GenoGroup assignment
g2 = g +
  geom_point(data=typesOfSpp %>%
               filter(is.na(GenoGroups)),
             aes(x = Longitude,
                 y = Elevation, shape=as.factor(PhenoGroups)),
             fill = 'white',
             size = 2) +
  scale_shape_manual(values = myShapes) +
  geom_point(data=typesOfSpp %>%
               filter(is.na(GenoGroups) & PhenoGroups=='6'),
             aes(x = Longitude,
                 y = Elevation),
             shape = 3,
             size = 1) +
  geom_point(data = typesOfSpp %>%
               filter(is.na(GenoGroups) & PhenoGroups=='7'),
             aes(x = Longitude,
                 y = Elevation),
             shape = 3,
             size = 1) +
  geom_point(data = typesOfSpp %>%
               filter(is.na(GenoGroups) & PhenoGroups=='8'),
             aes(x = Longitude,
                 y = Elevation),
             shape = 3,
             size = 1)
#g2

###### plots fully overlapping data (has both Geno- and Pheno- group assignments)
g3 = g2 +
  geom_point(data = typesOfSpp %>%
               filter(GenoGroups != 'NA' & PhenoGroups != 'NA'),
             aes(x = Longitude,
                 y = Elevation,
                 fill = as.factor(GenoGroups),
                 shape = as.factor(PhenoGroups)),
             size = 3.5,
             width = 0.75,
             height = 1.5) +
  scale_fill_manual(values = myCols)+
  geom_point(data = typesOfSpp %>%
               filter(GenoGroups != 'NA' & PhenoGroups=='6'),
             aes(x = Longitude,
                 y = Elevation),
             shape = 3,
             size = 1.5) +
  geom_point(data = typesOfSpp %>%
               filter(GenoGroups != 'NA' & PhenoGroups=='7'),
             aes(x = Longitude,
                 y = Elevation),
             shape = 3,
             size = 1.5) +
  geom_point(data = typesOfSpp %>%
               filter(GenoGroups != 'NA' & PhenoGroups=='8'),
             aes(x = Longitude,
                 y = Elevation),
             shape = 3,
             size = 1.5)
#g3

###### create the plot and save to an object
plotForLegend = ggplot(typesOfSpp,
                       aes(x = Longitude,
                           y = Elevation,
                           fill = as.factor(GenoGroups),
                           shape = as.factor(PhenoGroups))) +
  geom_point() +
  scale_fill_manual(values = myCols,
                    name = 'Genogroups',
                    labels=c('1','2','3','4','5','6','7','8','9','10','11','unknown')) +
  scale_shape_manual(values = myShapes,
                     name = 'Phenogroups',
                     labels=c('1','2','3','4','5','6','7','8','unknown')) +
  guides(fill = guide_legend(override.aes = list(fill = c(myCols[1:11],
                                                          'white'),
                                                 size = 3,
                                                 shape = 22,
                                                 color = 'black'))) +
  guides(shape = guide_legend(override.aes = list(shape = c(21,22,23,24,25,10,12,9,8),
                                                  size = 3))) +
  theme(legend.key = element_rect(fill = NA),
        text = element_text(size = 8))

legend = cowplot::get_legend(plotForLegend)
#g3 + legend


##########
# GK TAU # ----------------------------------------------------------------------------------------
##########
fullOverlap = typesOfSpp %>% filter(PhenoGroups != 'NA' & GenoGroups != 'NA')

gktau_morph2molecules = GKtau(fullOverlap$PhenoGroups,
                              fullOverlap$GenoGroups)

gktau_molecules2morph = GKtau(fullOverlap$GenoGroups,
                              fullOverlap$PhenoGroups)


####################################
# PLOT both together # --------------------------------------------
####################################
# one row
f2 + g3 + legend + plot_layout(widths=c(4,4,1))

```
Return to [Clade VI Data integration](#data-integration-6)
\pagebreak

# References
::: {#refs}
:::
\pagebreak

# (APPENDIX) Appendix {-}

# Appendix A: Manuscript in Spanish

**Un análisis genómico y fenómico integrado para investigar la naturaleza de las especies vegetales en _Escallonia_ (Escalloniaceae)**

Sarah J. Jacobs^1,2^, Michael C Grundler^1^, Claudia L. Henriquez^1^, Felipe Zapata^1^*

^1^Department of Ecology and Evolutionary Biology, University of California, Los Angeles, CA 90095

^2^Department of Botany, California Academy of Sciences, San Francisco CA 94118

\* Autor para correspondencia: fzapata@ucla.edu

**Resumen**

Lo que entendemos por especies y si tienen alguna realidad biológica se ha debatido desde los primeros días de la biología evolutiva. Algunos biólogos incluso sugieren que las especies vegetales son creadas por los taxónomos como una división subjetiva y artificial de la naturaleza. Sin embargo, la naturaleza de las especies vegetales rara vez se ha puesto a prueba de forma crítica con datos, ignorando la taxonomía. En este estudio, nosotros integramos datos fenómicos y genómicos recogidos en cientos de individuos a escala continental para investigar esta pregunta en _Escallonia_ (Escalloniaceae), un grupo de plantas que incluye 40 especies taxonómicas (las especies propuestas por los taxónomos). En primer lugar, mostramos que las especies taxonómicas pueden ser cuestionables, ya que se ajustan de manera limitada a los patrones de variación fenotípica y genética que muestran los individuos recogidos en la naturaleza. A continuación, utilizamos métodos estadísticos explícitos para la delimitación de las especies, diseñados para datos fenotípicos y genómicos, y mostramos que las especies vegetales existen en _Escallonia_ como una propiedad objetiva y discreta de la naturaleza, independiente de la taxonomía. Demostramos que tales especies corresponden de manera limitada con las especies taxonómicas actuales ($< 20\%$) y que los datos fenómicos y genómicos rara vez delimitan entidades congruentes ($< 20\%$). Estas discrepancias sugieren que fuerzas evolutivas adicionales al flujo génico pueden mantener la cohesión de las especies. Proponemos que los datos fenómicos y genómicos analizados en igualdad de condiciones permiten una perspectiva más holística para entender la naturaleza de las especies vegetales al ayudar a delimitar diferentes "tipos de especies". Nuestros resultados alertan los estudios que dan por sentada la exactitud de las especies taxonómicas y ponen en tela de juicio la noción de especies vegetales sin pruebas empíricas.

**Introducción**

Una pregunta perenne en biología se refiere a la posibilidad de que las especies vegetales no sean reales, sino presumiblemente construcciones artificales de la psique de los taxónomos [@lewis1959nature; @levin1979nature; @rieseberg2006nature]. Los investigadores que han abordado esta pregunta mediante el análsis de datos fenotípicos se han centrado en validar las especies taxonómicas (es decir, las especies propuestas por los taxónomos) [@rieseberg2006nature; @mayr1992local]. Esto significa utilizar las especies taxonómicas como referentes estándar para determinar la fuerza de la evidencia empírica en apoyo de la realidad de las especies cuando los investigadores analizan los datos fenotípicos con métodos de taxonomía numérica para identificar las especies [@sneath1973numerical]. En un artículo muy influyente, Rieseberg et al. [@rieseberg2006nature] recopilaron datos de 400 estudios que utilizaron métodos numéricos para identificar especies de plantas y animales con datos fenotípicos, y evaluaron hasta qué punto las especies delimitadas con métodos estadísticos coincidían con las especies taxonómicas. Este estudio reveló que la validación de las especies taxonómicas es baja ($< 60\%$ de los grupos discretos identificados estadísticamente son congruentes con las especies taxonómicas) aunque aparentemente existen grupos fenotípicos discretos en la mayoría de los grupos taxonómicos [@rieseberg2006nature]. Sin embargo, al utilizar un enfoque de validación de especies, en lugar de un enfoque de descubrimiento de especies [@carstens2013fail; @barraclough2019evolutionary], este estudio asumió que las especies taxonómicas existen como tal. Desgraciadamente, Rieseberg et al. [@rieseberg2006nature] no tuvieron acceso a enfoques estadísticos útiles para evaluar la realidad de las especies independientemente de la taxonomía ni a datos de secuencias multilocus como línea de evidencia adicional para investigar la naturaleza de las especies entre taxones. En consecuencia, la pregunta fundamental sobre la realidad de las especies vegetales independiente de la influencia de los taxónomos sigue sin entenderse de manera clara. Hasta la fecha, ningún estudio que integre datos fenotípicos y de ADN de todo el genoma ha evaluado la realidad de las especies de plantas para un grupo que incluye múltiples especies taxonómicas hipotéticas a una amplia escala geográfica. En este estudio, nosotros investigamos esta pregunta a través de análisis de delimitación de especies utilizando datos fenotípicos (aprox. 8.300 medidas cuantitativas) y genómicos (aprox. 1.000.000 de secuencias de ADN) en un conjunto de 848 individuos de _Escallonia_ (Escalloniaceae), un grupo de arbustos y árboles que abarca la región montañosa de Sudamérica (Fig. 1; Tabla Suplementaria S1).

Muchos estudios que incorporan el procedimiento de delimitación de especies presentan varias deficiencias relevantes para comprender la naturaleza de las especies vegetales. En primer lugar, la mayoría de los estudios que utilizan datos fenotípicos se basan en enfoques estadísticos desconectados de la teoría biológica y, por lo tanto, se ven comprometidos en la detección de especies biológicamente significativas [@cadena2018issues]. En particular, estos estudios suelen utilizar métodos que se basan en análisis gráficos visuales que transmiten poca información sobre las frecuencias de los fenotipos, excluyen rasgos fenotípicos potencialmente importantes para la detección de especies y utilizan medidas de tendencia central que son intrascendentes para evaluar el carácter distintivo de las especies [@cadena2018issues]. En segundo lugar, muchos estudios utilizan procedimientos numéricos explícitos para analizar los datos fenotípicos solo cuando analizan "taxones problemáticos" (es decir, complejos de especies, enjambres de híbridos), y por lo tanto pueden proporcionar una perspectiva general distorsionada sobre la naturaleza de las especies vegetales. En tercer lugar, algunos estudios no investigan directamente la naturaleza de las especies vegetales utilizando datos genéticos los cuales tienen una relación explícita con la divergencia evolutiva y el flujo génico, dos criterios relevantes en la delimitación de las especies [@de1998general]. Alternativamente, otros estudios se basan exclusivamente en datos genéticos, lo que puede llevar a que no se descubran especies que mantienen la cohesión y la independencia a través de fuerzas evolutivas adicionales al flujo génico [@cadena2021genomic]. Por último, varios estudios no consideran la delimitación de las especies en un contexto geográfico a pesar del papel central de la geografía en el estudio de la naturaleza de las especies [@mayr1970populations; @levin2000origin]. En este estudio, nosotros abordamos estas limitaciones en el examen de la naturaleza de las especies vegetales mediante la integración de múltiples tipos de datos y utilizando enfoques estadísticos adecuados bien fundamentados en la teoría evolutiva, en _Escallonia_, un género típico de plantas con flores, aparentemente compuesto por especies taxonómicas "buenas" [@Sleumer:1968ts].

Los árboles y arbustos del género _Escallonia_ constituyen un excelente caso de estudio para llevar a cabo este tipo de análisis para investigar la naturaleza de las especies vegetales. Estas plantas se encuentran en una variedad de hábitats a lo largo de los Andes y las montañas del sureste de Brasil, así como en cordilleras aisladas como la Sierra de Córdoba (Argentina), la Sierra Nevada de Santa Marta (Colombia) y la Cordillera de Talamanca (Costa Rica) [@zapata2013multilocus; @sede2013phylogenetics]. La mayoría de las especies taxonómicas en _Escallonia_ tienen rangos geográficos amplios, con algunas especies que tienen poblaciones separadas por miles de kilómetros; unas pocas especies son de distribución geográfica limitada que abarca menos de 200 kilómetros. Algunas especies taxonómicas son comunes localmente, con aproximadamente 30-40 plantas por localidad, mientras que otras son raras, encontrándose pocos individuos en cualquier lugar (F. Zapata, obs. pers.). Varias especies taxonómicas parecen segregarse según el hábitat o la elevación, sin embargo los rangos geográficos de muchas especies se superponen total o parcialmente, de tal manera que los individuos de una especie taxonómica pueden ocurrir dentro del rango de dispersión potencial de gametos (semillas o polen) de otras especies taxonómicas (es decir, las especies taxonómicas exhiben simpatría en mosaico _sensu_ [@mallet2008hybridization]). 

En todas las especies taxonómicas, el fruto es una cápsula seca que se abre y libera las semillas, las cuales caen y son probablemente dispersadas por el viento o la gravedad. Se conoce muy poco acerca de la biología de la polinización de cualquier especie taxonómica [@valdivia2006floral], y a partir de observaciones circunstanciales en el campo, las flores de diferentes especies taxonómicas de _Escallonia_ parecen ser visitadas por un grupo diverso de insectos locales que también visitan otras plantas de géneros no relacionados. Es imperativo realizar estudios que cuantifiquen las barreras de aislamiento reproductivo en _Escallonia_ para entender el papel de las señales florales en la especiación. Morfológicamente, las especies taxonómicas de _Escallonia_ muestran una variación sustancial en el tamaño y la forma general de las hojas, probablemente asociada a las condiciones ecológicas y a los cambios de hábitat (F. Zapata, sin publicar). Las especies taxonómicas pueden tener flores individuales o inflorescencias con decenas o cientos de flores. Las flores muestran una considerable variación geográfica en el tamaño y la forma de los sépalos, pétalos y ovarios. El color de los pétalos varía de blanco verdoso a rosa o rojo intenso. La morfología y el número de cromosomas (n = 12) son los mismos para todas las especies taxonómicas examinadas hasta el momento [@zielinski1955escallonia; @sanders1983chromosome; @hanson2003first], y los horticultores han generado híbridos artificiales entre especies morfológicamente distintas que no crecen juntas en la naturaleza (por ejemplo, [@eastwood1929escallonias]). Sin embargo, no hay casos documentados de especiación híbrida o zonas híbridas estables en la naturaleza.

Por lo tanto, _Escallonia_ parece ser un género "típico" de plantas con flores que no se considera único o problemático desde el punto de vista taxonómico. Desde el punto de vista genético, no existen estudios que utilicen datos genómicos que incluyan varios individuos por especie taxonómica para todas las especies del área de distribución geográfica de _Escallonia_ (es decir, no se conoce el estado de las especies taxonómicas desde una perspectiva multilocus). Sin embargo, es útil reiterar que no hay hibridación o introgresión natural rampante documentada, no hay casos conocidos de poliploidía y, hasta donde sabemos, no hay agamospermia ni apomixis en este género. Desde una perspectiva morfológica, las especies taxonómicas parecen estar más o menos bien definidas; existe cierta variación, pero el género no es notable o inusual en este sentido. En conjunto, _Escallonia_ ofrece una gran oportunidad para estudiar en detalle los patrones geográficos de variación de los rasgos fenotípicos y la genómica para examinar la naturaleza de las especies vegetales.

Dilucidar la naturaleza de las especies vegetales tiene implicaciones más amplias que la taxonomía. En particular, determinar si las especies existen como propiedades objetivas de la naturaleza puede tener repercusiones en otras áreas de la biología que utilizan las especies como unidad de análisis. Además, la comparación de los patrones geográficos de variación en los datos fenotípicos y genéticos puede empezar a arrojar luz sobre el papel de las fuerzas evolutivas que intervienen en el origen, la evolución y la estructura de la biodiversidad. 

\setcounter{figure}{0}
\setcounter{table}{0}
\let\thefigure=\oldthefigure
\let\thetable=\oldthetable

```{r figure1, echo = F, fig.align = 'center', out.width="70%", fig.cap = "Historia filogenetica, muestreo y delimitacion de especies basada en modelos evolutivos. Arbol de maxima verosimilitud (ML) de Escallonia basado en datos de todo el genoma (izquierda) indicando los seis clados focales (Clade I-VI) de nuestro estudio. Para cada clado, la primera columna de la izquierda muestra el muestreo, con simbolos rellenos que indican los especimenes utilizados en los analisis fenotipicos y simbolos vacios los especimenes utilizados en los analisis genomicos; los recuadros muestran la distribucion de los especimenes a lo largo de la elevacion. La segunda columna a la derecha muestra los resultados del mejor modelo para la delimitacion de especies con datos fenotipicos (aca llamados fenogrupos); los fenogrupos se muestran con simbolos de diferentes formas en el espacio geografico. La tercera columna muestra los resultados del mejor modelo para la delimitacion de especies con datos genomicos (aca llamados genogrupos); los genogrupos se indican con simbolos de diferentes colores y como apice de los arboles ML no enraizados basados en matrices de loci concatenados y mapeados en el espacio geografico. La cuarta columna muestra la integracion de los fenogrupos y genogrupos con la historia evolutiva y la distribucion geografica para dilucidar la naturaleza de las especies vegetales; los especimenes sin datos fenotipicos y genomicos  estan designados como especimenes desconocidos."}

fig1_path = "figures/Figure1.png"
knitr::include_graphics(fig1_path)
```

**Resultados y Discusión**

A continuación presentamos y discutimos nuestros resultados de manera global en el contexto de toda la radiación de _Escallonia_. Los resultados detallados se pueden encontrar en el material suplementario (en idioma inglés).

**El estado actual de las especies taxonómicas**

Primero caracterizamos la historia evolutiva de _Escallonia_ utilizando diferentes enfoques filogenéticos con un subconjunto de especímenes que abarcan el rango geográfico de estas plantas a lo largo de Sudamérica (Fig. 1; Fig. Suplementaria S1, S2). En todos estos análisis, recuperamos consistentemente seis grupos de especies taxonómicas (en adelante, clados I-VI), en línea con un estudio previo basado en menos loci [@zapata2013multilocus]. Todos los clados están marcadamente restringidos a regiones geográficas, excepto el clado VI; este clado está principalmente restringido al sureste de Brasil, Uruguay y el noreste de Argentina, pero incluye algunas especies en los Andes (Fig. 1). Un examen más detallado de la relación entre la composición de los clados y las distribuciones geográficas y elevaciones dentro de los clados revela que cuando los especímenes de diferentes clados co-ocurren en estrecha proximidad espacial (por ejemplo, los clados I, II, III, IV en los Andes Tropicales), los clados son genéticamente distintos sin mezclarse (Fig. 1; Fig. Suplementaria S1, S2). Además, todos los clados tienen una composición consistente y reciben un fuerte apoyo estadístico cuando utilizamos diferentes enfoques para el análisis filogenético (Ver Métodos). Sin embargo, cuando incluimos múltiples especímenes de la misma especie taxonómica, varios de estos especímenes no son siempre los parientes más cercanos entre sí dentro (es decir, las especies taxonómicas son parafiléticas o polifiléticas; Fig. Suplementaria S2). Este resultado, junto con la marcada concordancia geográfica filogenética y la composición consistente de los clados, sugiere que aunque los clados son evolutivamente distintos, los límites de las especies dentro de los clados se beneficiarían de un análisis más minucioso[@zapata2013multilocus]. Por lo tanto, centramos nuestros análisis subsequentes de la variación fenotípica y genomica para investigar la naturaleza de las especies en _Escallonia_ clado por clado.

Para investigar el estado actual de las especies taxonómicas en _Escallonia_ a través de los datos fenotípicos, primero nos preguntamos si las especies taxonómicas son cuantitativamente distintas y luego nos preguntamos si los especímenes que se supone pertenecen a una especie taxonómica ocupan el morfoespacio delimitado por la combinación de rasgos que definen cada especie taxonómica. Para estos análisis, utilizamos los rasgos morfológicos -de hojas y flores- proporcionados en la descripción taxonómica de cada especie [@Sleumer:1968ts]. Nos centramos en estos rasgos porque las descripciones taxonómicas incluyen los caracteres útiles para distinguir todas las especies y para compararlas con otras especies [@winston1999describing]. Reconocemos que al centrarnos sólo en estos rasgos, podemos estar excluyendo rasgos relacionados con las diferencias funcionales de las especies (por ejemplo, rasgos funcionales de las plantas). Sin embargo, los rasgos utilizados en las descripciones taxonómicas proporcionan un punto de partida lógico para evaluar la naturaleza de las especies. Es a lo largo de tales dimensiones del fenotipo donde los taxónomos han hipotetizado previamente discontinuidades naturales y muchos de estos rasgos (ciertamente los rasgos florales) tienen relevancia biológica con respecto a la función reproductiva. Además, nuestro examen de aproximadamente 3.500 especímenes de herbario y el extenso trabajo de campo confirman una variación sustancial en los rasgos foliares y florales entre las especies taxonómicas.

Primero tabulamos los valores máximos y mínimos de diez rasgos cuantitativos continuos proporcionados en la descripción de cada especie (estos valores se derivan de especímenes no incluidos en el conjunto de datos actual). A continuación, utilizamos estos valores como vértices de un cubo de 10 lados para representar geométricamente cada especie en el espacio fenotípico y estimamos el solapamiento por pares entre todos los cubos de 10 lados dentro de cada clado. Este análisis muestra que las especies taxonómicas dentro de los clados ocupan regiones distintas del espacio fenotípico de 10 dimensiones con poco o ningún solapamiento (Tabla 1, Fig. Suplementaria S5, S16, S27, S38, S49, S60). Seguimos estos análisis basados en la geometría con un análisis de "predicción de coincidencia" en el que evaluamos si cada espécimen identificado a una especie taxonómica estaba dentro o fuera del cubo de 10 lados de su correspondiente especie basado en medidas cuantitativas de los mismo rasgos morfológicos que definen el cubo de 10 lados (Ver Métodos). En contra de lo esperado, estos análisis muestran que la mayoría ($99,2\%$) de los especímenes se encuentran fuera de su respectivo cubo de 10 lados. Además, el $98,4\%$ de los especímenes quedan fuera de cualquier cubo de 10 lados (Tabla 1, Fig. Suplementaria S5, S16, S27, S38, S49, S60). Esto significa que la mayoría de los especímenes tienen al menos una medida que cae fuera del rango de variación proporcionado en sus descripciones taxonómicas. El uso de rangos fijos para los valores de los rasgos en las descripciones de las especies implica que las especies corresponden a formas geométricas con límites definidos (por ejemplo, cubos de 10 lados). Dadas las propiedades estadísticas y matemáticas de los espacios de alta dimensionalidad, una vez que un espécimen supera el límite impuesto por incluso una dimensión del cubo de 10 lados correspondiente a su especie taxonómica, dicho espécimen cae inmediatamente fuera del cubo de 10 lados por completo (por ejemplo, la maldición de la dimensionalidad [@bellman1958dynamic; @hastie2009elements]). Debido a que la mayoría de los especímenes examinados aquí caen fuera de su respectivo cubo de 10 lados, sugerimos que las especies taxonómicas en _Escallonia_ pueden tener un poder limitado para capturar los patrones multidimensionales de variación fenotípica mostrados por los organismos en la naturaleza. 

Es poco probable que este resultado sea un artefacto de la monografía taxonómica [@Sleumer:1968ts] porque las descripciones originales de las especies citan un gran número de especímenes examinados que cubren el rango geográfico conocido de todas las especies. Los especímenes incluidos en nuestro análisis se recogieron en las mismas localidades donde se recogieron los especímenes citados en la monografía; incluso medimos algunos de los especímenes de herbario citados en las descripciones originales de las especies. Nuestros resultados ponen de manifiesto la necesidad de incluir datos a nivel de espécimen (y no solo de especie) en las descripciones taxonómicas y monografías en el futuro, y de utilizar enfoques probabilísticos que incorporen la varianza y la covarianza entre los rasgos para definir las especies con el fin de capturar la forma de las especies en la naturaleza. Aunque nuestros resultados se limitan a _Escallonia_, especulamos que esto puede ser un fenómeno generalizado en otros grupos [@pineda2020nature] porque las especies de plantas delimitadas y descritas con la morfología rara vez se basan en análisis estadísticos explícitos de la variación fenotípica basados en la teoría biológica [@mcdade1995species; @stevens2000botanical]. Por lo tanto, sugerimos que investigar la naturaleza de las especies vegetales basándose únicamente en la validación de las especies taxonómicas puede ser generalmente problemático.

```{r table1, warning=FALSE, message=FALSE, echo=FALSE}

allClades_currentStateTaxonomy = read_csv("tables/allClades_currentStateSpecies.csv")

kable(allClades_currentStateTaxonomy,
      format = "latex",
      caption = "\\textbf{El estado actual de las especies taxonomicas}",
      booktabs = T,
      longtable = T,
      align ="c") %>%
  kable_styling( font_size = 8,
                 full_width = T,
                 latex_options = "HOLD_position")
```

**Pruebas basadas en modelos evolutivos para identificar especies como entidades objetivas**

Utilizamos modelos de mezclas finitas Gaussianas (GFMM) [@mclachlan2004finite] dentro de los clados para determinar tanto el número de especies como la asignación de especímenes a las especies utilizando datos fenotípicos sin información previa sobre la taxonomía. Este marco analítico es adecuado para este problema porque implementa el modelo evolutivo que subyace al uso de la variación fenotípica cuantitativa y continua en el descubrimiento y delimitación de especies [@fisher1919xv; @cadena2018issues]. Para realizar este análisis, utilizamos los mismos especímenes y los mismos diez rasgos morfológicos diagnósticos que en nuestro análisis anterior (véase más arriba). Primero rotamos la matriz de datos original en ejes ortogonales utilizando estimadores de covarianza robustos y redujimos la dimensionalidad de los ejes ortogonales a sólo aquellos que optimizaban la forma, la orientación y el número de especies basadas en el fenotipo (en adelante, fenogrupos). Identificamos el mejor modelo de mezclas Gaussianas - GMM (modelo ingenuo) en cada clado en un marco de criterio de información bayesiano (BIC) y de probabilidad de datos completos integrados (ICL). Además, evaluamos el apoyo a modelos alternativos en los que asignamos especímenes a grupos definidos _a priori_, incluyendo especies taxonómicas (modelo Taxonomía), así como fenogrupos que definimos a ojo durante el examen de los especímenes y que eran independientes de la taxonomía (modelo Inconsciente de la Taxonomía). Los resultados de estos análisis se muestran en la Fig. 1 y en la Tabla 2. El modelo ingenuo fue el mejor apoyado para cinco de los seis clados ($\Delta$BIC$>8$), mientras que un clado tuvo apoyo ($\Delta$BIC$<1$) aunque el modelo no fue el mejor apoyado para este clado (Fig. Suplementaria S39). Estos resultados fueron insensibles al enfoque de selección del modelo (BIC o ICL) (véase el material suplementario). El fuerte rendimiento del modelo ingenuo no es inesperado debido a las severas limitaciones de los otros modelos no estadísticos para porponer límites de las especies sin considerar la forma, la orientación y el solapamiento arbitrario de los grupos en el espacio fenotípico multidimensional [@cadena2018issues] (Fig. Suplementaria S6, S17, S28, S39, S50, S61). Esto también es coherente con la predicción de que la naturaleza es, de hecho, discontinua [@dobzhansky1937genetics; @barraclough2015evolutionary] a pesar de las sugerencias de que las especies no son entidades objetivas discretas [@levin1979nature]. Además, debido a que la mayoría de los fenogrupos identificados dentro de los clados coocurren localmente en simpatría (Fig. 1; Fig. Suplementaria S6, S17, S28, S39, S50, S61), el estatus de las especies para estos grupos es aceptado bajo una amplia gama de definiciones de especie [@de1998general; @cadena2018issues; @coyne2004speciation; @mallet2008hybridization]. Sin embargo, los fenogrupos pueden ocultar especies distintas cuando fenotipos similares han evolucionado (o están evolucionando) de forma independiente [@fivser2018cryptic]. Por lo tanto, la incorporación de información filogenética es beneficiosa para comprender la naturaleza de las especies y decidir si todos los fenogrupos corresponden a especies distintas.

```{r table2, warning=FALSE, message=FALSE, echo=FALSE}

allClades_phenogroups = read_csv("tables/allClades_GMMresults.csv")

allClades_phenogroups %>%
  kable(format = "latex",
        escape = F,
        booktabs = T,
        linesep ="",
        caption = "\\textbf{Modelos de mezclas finitas gaussianas (GFMM) para delimitacion de fenogrupos y seleccion de modelos empleando el criterio de informacion bayesiano (BIC)}",
        align="c") %>%
  kable_styling(font_size = 8,
                latex_options = "HOLD_position")
```

Para identificar especies y asignar especímenes a especies dentro de clados utilizando datos genéticos, evaluamos el ajuste de tres modelos comunes de delimitación de especies. Estos modelos implementan tres definiciones diferentes de especies, a saber, especies definidas como grupos genotípicos [@mallet1995species; @hausdorf2010species] (modelo GC), especies definidas como el punto de transición de la cladogénesis a la anagénesis [@baum1995genealogical; @kapli2017multi] (modelo CA), y especies definidas como linajes reproductivamente aislados [@mayr1970populations; @yang2010bayesian] (modelo RI). Aclaramos que estas definiciones de especies no están vinculadas a ningún mecanismo de especiación concreto. Por ejemplo, bajo diferentes mecanismos de especiación ecológica o geográfica, las especies podrían diagnosticarse como la transición de la cladogénesis a la anagénesis, o como grupos genéticos aislados. Nuestro análisis no es una inferencia del proceso de especiación en sí mismo. Más bien, nuestro estudio es una búsqueda de patrones (es decir, especies), que luego interpretamos a la luz de escenarios de especiación plausibles (véase la sección siguiente). Para este análisis, recopilamos datos de todo el genoma para un subconjunto de los especímenes utilizados en nuestros análisis fenotípicos y comparamos modelos de delimitación de especies que compiten entre sí en un marco bayesiano utilizando factores de Bayes [@leache2014species] para identificar especies basadas en variación genética (en adelante, genogrupos). Dado que no se han propuesto especies taxonómicas ni otros grupos _a priori_ basados en datos genéticos, no evaluamos el apoyo a ningún otro modelo alternativo de delimitación de especies. La Fig. 1 y la Tabla 3 muestran los resultados de estos análisis. En general, el modelo CA superó a los modelos alternativos; en cinco de los seis clados, el modelo CA fue el modelo mejor soportado, mientras que el modelo GC se ajustó mejor sólo para un clado. Además, el modelo CA es adecuado para capturar las especies que acá identificamos dado los resultados de un análsis de adecuación (Tabla S2). En todos los clados, el modelo que mejor se ajustaba identificaba el mayor número de genogrupos. La razón por la que los modelos con más genogrupos se ajustan mejor en todos los clados es probablemente el resultado de la mayor variación genética entre genogrupos que dentro de los mismos, lo cual es evidente como largas ramas en los árboles filogenéticos de especies (Fig. 1). Esto sugiere que los genogrupos son linajes divergentes en trayectorias evolutivas separadas, y es consistente con la hipótesis de que tales linajes son especies distintas [@de1998general; @barraclough2019evolutionary]. Además, varios de estos genogrupos dentro de los clados coexisten localmente en simpatría y, por lo tanto, el estatus de especie para tales grupos se puede aceptar bajo múltiples definiciones de especie [@mayr1970populations; @coyne2004speciation; @mallet2008hybridization]. Sin embargo, en algunos clados los genogrupos forman grupos aislados y alopátricos de especímenes, lo que presumiblemente podría ser el resultado de un muestreo geográfico escaso dentro de una sola especie [@mason2020coalescent]. Por lo tanto, el peso de la evidencia en apoyo del estatus de especie para estos genogrupos es débil y requiere considerar otras líneas de evidencia en igualdad de condiciones.

```{r table3, warning=FALSE, message=FALSE, echo=FALSE}

allClades_Genogroups = read_csv("tables/allClades_BFDresults.csv")

allClades_Genogroups %>%
  kable(format = "latex",
        escape = F,
        booktabs = T,
        linesep = "",
        caption ="\\textbf{Modelos genomicos para la delimitacion de genogrupos y seleccion de modelos empleado factores de Bayes (BF)}",
        align = "c") %>%
  kable_styling(font_size = 8,
                latex_options = "HOLD_position") %>%
  footnote(alphabet = c("specimens assigned to demes using MAVERICK",
                        "specimens assigned to demes using STRUCTURE"),
           threeparttable = T)
```

**Integrando la variación fenotípica y genómica, con información espacial e historia evolutiva.**

Con los fenogrupos y genogrupos derivados de los análisis basados en los modelos evolutivos, pudimos examinar la naturaleza de las especies integrando los datos fenotípicos y genómicos en un contexto espacial y evolutivo explícito (Fig. 1; Fig. Suplementaria S13, S24, S35, S46, S57, S68). Para este análisis, primero asignamos cada espécimen a su correspondiente fenogrupo y genogrupo, similar a una tabla de contingencia de dos vías (Fig. 2). Esta asignación permitió identificar la congruencia -o la falta de ella- entre los grupos fenotípicos y genómicos. Algunos especímenes estaban incompletos (por ejemplo, estériles) y no pudieron ser evaluados para todos los rasgos fenotípicos, mientras que otros especímenes fallaron durante el procesamiento para el trabajo genómico (en adelante, especímenes desconocidos); no obstante, la distribución geográfica de estos especímenes desconocidos en relación con los especímenes con ambos tipos de datos puede informar de la asignación más parsimoniosa a fenogrupos o genogrupos (por ejemplo, en el clado IV todos los especímenes desconocidos del norte de Sudamérica probablemente pertenecen al fenogrupo $2$ y al genogrupo $1$; Fig. 1, panel 2). En general, encontramos que sólo un pequeño porcentaje de fenogrupos corresponden directamente con genogrupos únicos ($15\%$), incluso asumiendo una asignación de grupo concordante para todos los especímenes desconocidos ($18\%$). Por el contrario, encontramos que en la mayoría de los clados un determinado fenogrupo ocurre a través de múltiples genogrupos (por ejemplo, ver el fenogrupo $2$ en el clado IV, Fig. 2), y con menos frecuencia que un determinado genogrupo ocurre a través de diferentes fenogrupos (por ejemplo, ver el genogrupo $9$ en el clado V, Fig. 2). En conjunto, nuestros resultados sugieren que la proporción de "especies buenas" (es decir, grupos fenotípicos y genómicos distintos y congruentes) en _Escallonia_ es notablemente baja, especialmente teniendo en cuenta la noción generalizada en biología de que las "especies buenas" son la norma, y sugieren que otros tipos de especies, incluidas las "especies fenotípicas crípticas" [@fivser2018cryptic] (es decir, un fenogrupo a través de múltiples genogrupos) y "especies crípticas genéticas" [@cadena2021genomic] (es decir, un genogrupo a través de múltiples fenogrupos), son más comunes. La existencia de estos diferentes tipos de especies es coherente con la idea de que las propiedades de las especies, como la distinguibilidad morfológica o la exclusividad genealógica de los alelos, pueden evolucionar en diferentes momentos y orden secuencial debido a la naturaleza heterogénea del proceso de especiación [@baum1998individuality; @de2007species].

```{r figure2, echo = F, fig.align = 'center', out.width="80%", fig.cap = "Integracion de la variacion fenotipica y genomica para delimitar especies. Para cada clado (vease la Fig. 1), asignamos especimenes a su correspondiente fenogrupo y genogrupo basandonos en los modelos de mejor ajuste para cada tipo de datos. Las celdas sombreadas muestran los especimenes asignados a una combinacion particular de fenogrupo y genogrupo de mejor ajuste (es decir, cada celda sombreada es una especie). Se reconocen tres tipos de especies. En primer lugar, los especimenes asignados unicamente a un solo fenogrupo y a un solo genogrupo se reconocen como 'especies buenas' (por ejemplo, fenogrupo $4$, genogrupo $3$ en el clado III). En segundo lugar, los especimenes asignados a un unico fenogrupo a traves de multiples genogrupos se reconocen como 'especies fenotipicas cripticas' (por ejemplo, fenogrupo $2$, genogrupos $1,2$ en el clado III). En tercer lugar, los especimenes asignados a un unico genogrupo a traves de multiples fenogrupos se reconocen como 'especies geneticas cripticas' (por ejemplo, fenogrupos $1,3$, genogrupo $5$, en el Clado III). Las filas o columnas vacias corresponden a especimenes que no tenian datos fenotipicos y genomicos sobrepuestos y, por lo tanto, se asignaron solo a su correspondiente fenogrupo o genogrupo, segun corresponda (por ejemplo, el genogrupo $2$ en el Clado I)."}

fig2_path = "figures/Figure2.png"
knitr::include_graphics(fig2_path)
```

La interpretación de las especies que identificamos en un contexto espacial y filogenético explícito puede dilucidar aún más la naturaleza de las especies vegetales. Nuestra motivación es ofrecer una interpretación del tipo de especies que descubrimos (patrón) a la luz de los mecanismos de especiación plausibles (proceso). No obstante, señalamos que es fundamental seguir trabajando con un muestreo más denso y con enfoques analíticos específicos para inferir el proceso de especiación real. La mayoría de las "especies buenas" se encuentran en simpatría local o se segregan según la elevación con otras especies (Fig. 1, Fig. 2; Fig. Suplementaria S13, S24, S35, S46, S57, S68). Esto sugiere que la selección mediada por el medio ambiente en simpatría o a lo largo de gradientes de elevación en parapatría puede ser una importante fuerza evolutiva que impulsa la especiación [@filatov2016demographic] o al menos mantiene las diferencias entre especies en _Escallonia._ Aunque estas especies pueden diferir en reasgos florales y foliares, se necesitan estudios sobre la biología reproductiva y el papel de otros factores bióticos y abióticos para desentrañar cómo se originan y se mantienen las "buenas especies" en _Escallonia_ en la naturaleza. Alternativamente, es posible que estas especies simplemente estén más adelantadas en el proceso de especiación y hayan acumulado suficientes diferencias [@weir2011limits; @singhal2013reproductive]. Se necesitan más muestreos en combinación con enfoques de datación filogenética y datos experimentales en _Escallonia_ para evaluar estas hipótesis con mayor rigor. 

Cuando los genogrupos de las "especies fenotípicas crípticas" están relacionados de forma distante, una hipótesis razonable para explicar este patrón es la idea de una evolución convergente en los fenotipos en respuesta a regímenes selectivos similares, ya sea en simpatría o alopatría [@struck2018finding] (por ejemplo, véase el fenogrupo $1$, los genogrupos $2, 4, 10, 11$, el clado VI; Fig. 1). _Escallonia_ ocurre en hábitats de montaña que muestran condiciones ambientales similares a través de regiones geográficas separadas (por ejemplo, las montañas del sureste de Brasil, el sur de los Andes y los Andes Tropicales de alta elevación) [@zapata2013multilocus]. La posibilidad de una evolución replicada de especies con rasgos foliares y florales similares a través de regiones geográficas separadas como un archipiélago de montaña es intrigante y debe ser investigada en detalle. Por el contrario, cuando estos genogrupos son los parientes más cercanos entre sí y no coocurren localmente en simpatría (por ejemplo, véase el fenogrupo $2$, genogrupos $1, 2$, clado III; Fig. 1), bajo algunas definiciones de especies tales genogrupos pueden corresponder a poblaciones alopátricas dentro de una misma especie [@mayr1970populations] en lugar de ser especies distintas resultantes de una especiación reciente con poco tiempo para la diferenciación fenotípica, o una especiación con conservatismo de nicho [@wiens2004speciation; @struck2018finding]. Es necesario un muestreo geográfico exhaustivo antes de poder confrontar estas hipótesis con seguridad y entender mejor la naturaleza de estas especies en _Escallonia_. 

En todas las "especies genéticas crípticas" que identificamos, los fenogrupos no muestran una fuerte estructura geográfica (por ejemplo, véase el genogrupo $10$, los fenogrupos $2, 3, 5, 7$, el clado V; Fig. 1). Esto es consistente con la intrigante posibilidad de que estas especies, que pese a ser fenotípicamente distintas, podrían estar potencialmente interconectadas a través del intercambio de genes [@lotsy1925species; @cronk2018role], probablemente facilitado por su amplio solapamiento en el espacio geográfico [@zapata2013multilocus]. Todavía no se sabe si este patrón refleja especiación con flujo génico o si el flujo génico ocurre después del contacto secundario. Nuestro muestreo actual en _Escallonia_ no está diseñado para desentrañar estas posibilidades y se requieren más análisis. Sin embargo, observamos que la evidencia genómica para este tipo de especies se está acumulando rápidamente para otras plantas [@novikova2016sequencing; @cannon2020oak; @wang2020genes] así como para varios taxones a través del árbol de la vida [@mallet2016reticulated; @cadena2021genomic]. En otros grupos taxonómicos, este tipo de especies incluye tanto especies de divergencia reciente, que posiblemente se diferencian con flujo génico, como especies con más de 10-20 millones de años de divergencia con el subsiguiente flujo génico que se produce tras el contacto secundario [@hipp2020genomic, @barth2020stable]. Sin embargo, es necesario estudiar con más detalle cómo se inician y persisten estos grupos de especies, y qué parte de sus genomas se intercambia libremente a través de los límites de las especies sin que éstas colapsen [@harrison2014hybridization]. Además, argumentamos que el enfoque de descubrimiento de especies que empleamos aquí, en el que tanto el fenotipo como el genotipo contribuyen por igual y de forma independiente al patrón de las especies, es esencial para detectar este tipo de grupos de especies donde de otro modo son inaccesibles. _Escallonia_ constituye un excelente caso de estudio para abordar estas preguntas críticas en biología, aunque es necesario realizar más muestreos genómicos, fenómicos y geográficos. 

Alternativamente, estas "especies genéticas crípticas" pueden ser el resultado de eventos de divergencia rápida impulsados por factores que influyen en los rasgos relevantes para el aislamiento ecológico con poco tiempo para que los alelos se separen completamente entre las especies hermanas [@rundell2009adaptive]. Dado que varios fenogrupos dentro de un genogrupo a veces coocurren en simpatría de mosaico [@mallet2008hybridization] o se sustituyen entre sí a lo largo de la elevación [@zapata2013multilocus] (Resultados suplementarios), es plausible que la rápida divergencia en _Escallonia_ haya sido impulsada por nuevas oportunidades ecológicas debidas a los ciclos climáticos y a la orogenia de las montañas [@nevado2018pleistocene]. La falta de estudios experimentales sobre la ecología funcional de los rasgos foliares y florales en _Escallonia_ nos impide saber qué factores son los responsables de mantener la divergencia fenotípica mostrada por diferentes fenogrupos dentro de un mismo genogrupo. Algunos fenogrupos pueden diferir en rasgos florales que podrían tener relación con los polinizadores. Otros fenogrupos pueden variar más fuertemente en rasgos foliares que podrían estar relacionados con la adaptación a los entornos locales. Por lo tanto, es plausible que diferentes formas de selección mantengan las diferencias fenotípicas y contrarresten los efectos homogeneizadores del flujo génico en las especies nacientes, una posibilidad que requiere más investigación. Por lo tanto, se requieren más muestreos de taxones y genomas en combinación con modelos genómicos poblacionales explícitos que incorporen diferentes formas de selección en _Escallonia_ para aislar la señal de la separación incompleta de linajes de la hibridación [@edelman2019genomic] y modelar el papel de la selección entre especies hermanas y no hermanas en el contacto secundario. 

**Conclusión**

En resumen, nuestros análisis de un conjunto de datos fenotípicos y genómicos a gran escala, utilizando enfoques basados en modelos de última generación para el descubrimiento y la delimitación de especies, revelan que las especies vegetales existen en _Escallonia_ como una propiedad de la naturaleza independiente de la taxonomía [@barraclough2015evolutionary; @barraclough2019evolutionary]. Sin embargo, el patrón observado de discordancia excesiva entre las especies identificadas con datos fenotípicos y genómicos sugiere que, en ausencia de datos y pruebas explícitas, la suposición prevalente que las entidades fenotípicamente (o genéticamente) distintas son necesariamente "buenas especies" no está justificada. Además, la señal paralela de tal discordancia a través de clados divergentes en _Escallonia_ sugiere que esto puede ser un fenómeno generalizado, lo cual es consistente con los patrones emergentes sobre la naturaleza de las especies a través del árbol de la vida [@mallet2016reticulated; @fivser2018cryptic; @cannon2020oak; @wang2020genes; @barth2020stable; @cadena2021genomic]. El enfoque de descubrimiento de especies que utilizamos aquí, el cual considera explícitamente tanto los datos fenotípicos como los genéticos en igualdad de condiciones, es esencial para revelar patrones útiles para guiar nuestra inferencia de los probables procesos evolutivos en juego en la especiación. Estudios anteriores han propuesto que aproximadamente el 70% de las especies taxonómicas de plantas representan "especies buenas" [@rieseberg2006nature], pero esto nuestro estudio no apoya tal resultado. Por el contrario, nuestros resultados sugieren que el porcentaje de especies taxonómicas de _Escallonia_ que corresponden a "especies buenas" puede ser tan bajo como el 17%$ (Tabla 4, Tabla Suplementaria S4, S7, S10, S13, S16, S19). Dado que _Escallonia_ parece ser un género "típico" de plantas que no se considera único o problemático desde el punto de vista taxonómico (véase la Introducción), este resultado es notable. No conocemos conjuntos de datos de magnitud similar para otros grupos de plantas, pero especulamos que nuestros resultados pueden ser generalizados. En la medida en que nuestros resultados captan cualquier perspectiva generalizable sobre la naturaleza de las especies vegetales, reforzada por la pobre base teórica que subyace la delimitación de las especies vegetales en general [@mcdade1995species; @stevens2000botanical], nuestros resultados sugieren que los estudios en otras áreas de la biología que asumen que las especies taxonómicas representan entidades buenas y biológicamente reales pueden necesitar una re-evaluación crítica. Nuestros resultados subrayan la necesidad de realizar más estudios comparativos que combinen datos fenotípicos y genotípicos en otros taxones y a través de escalas espaciales amplias y estrechas para comprender de forma exhaustiva la naturaleza de las especies vegetales y así arrojar luz sobre las fuerzas evolutivas que actúan en la especiación y en el mantenimiento de las especies en la naturaleza [@barraclough2019evolutionary]. Dados los avances sin precedentes en fenómica, genómica y computación, nunca ha habido un momento más próspero para ser taxónomo que ahora.


```{r table4, warning=FALSE, message=FALSE, echo=FALSE}

allClades_finalSummary = read_csv("tables/allClades_finalSummary.csv")

kable(allClades_finalSummary,
      format = "latex",
      caption = "\\textbf{Correspondencia entre especies taxonomicas y los mejores fenogrupos y genogrupos.}",
      booktabs = T,
      longtable = T,
      align = "c") %>%
  kable_styling(font_size = 8,
                full_width = T,
                latex_options = "HOLD_position")
```
\pagebreak
# Appendix B: Analysis Session Information

These are the packages used to run all analyses and generate both the manuscript and supplementary material files:

```{r, sessioninfo, echo=FALSE, message=FALSE}
print(sessionInfo())
```
